<!DOCTYPE html>

<html lang="en">


<!doctype html>
<html lang="en" class="no-js">
  <head>
    
      <meta charset="utf-8">
      <meta name="viewport" content="width=device-width,initial-scale=1">
      
      
      
      
        <link rel="prev" href="../Latent%20space%20simulators/">
      
      
        <link rel="next" href="../PINNs/">
      
      
      <link rel="icon" href="../assets/images/favicon.png">
      <meta name="generator" content="mkdocs-1.5.3, mkdocs-material-9.5.12">
    
    
<title>Literature Survey (VPE)</title>

    
      <link rel="stylesheet" href="../assets/stylesheets/main.7e359304.min.css">
      
        
        <link rel="stylesheet" href="../assets/stylesheets/palette.06af60db.min.css">
      
      


    
    
  <!-- Add scripts that need to run before here -->
  <!-- Add jquery script -->
  <script src="https://code.jquery.com/jquery-3.7.1.js"></script>
  <!-- Add data table libraries -->
  <script src="https://cdn.datatables.net/2.0.1/js/dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/2.0.1/css/dataTables.dataTables.css">
  <!-- Load plotly.js into the DOM -->
	<script src='https://cdn.plot.ly/plotly-2.29.1.min.js'></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/buttons/3.0.1/css/buttons.dataTables.css">
  <!-- fixedColumns -->
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/dataTables.fixedColumns.js"></script>
  <script src="https://cdn.datatables.net/fixedcolumns/5.0.0/js/fixedColumns.dataTables.js"></script>
  <link rel="stylesheet" href="https://cdn.datatables.net/fixedcolumns/5.0.0/css/fixedColumns.dataTables.css">
  <!-- Already specified in mkdocs.yml -->
  <!-- <link rel="stylesheet" href="../docs/custom.css"> -->
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/dataTables.buttons.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.dataTables.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.10.1/jszip.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/pdfmake.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/pdfmake/0.2.7/vfs_fonts.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.html5.min.js"></script>
  <script src="https://cdn.datatables.net/buttons/3.0.1/js/buttons.print.min.js"></script>
  <!-- Google fonts -->
  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
  <!-- Intro.js -->
  <script src="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/intro.min.js"></script>
  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/intro.js@7.2.0/minified/introjs.min.css">


  <!-- 
      
     -->
  <!-- Add scripts that need to run afterwards here -->

    
      
        
        
        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&display=fallback">
        <style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
      
    
    
      <link rel="stylesheet" href="../assets/_mkdocstrings.css">
    
      <link rel="stylesheet" href="../custom.css">
    
    <script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
    
      

    
    
    
  </head>
  
  
    
    
      
    
    
    
    
    <body dir="ltr" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black">
  
    
    <input class="md-toggle" data-md-toggle="drawer" type="checkbox" id="__drawer" autocomplete="off">
    <input class="md-toggle" data-md-toggle="search" type="checkbox" id="__search" autocomplete="off">
    <label class="md-overlay" for="__drawer"></label>
    <div data-md-component="skip">
      
    </div>
    <div data-md-component="announce">
      
    </div>
    
    
      

<header class="md-header" data-md-component="header">
  <nav class="md-header__inner md-grid" aria-label="Header">
    <a href=".." title="Literature Survey (VPE)" class="md-header__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    <label class="md-header__button md-icon" for="__drawer">
      
      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"/></svg>
    </label>
    <div class="md-header__title" data-md-component="header-title">
      <div class="md-header__ellipsis">
        <div class="md-header__topic">
          <span class="md-ellipsis">
            Literature Survey (VPE)
          </span>
        </div>
        <div class="md-header__topic" data-md-component="header-topic">
          <span class="md-ellipsis">
            
              Parametrizing using ML
            
          </span>
        </div>
      </div>
    </div>
    
      
        <form class="md-header__option" data-md-component="palette">
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="default" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to dark mode"  type="radio" name="__palette" id="__palette_0">
    
      <label class="md-header__button md-icon" title="Switch to dark mode" for="__palette_1" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 6H7c-3.31 0-6 2.69-6 6s2.69 6 6 6h10c3.31 0 6-2.69 6-6s-2.69-6-6-6zm0 10H7c-2.21 0-4-1.79-4-4s1.79-4 4-4h10c2.21 0 4 1.79 4 4s-1.79 4-4 4zM7 9c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3z"/></svg>
      </label>
    
  
    
    
    
    <input class="md-option" data-md-color-media="" data-md-color-scheme="slate" data-md-color-primary="white" data-md-color-accent="black"  aria-label="Switch to light mode"  type="radio" name="__palette" id="__palette_1">
    
      <label class="md-header__button md-icon" title="Switch to light mode" for="__palette_0" hidden>
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M17 7H7a5 5 0 0 0-5 5 5 5 0 0 0 5 5h10a5 5 0 0 0 5-5 5 5 0 0 0-5-5m0 8a3 3 0 0 1-3-3 3 3 0 0 1 3-3 3 3 0 0 1 3 3 3 3 0 0 1-3 3Z"/></svg>
      </label>
    
  
</form>
      
    
    
      <script>var media,input,key,value,palette=__md_get("__palette");if(palette&&palette.color){"(prefers-color-scheme)"===palette.color.media&&(media=matchMedia("(prefers-color-scheme: light)"),input=document.querySelector(media.matches?"[data-md-color-media='(prefers-color-scheme: light)']":"[data-md-color-media='(prefers-color-scheme: dark)']"),palette.color.media=input.getAttribute("data-md-color-media"),palette.color.scheme=input.getAttribute("data-md-color-scheme"),palette.color.primary=input.getAttribute("data-md-color-primary"),palette.color.accent=input.getAttribute("data-md-color-accent"));for([key,value]of Object.entries(palette.color))document.body.setAttribute("data-md-color-"+key,value)}</script>
    
    
    
      <label class="md-header__button md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
      </label>
      <div class="md-search" data-md-component="search" role="dialog">
  <label class="md-search__overlay" for="__search"></label>
  <div class="md-search__inner" role="search">
    <form class="md-search__form" name="search">
      <input type="text" class="md-search__input" name="query" aria-label="Search" placeholder="Search" autocapitalize="off" autocorrect="off" autocomplete="off" spellcheck="false" data-md-component="search-query" required>
      <label class="md-search__icon md-icon" for="__search">
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"/></svg>
        
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"/></svg>
      </label>
      <nav class="md-search__options" aria-label="Search">
        
        <button type="reset" class="md-search__icon md-icon" title="Clear" aria-label="Clear" tabindex="-1">
          
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"/></svg>
        </button>
      </nav>
      
    </form>
    <div class="md-search__output">
      <div class="md-search__scrollwrap" data-md-scrollfix>
        <div class="md-search-result" data-md-component="search-result">
          <div class="md-search-result__meta">
            Initializing search
          </div>
          <ol class="md-search-result__list" role="presentation"></ol>
        </div>
      </div>
    </div>
  </div>
</div>
    
    
      <div class="md-header__source">
        <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
      </div>
    
  </nav>
  
</header>
    
    <div class="md-container" data-md-component="container">
      
      
        
          
            
<nav class="md-tabs" aria-label="Tabs" data-md-component="tabs">
  <div class="md-grid">
    <ul class="md-tabs__list">
      
        
  
  
  
    <li class="md-tabs__item">
      <a href=".." class="md-tabs__link">
        
  
    
  
  Home

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Time-series%20forecasting/" class="md-tabs__link">
        
  
    
  
  Time-series forecasting

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Symbolic%20regression/" class="md-tabs__link">
        
  
    
  
  Symbolic regression

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Neural%20ODEs/" class="md-tabs__link">
        
  
    
  
  Neural ODEs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Physics-based%20GNNs/" class="md-tabs__link">
        
  
    
  
  Physics-based GNNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Latent%20space%20simulators/" class="md-tabs__link">
        
  
    
  
  Latent space simulators

      </a>
    </li>
  

      
        
  
  
    
  
  
    <li class="md-tabs__item md-tabs__item--active">
      <a href="./" class="md-tabs__link">
        
  
    
  
  Parametrizing using ML

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../PINNs/" class="md-tabs__link">
        
  
    
  
  PINNs

      </a>
    </li>
  

      
        
  
  
  
    <li class="md-tabs__item">
      <a href="../Koopman%20operator/" class="md-tabs__link">
        
  
    
  
  Koopman operator

      </a>
    </li>
  

      
    </ul>
  </div>
</nav>
          
        
      
      <main class="md-main" data-md-component="main">
        <div class="md-main__inner md-grid">
          
            
              
                
              
              <div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation" hidden>
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    


  


<nav class="md-nav md-nav--primary md-nav--lifted" aria-label="Navigation" data-md-level="0">
  <label class="md-nav__title" for="__drawer">
    <a href=".." title="Literature Survey (VPE)" class="md-nav__button md-logo" aria-label="Literature Survey (VPE)" data-md-component="logo">
      
  <img src="../assets/VPE.png" alt="logo">

    </a>
    Literature Survey (VPE)
  </label>
  
    <div class="md-nav__source">
      <a href="https://github.com/VirtualPatientEngine/literatureSurvey" title="Go to repository" class="md-source" data-md-component="source">
  <div class="md-source__icon md-icon">
    
    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512"><!--! Font Awesome Free 6.5.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2023 Fonticons, Inc.--><path d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"/></svg>
  </div>
  <div class="md-source__repository">
    VPE/LiteratureSurvey
  </div>
</a>
    </div>
  
  <ul class="md-nav__list" data-md-scrollfix>
    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href=".." class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Home
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Time-series%20forecasting/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Time-series forecasting
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Symbolic%20regression/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Symbolic regression
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Neural%20ODEs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Neural ODEs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Physics-based%20GNNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Physics-based GNNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Latent%20space%20simulators/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Latent space simulators
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
    
  
  
  
    <li class="md-nav__item md-nav__item--active">
      
      <input class="md-nav__toggle md-toggle" type="checkbox" id="__toc">
      
      
      
      <a href="./" class="md-nav__link md-nav__link--active">
        
  
  <span class="md-ellipsis">
    Parametrizing using ML
  </span>
  

      </a>
      
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../PINNs/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    PINNs
  </span>
  

      </a>
    </li>
  

    
      
      
  
  
  
  
    <li class="md-nav__item">
      <a href="../Koopman%20operator/" class="md-nav__link">
        
  
  <span class="md-ellipsis">
    Koopman operator
  </span>
  

      </a>
    </li>
  

    
  </ul>
</nav>
                  </div>
                </div>
              </div>
            
            
              
                
              
              <div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc" >
                <div class="md-sidebar__scrollwrap">
                  <div class="md-sidebar__inner">
                    

<nav class="md-nav md-nav--secondary" aria-label="Table of contents">
  
  
  
  
</nav>
                  </div>
                </div>
              </div>
            
          
          
            <div class="md-content" data-md-component="content">
              <article class="md-content__inner md-typeset">
                
                  

  
  


  <h1>Parametrizing using ML</h1>

<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
</head>

<body>
  <p>
  <i class="footer">This page was last updated on 2024-07-23 15:59:58 UTC</i>
  </p>

  <div class="note info" onclick="startIntro()">
    <p>
      <button type="button" class="buttons">
        <div style="display: flex; align-items: center;">
        Click here for a quick intro of the page! <i class="material-icons">help</i>
        </div>
      </button>
    </p>
  </div>

  <!--
  <div data-intro='Table of contents'>
    <p>
    <h3>Table of Contents</h3>
      <a href="#plot1">1. Citations over time on Parametrizing using ML</a><br>
      <a href="#manually_curated_articles">2. Manually curated articles on Parametrizing using ML</a><br>
      <a href="#recommended_articles">3. Recommended articles on Parametrizing using ML</a><br>
    <p>
  </div>

  <div data-intro='Plot displaying number of citations over time 
                  on the given topic based on recommended articles'>
    <p>
    <h3 id="plot1">1. Citations over time on Parametrizing using ML</h3>
      <div id='myDiv1'>
      </div>
    </p>
  </div>
  -->

  <div data-intro='Manually curated articles on the given topic'>
    <p>
    <h3 id="manually_curated_articles">Manually curated articles on <i>Parametrizing using ML</i></h3>
    <table id="table1" class="display" style="width:100%">
    <thead>
      <tr>
          <th data-intro='Click to view the abstract (if available)'>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th data-intro='Highest h-index among the authors'>Highest h-index</th>
          <th data-intro='Recommended articles extracted by considering
                          only the given article'>
              View recommendations
              </th>
      </tr>
    </thead>
    <tbody>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e837965494c4edbec4d30832d31ba5639996da8" target='_blank'>
                Deep learning prediction of patient response time course from early data via neural-pharmacokinetic/pharmacodynamic modelling
                </a>
              </td>
          <td>
            James Lu, B. Bender, Jin Y. Jin, Y. Guan
          </td>
          <td>2020-10-22</td>
          <td>Nature Machine Intelligence</td>
          <td>35</td>
          <td>38</td>

            <td><a href='../recommendations/4e837965494c4edbec4d30832d31ba5639996da8' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="Many real-world systems such as social networks and moving planets are dynamic in nature, where a set of coupled objects are connected via the interaction graph and exhibit complex behavior along the time. For example, the COVID-19 pandemic can be considered as a dynamical system, where objects represent geographical locations (e.g., states) whose daily confirmed cases of infection evolve over time. Outbreak at one location may influence another location as people travel between these locations, forming a graph. Thus, how to model and predict the complex dynamics for these systems becomes a critical research problem. Existing work on modeling graph-structured data mostly assumes a static setting. How to handle dynamic graphs remains to be further explored. On one hand, features of objects change over time, influenced by the linked objects in the interaction graph. On the other hand, the graph itself can also evolve, where new interactions (links) may form and existing links may drop, which may in turn be affected by the dynamic features of objects. In this paper, we propose coupled graph ODE: a novel latent ordinary differential equation (ODE) generative model that learns the coupled dynamics of nodes and edges with a graph neural network (GNN) based ODE in a continuous manner. Our model consists of two coupled ODE functions for modeling the dynamics of edges and nodes based on their latent representations respectively. It employs a novel encoder parameterized by a GNN for inferring the initial states from historical data, which serves as the starting point of the predicted latent trajectories. Experiment results on the COVID-19 dataset and the simulated social network dataset demonstrate the effectiveness of our proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aaf2145f9998f304513c0c9b530ee9f7750c6f55" target='_blank'>
                Coupled Graph ODE for Learning Interacting System Dynamics
                </a>
              </td>
          <td>
            Zijie Huang, Yizhou Sun, Wei Wang
          </td>
          <td>2021-08-14</td>
          <td>Knowledge Discovery and Data Mining, Proceedings of the 27th ACM SIGKDD Conference on Knowledge Discovery & Data Mining</td>
          <td>51</td>
          <td>57</td>

            <td><a href='../recommendations/aaf2145f9998f304513c0c9b530ee9f7750c6f55' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6bd28606fbae3449f831248804264c9885e992f9" target='_blank'>
                CellBox: Interpretable Machine Learning for Perturbation Biology with Application to the Design of Cancer Combination Therapy.
                </a>
              </td>
          <td>
            Bo Yuan, Ciyue Shen, Augustin Luna, Anil Korkut, D. Marks, John Ingraham, C. Sander
          </td>
          <td>2020-12-21</td>
          <td>Cell systems</td>
          <td>71</td>
          <td>153</td>

            <td><a href='../recommendations/6bd28606fbae3449f831248804264c9885e992f9' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

        <tr id="We introduce a flexible, scalable Bayesian inference framework for nonlinear dynamical systems characterised by distinct and hierarchical variability at the individual, group, and population levels. Our model class is a generalisation of nonlinear mixed-effects (NLME) dynamical systems, the statistical workhorse for many experimental sciences. We cast parameter inference as stochastic optimisation of an end-to-end differentiable, block-conditional variational autoencoder. We specify the dynamics of the data-generating process as an ordinary differential equation (ODE) such that both the ODE and its solver are fully differentiable. This model class is highly flexible: the ODE right-hand sides can be a mixture of user-prescribed or “white-box” sub-components and neural network or “black-box” sub-components. Using stochastic optimisation, our amortised inference algorithm could seamlessly scale up to massive data collection pipelines (common in labs with robotic automation). Finally, our framework supports interpretability with respect to the underlying dynamics, as well as predictive generalization to unseen combinations of group components (also called “zero-shot” learning). We empirically validate our method by predicting the dynamic behaviour of bacteria that were genetically engineered to function as biosensors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/309c5ae93a4cabfd37747abd130866240e265b2d" target='_blank'>
                Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems
                </a>
              </td>
          <td>
            Geoffrey Roeder, Paul K. Grant, Andrew Phillips, Neil Dalchau, Edward Meeds
          </td>
          <td>2019-05-24</td>
          <td>ArXiv, International Conference on Machine Learning</td>
          <td>21</td>
          <td>28</td>

            <td><a href='../recommendations/309c5ae93a4cabfd37747abd130866240e265b2d' target='_blank'>
              <i class="material-icons">open_in_new</i></a>
            </td>

        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/ Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
          <th>View recommendations</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

  <div data-intro='Recommended articles extracted by contrasting
                  articles that are relevant against not relevant for Parametrizing using ML'>
    <p>
    <h3 id="recommended_articles">Recommended articles on <i>Parametrizing using ML</i></h3>
    <table id="table2" class="display" style="width:100%">
    <thead>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </thead>
    <tbody>

        <tr id="The advent of machine learning has led to innovative approaches in dealing with clinical data. Among these, Neural Ordinary Differential Equations (Neural ODEs), hybrid models merging mechanistic with deep learning models have shown promise in accurately modeling continuous dynamical systems. Although initial applications of Neural ODEs in the field of model-informed drug development and clinical pharmacology are becoming evident, applying these models to actual clinical trial datasets-characterized by sparse and irregularly timed measurements-poses several challenges. Traditional models often have limitations with sparse data, highlighting the urgent need to address this issue, potentially through the use of assumptions. This review examines the fundamentals of Neural ODEs, their ability to handle sparse and irregular data, and their applications in model-informed drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d55a1a4acc93d905399f2126e7beb524fd64ec38" target='_blank'>
              Bridging pharmacology and neural networks: A deep dive into neural ordinary differential equations.
              </a>
            </td>
          <td>
            Idris Bachali Losada, N. Terranova
          </td>
          <td>2024-07-11</td>
          <td>CPT: pharmacometrics & systems pharmacology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6fbaa841f1f012a9be06d56da6a1921f5d50f305" target='_blank'>
              MGLEP: Multimodal Graph Learning for Modeling Emerging Pandemics with Big Data
              </a>
            </td>
          <td>
            Khanh-Tung Tran, T. Hy, Lili Jiang, Xuan-Son Vu
          </td>
          <td>2023-10-23</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Despite the wealth of single-cell multi-omics data, it remains challenging to predict the consequences of novel genetic and chemical perturbations in the human body. It requires knowledge of molecular interactions at all biological levels, encompassing disease models and humans. Current machine learning methods primarily establish statistical correlations between genotypes and phenotypes but struggle to identify physiologically significant causal factors, limiting their predictive power. Key challenges in predictive modeling include scarcity of labeled data, generalization across different domains, and disentangling causation from correlation. In light of recent advances in multi-omics data integration, we propose a new artificial intelligence (AI)-powered biology-inspired multi-scale modeling framework to tackle these issues. This framework will integrate multi-omics data across biological levels, organism hierarchies, and species to predict causal genotype-environment-phenotype relationships under various conditions. AI models inspired by biology may identify novel molecular targets, biomarkers, pharmaceutical agents, and personalized medicines for presently unmet medical needs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2cb6f6c3c33448b3b08be22a2e7a7daad280d9d9" target='_blank'>
              AI-driven multi-omics integration for multi-scale predictive modeling of causal genotype-environment-phenotype relationships
              </a>
            </td>
          <td>
            You Wu, Lei Xie Ph.D. Program in Computer Science, The Graduate Center, The City College of New York, New York., Usa, Ph.D. Program in Biology, Biochemistry, Department of Computer Science, Hunter College, Helen, Robert Appel Alzheimers Disease Research Institute, Feil Family Brain, Mind Research Institute, Weill Cornell Medicine, Cornell University
          </td>
          <td>2024-07-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>63</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9de2e8855c3ac85fb287f6356a1f811bb41e6472" target='_blank'>
              Mechanism-based organization of neural networks to emulate systems biology and pharmacology models
              </a>
            </td>
          <td>
            John Mann, Hamed Meshkin, Joel Zirkle, Xiaomei Han, Bradlee Thrasher, Anik Chaturbedi, Ghazal Arabidarrehdor, Zhihua Li
          </td>
          <td>2024-05-27</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Predicting the trajectories of systems with unknown dynamics (\textit{i.e.} the governing rules) is crucial in various research fields, including physics and biology. This challenge has gathered significant attention from diverse communities. Most existing works focus on learning fixed system dynamics within one single system. However, real-world applications often involve multiple systems with different types of dynamics or evolving systems with non-stationary dynamics (dynamics shifts). When data from those systems are continuously collected and sequentially fed to machine learning models for training, these models tend to be biased toward the most recently learned dynamics, leading to catastrophic forgetting of previously observed/learned system dynamics. To this end, we aim to learn system dynamics via continual learning. Specifically, we present a novel framework of Mode-switching Graph ODE (MS-GODE), which can continually learn varying dynamics and encode the system-specific dynamics into binary masks over the model parameters. During the inference stage, the model can select the most confident mask based on the observational data to identify the system and predict future trajectories accordingly. Empirically, we systematically investigate the task configurations and compare the proposed MS-GODE with state-of-the-art techniques. More importantly, we construct a novel benchmark of biological dynamic systems, featuring diverse systems with disparate dynamics and significantly enriching the research field of machine learning for dynamic systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb4045f3e6f4dfdaceedc6d8a70fc7a232722b3e" target='_blank'>
              Learning System Dynamics without Forgetting
              </a>
            </td>
          <td>
            Xikun Zhang, Dongjin Song, Yushan Jiang, Yixin Chen, Dacheng Tao
          </td>
          <td>2024-06-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="The COVID-19 pandemic and influenza outbreaks have underscored the critical need for predictive models that can effectively integrate spatial and temporal dynamics to enable accurate epidemic forecasting. Traditional time-series analysis approaches have fallen short in capturing the intricate interplay between these factors. Recent advancements have witnessed the incorporation of graph neural networks and machine learning techniques to bridge this gap, enhancing predictive accuracy and providing novel insights into disease spread mechanisms. Notable endeavors include leveraging human mobility data, employing transfer learning, and integrating advanced models such as Transformers and Graph Convolutional Networks (GCNs) to improve forecasting performance across diverse geographies for both influenza and COVID-19. However, these models often face challenges related to data quality, model transferability, and potential overfitting, highlighting the necessity for more adaptable and robust approaches. This paper introduces the Graph Attention-based Spatial Temporal (GAST) model, which employs graph attention networks (GATs) to overcome these limitations by providing a nuanced understanding of epidemic dynamics through a sophisticated spatio-temporal analysis framework. Our contributions include the development and validation of the GAST model, demonstrating its superior forecasting capabilities for influenza and COVID-19 spread, with a particular focus on short-term, daily predictions. The model’s application to both influenza and COVID-19 datasets showcases its versatility and potential to inform public health interventions across a range of infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/628bd28abb35127fc3d4e794dfc8e0860b738427" target='_blank'>
              Modeling epidemic dynamics using Graph Attention based Spatial Temporal networks
              </a>
            </td>
          <td>
            Xiaofeng Zhu, Yi Zhang, Haoru Ying, Huanning Chi, Guanqun Sun, Lingxia Zeng
          </td>
          <td>2024-07-15</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="From pedestrians to Kuramoto oscillators, interactions between agents govern how a multitude of dynamical systems evolve in space and time. Discovering how these agents relate to each other can improve our understanding of the often complex dynamics that underlie these systems. Recent works learn to categorize relationships between agents based on observations of their physical behavior. These approaches are limited in that the relationship categories are modelled as independent and mutually exclusive, when in real world systems categories are often interacting. In this work, we introduce a level of abstraction between the physical behavior of agents and the categories that define their behavior. To do this, we learn a mapping from the agents' states to their affinities for each category in a graph neural network. We integrate the physical proximity of agents and their affinities in a nonlinear opinion dynamics model which provides a mechanism to identify mutually exclusive categories, predict an agent's evolution in time, and control an agent's behavior. We demonstrate the utility of our model for learning interpretable categories for mechanical systems, and demonstrate its efficacy on several long-horizon trajectory prediction benchmarks where we consistently out perform existing methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/667ae92834a3c09ccd46dc0cf9ad000cab20dfe8" target='_blank'>
              Relational Reasoning On Graphs Using Opinion Dynamics
              </a>
            </td>
          <td>
            Yulong Yang, Bowen Feng, Keqin Wang, Naomi Leonard, Adji B. Dieng, Christine Allen-Blanchette
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Therapeutics Data Commons (tdcommons.ai) is an open science initiative with unified datasets, AI models, and benchmarks to support research across therapeutic modalities and drug discovery and development stages. The Commons 2.0 (TDC-2) is a comprehensive overhaul of Therapeutic Data Commons to catalyze research in multimodal models for drug discovery by unifying single-cell biology of diseases, biochemistry of molecules, and effects of drugs through multimodal datasets, AI-powered API endpoints, new multimodal tasks and model frameworks, and comprehensive benchmarks. TDC-2 introduces over 1,000 multimodal datasets spanning approximately 85 million cells, pre-calculated embeddings from 5 state-of-the-art single-cell models, and a biomedical knowledge graph. TDC-2 drastically expands the coverage of ML tasks across therapeutic pipelines and 10+ new modalities, spanning but not limited to single-cell gene expression data, clinical trial data, peptide sequence data, peptidomimetics protein-peptide interaction data regarding newly discovered ligands derived from AS-MS spectroscopy, novel 3D structural data for proteins, and cell-type-specific protein-protein interaction networks at single-cell resolution. TDC-2 introduces multimodal data access under an API-first design using the model-view-controller paradigm. TDC-2 introduces 7 novel ML tasks with fine-grained biological contexts: contextualized drug-target identification, single-cell chemical/genetic perturbation response prediction, protein-peptide binding affinity prediction task, and clinical trial outcome prediction task, which introduce antigen-processing-pathway-specific, cell-type-specific, peptide-specific, and patient-specific biological contexts. TDC-2 also releases benchmarks evaluating 15+ state-of-the-art models across 5+ new learning tasks evaluating models on diverse biological contexts and sampling approaches. Among these, TDC-2 provides the first benchmark for context-specific learning. TDC-2, to our knowledge, is also the first to introduce a protein-peptide binding interaction benchmark.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f567716b20ddccaa9ad1764f64b79d4c2c6ec114" target='_blank'>
              TDC-2: Multimodal Foundation for Therapeutic Science
              </a>
            </td>
          <td>
            Alejandro Velez-Arce, Kexin Huang, Michelle M. Li, Xiang Lin, Wenhao Gao, Tianfan Fu, M. Kellis, Bradley L. Pentelute, M. Zitnik
          </td>
          <td>2024-06-21</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Synergistic drug combination prediction tasks based on the computational models have been widely studied and applied in the cancer field. However, most of models only consider the interactions between drug pairs and specific cell lines, without taking into account the multiple biological relationships of drug-drug and cell line-cell line that also largely affect synergistic mechanisms. To this end, here we propose a multi-modal deep learning framework, termed MDNNSyn, which adequately applies multi-source information and trains multi-modal features to infer potential synergistic drug combinations. MDNNSyn extracts topology modality features by implementing the multi-layer hypergraph neural network on drug synergy hypergraph and constructs semantic modality features through similarity strategy. A multi-modal fusion network layer with gated neural network is then employed for synergy score prediction. MDNNSyn is compared to five classic and state-of-the-art prediction methods on DrugCombDB and Oncology-Screen datasets. The model achieves area under the curve (AUC) scores of 0.8682 and 0.9013 on two datasets, an improvement of 3.70% and 2.71% over the second-best model. Case study indicates that MDNNSyn is capable of detecting potential synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfc49f5c1e8bd773b6cd154ec4110f9a46b6b817" target='_blank'>
              MDNNSyn: A Multi-Modal Deep Learning Framework for Drug Synergy Prediction.
              </a>
            </td>
          <td>
            Lei Li, Haitao Li, Tseren-Onolt Ishdorj, Chunhou Zheng, Yansen Su
          </td>
          <td>2024-07-02</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Recent innovations in diffusion probabilistic models have paved the way for significant progress in image, text and audio generation, leading to their applications in generative time series forecasting. However, leveraging such abilities to model highly stochastic time series data remains a challenge. In this paper, we propose a novel Stochastic Diffusion (StochDiff) model which learns data-driven prior knowledge at each time step by utilizing the representational power of the stochastic latent spaces to model the variability of the multivariate time series data. The learnt prior knowledge helps the model to capture complex temporal dynamics and the inherent uncertainty of the data. This improves its ability to model highly stochastic time series data. Through extensive experiments on real-world datasets, we demonstrate the effectiveness of our proposed model on stochastic time series forecasting. Additionally, we showcase an application of our model for real-world surgical guidance, highlighting its potential to benefit the medical community.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73dd22173a15d341be31e1667eb4f01cb6ece173" target='_blank'>
              Stochastic Diffusion: A Diffusion Probabilistic Model for Stochastic Time Series Forecasting
              </a>
            </td>
          <td>
            Yuansan Liu, S. Wijewickrema, Dongting Hu, C. Bester, Stephen O'Leary, James Bailey
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Drug-drug interaction (DDI) is the combined effects of multiple drugs taken together, which can either enhance or reduce each other’s efficacy. Thus, drug interaction analysis plays an important role in improving treatment effectiveness and patient safety. It has become a new challenge to use computational methods to accelerate drug interaction time and reduce its cost-effectiveness. The existing methods often do not fully explore the relationship between the structural information and the functional information of drug molecules, resulting in low prediction accuracy for drug interactions, poor generalization, and other issues. In this paper, we propose a novel method, which is a deep graph contrastive learning model for drug-drug interaction prediction (DeepGCL for brevity). DeepGCL incorporates a contrastive learning component to enhance the consistency of information between different views (molecular structure and interaction network), which means that the DeepGCL model predicts drug interactions by integrating molecular structure features and interaction network topology features. Experimental results show that DeepGCL achieves better performance than other methods in all datasets. Moreover, we conducted many experiments to analyze the necessity of each component of the model and the robustness of the model, which also showed promising results. The source code of DeepGCL is freely available at https://github.com/jzysj/DeepGCL.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/42bf8bf7852f5abca1db4b198c2dc1ec4b4a716e" target='_blank'>
              Deep graph contrastive learning model for drug-drug interaction prediction
              </a>
            </td>
          <td>
            Zhenyu Jiang, Zhi Gong, Xiaopeng Dai, Hongyan Zhang, Pingjian Ding, Cong Shen
          </td>
          <td>2024-06-17</td>
          <td>PLOS ONE</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Machine learning (ML) has been applied to predict the efficacy of biologic agents in ulcerative colitis (UC). ML can offer precision, personalization, efficiency, and automation. Moreover, it can improve decision support in predicting clinical outcomes. However, it faces challenges related to data quality and quantity, overfitting, generalization, and interpretability. This paper comments on two recent ML models that predict the efficacy of vedolizumab and ustekinumab in UC. Models that consider multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data are required for optimal shared decision-making and precision medicine. This paper also highlights the potential of combining ML with computational models to enhance clinical outcomes and personalized healthcare. Key Insights: (1) ML offers precision, personalization, efficiency, and decision support for predicting the efficacy of biologic agents in UC. (2) Challenging aspects in ML prediction include data quality, overfitting, and interpretability. (3) Multiple pathways, multiple ethnicities, and combinations of real-world and clinical trial data should be considered in predictive models for optimal decision-making. (4) Combining ML with computational models may improve clinical outcomes and personalized healthcare.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4fd22b7c6f685cae80958a1bc2dd8ae70573c792" target='_blank'>
              Machine Learning for Predicting Biologic Agent Efficacy in Ulcerative Colitis: An Analysis for Generalizability and Combination with Computational Models
              </a>
            </td>
          <td>
            Philippe Pinton
          </td>
          <td>2024-06-21</td>
          <td>Diagnostics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Abstract Summary Drug response is conventionally measured at the cell level, often quantified by metrics like IC50. However, to gain a deeper understanding of drug response, cellular outcomes need to be understood in terms of pathway perturbation. This perspective leads us to recognize a challenge posed by the gap between two widely used large-scale databases, LINCS L1000 and GDSC, measuring drug response at different levels—L1000 captures information at the gene expression level, while GDSC operates at the cell line level. Our study aims to bridge this gap by integrating the two databases through transfer learning, focusing on condition-specific perturbations in gene interactions from L1000 to interpret drug response integrating both gene and cell levels in GDSC. This transfer learning strategy involves pretraining on the transcriptomic-level L1000 dataset, with parameter-frozen fine-tuning to cell line-level drug response. Our novel condition-specific gene–gene attention (CSG2A) mechanism dynamically learns gene interactions specific to input conditions, guided by both data and biological network priors. The CSG2A network, equipped with transfer learning strategy, achieves state-of-the-art performance in cell line-level drug response prediction. In two case studies, well-known mechanisms of drugs are well represented in both the learned gene–gene attention and the predicted transcriptomic profiles. This alignment supports the modeling power in terms of interpretability and biological relevance. Furthermore, our model’s unique capacity to capture drug response in terms of both pathway perturbation and cell viability extends predictions to the patient level using TCGA data, demonstrating its expressive power obtained from both gene and cell levels. Availability and implementation The source code for the CSG2A network is available at https://github.com/eugenebang/CSG2A.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/33a0e168c290638894183c35c3f97f05d25895b1" target='_blank'>
              Transfer learning of condition-specific perturbation in gene interactions improves drug response prediction
              </a>
            </td>
          <td>
            D. Bang, Bonil Koo, Sun Kim
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="In pharmacometrics, developing and selecting models is crucial for quantitatively assessing drug-biological interactions, treatment planning, and gaining insights into underlying processes. These validated models are essential for predictive analytics and strategic decision-making in drug development and clinical practice. Unlike traditional methods, machine learning (ML) offers a data-driven alternative to conventional, first-principle approaches. This paper presents an automatic method to derive unknown or uncertain (sub)models using longitudinal, heterogeneous data. Initially, we employ deep nonlinear mixed effect (DeepNLME) to train a neural network as a universal approximator, which then generates a parameterized representation of the underlying process. Subsequently, we apply symbolic regression to identify a set of potential models expressed as equations. Within the study parameters, the proposed method outperforms the baseline models and demonstrates the validity of both the DeepNLME approach and the symbolic regression approach.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3e126fc4a7b791eeb4ca10543f627ff0e54a67c1" target='_blank'>
              Data-Driven Discovery of Feedback Mechanisms in Acute Myeloid Leukaemia: Alternatives to classical models using Deep Nonlinear Mixed Effect modeling and Symbolic Regression
              </a>
            </td>
          <td>
            Carl Julius Martensen, Niklas Korsbo, V. Ivaturi, Sebastian Sager
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Modeling dynamical systems is a fundamental task in scientific and engineering fields, often accomplished by applying theory-based models with mathematical equations. Yet, in cases where these equations cannot be established or parameterized properly, theory-based models are not applicable. Instead, a viable alternative is to learn the system dynamics directly from data, for example with deep learning models. However, traditional deep learning models often produce physically inconsistent results and struggle to generalize to unseen data, especially when training data is limited. One solution to this shortcoming is knowledge-guided deep learning, leveraging prior knowledge about the expected behavior of a dynamical system. In this work, we identify and formalize permissible system states, a novel type of prior knowledge that is often available for systems in the context of temporal dynamics modeling. This prior knowledge describes dynamic states that the system is allowed to take during its operation. We propose a knowledge-guided multi-state constraint to encode this type of prior knowledge through a loss function, making it applicable to any deep learning model. This approach allows to create an accurate data-driven model with minimal effort and data requirements. We validate the effectiveness of our method by applying it to model the temporal behavior of a gas turbine in response to an input control signal. Our results indicate that the proposed method reduces the prediction error by up to 40%. In addition to reducing the dependency on extensive training data, our method mitigates training randomness and enhances the consistency of predictions with the expected behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3ba124a5ffc81be0320f4553459c69561edd30f5" target='_blank'>
              Knowledge-Guided Learning of Temporal Dynamics and its Application to Gas Turbines
              </a>
            </td>
          <td>
            Pawel Bielski, Aleksandr Eismont, Jakob Bach, Florian Leiser, D. Kottonau, Klemens Böhm
          </td>
          <td>2024-05-31</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4648ca55b2c3ba80a310409f8b48533ab6d4a66a" target='_blank'>
              Learning interpretable dynamics of stochastic complex systems from experimental data
              </a>
            </td>
          <td>
            Tingting Gao, B. Barzel, Gang Yan
          </td>
          <td>2024-07-17</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Combination therapy is an important direction of continuous exploration in the field of medicine, with the core goals of improving treatment efficacy, reducing adverse reactions, and optimizing clinical outcomes. Machine learning technology holds great promise in improving the prediction of drug synergy combinations. However, most studies focus on single disease-oriented collaborative predictive models or involve excessive feature categories, making it challenging to predict the majority of new drugs. To address these challenges, the DrugSK comprehensive model was developed, which utilizes SMILES-BERT to extract structural information from 3492 drugs and trains on reactions from 48,756 drug combinations. DrugSK is an integrated learning model capable of predicting interactions among various drug categories. First, the primary learner is trained from the initial data set. Random forest, support vector machine, and XGboost model are selected as primary learners and logistic regression as secondary learners. A new data set is then "generated" to train level 2 learners, which can be thought of as a prediction for each model. Finally, the results are filtered using logistic regression. Furthermore, the combination of the new antibacterial drug Drafloxacin with other antibacterial agents was tested. The synergistic effect of Drafloxacin and Isavuconazonium in the fight against Candida albicans has been confirmed, providing enlightenment for the clinical treatment of skin infection. DrugSK's prediction is accurate in practical application and can also predict the probability of the outcome. In addition, the tendency of Drafloxacin and antifungal drugs to be synergistic was found. The development of DrugSK will provide a new blueprint for predicting drug combination synergies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/055325b217f8174f457e2c50697ff15bdc4f66ea" target='_blank'>
              DrugSK: A Stacked Ensemble Learning Framework for Predicting Drug Combinations of Multiple Diseases.
              </a>
            </td>
          <td>
            Siqi Chen, Nana Gao, Chunzhi Li, Fei Zhai, Xiwei Jiang, Peng Zhang, Jibin Guan, Kefeng Li, Rongwu Xiang, Guixia Ling
          </td>
          <td>2024-06-20</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Physics informed neural networks have been gaining popularity due to their unique ability to incorporate physics laws into data-driven models, ensuring that the predictions are not only consistent with empirical data but also align with domain-specific knowledge in the form of physics equations. The integration of physics principles enables the method to require less data while maintaining the robustness of deep learning in modeling complex dynamical systems. However, current PINN frameworks are not sufficiently mature for real-world ODE systems, especially those with extreme multi-scale behavior such as mosquito population dynamical modelling. In this research, we propose a PINN framework with several improvements for forward and inverse problems for ODE systems with a case study application in modelling the dynamics of mosquito populations. The framework tackles the gradient imbalance and stiff problems posed by mosquito ordinary differential equations. The method offers a simple but effective way to resolve the time causality issue in PINNs by gradually expanding the training time domain until it covers entire domain of interest. As part of a robust evaluation, we conduct experiments using simulated data to evaluate the effectiveness of the approach. Preliminary results indicate that physics-informed machine learning holds significant potential for advancing the study of ecological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bf1c97b75409ff1ee6e5c4298dd05295c9b772df" target='_blank'>
              Adapting Physics-Informed Neural Networks To Optimize ODEs in Mosquito Population Dynamics
              </a>
            </td>
          <td>
            D. V. Cuong, Branislava Lali'c, Mina Petri'c, Binh Nguyen, M. Roantree
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="
 The growing time-series data make it possible to glimpse the hidden dynamics in various fields. However, developing a computational toolbox with high interpretability to unveil the interaction dynamics from data remains a crucial challenge. Here, we propose a new computational approach called Automated Dynamical Model Inference based on Expression Trees (ADMIET), in which the machine learning algorithm, the numerical integration of ordinary differential equations and the interpretability from prior knowledge are embedded into the symbolic learning scheme to establish a general framework for revealing the hidden dynamics in time-series data. ADMIET takes full advantage of both machine learning algorithm and expression tree. Firstly, we translate the prior knowledge into constraints on the structure of expression tree, reducing the search space and enhancing the interpretability. Secondly, we utilize the proposed adaptive penalty function to ensure the convergence of gradient descent algorithm and the selection of the symbols. Compared to gene expression programming, ADMIET exhibits its remarkable capability in function fitting with higher accuracy and broader applicability. Moreover, ADMIET can better fit parameters in nonlinear forms compared to regression methods. Furthermore, we apply ADMIET to two typical biological systems and one real data with different prior knowledge to infer the dynamical equations. The results indicate that ADMIET can not only discover the interaction relationships but also provide accurate estimates of the parameters in the equations. These results demonstrate ADMIET's superiority in revealing interpretable dynamics from time-series biological data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9687aa0373c8e0262b0df16232ea9a08145ff46c" target='_blank'>
              Inferring dynamical models from time-series biological data using an interpretable machine learning method based on weighted expression trees
              </a>
            </td>
          <td>
            Yu Zhou, Xiufen Zou
          </td>
          <td>2024-07-09</td>
          <td>Inverse Problems</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this article, a distributed neural network modeling framework including a novel neural hybrid system model is proposed for enhancing the scalability of neural network models in modeling dynamical systems. First, high-dimensional training data samples will be mapped to a low-dimensional feature space through the principal component analysis (PCA) featuring process. Following that, the feature space is bisected into multiple partitions based on the variation of the Shannon entropy under the maximum entropy (ME) bisecting process. The behavior of subsystems in the prespecified state space partitions will then be approximated using a group of shallow neural networks (SNNs) known as extreme learning machines (ELMs), and then it can further simplify the model by merging the redundant lattices based on their training error performance. The proposed modeling framework can handle high-dimensional dynamical system modeling problems with the advantages of reducing model complexity and improving model performance in training and verification. To demonstrate the effectiveness of the proposed modeling framework, examples of modeling the LASA dataset and an industrial robot are presented.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7a2a2b97f2bbf0735400cc24119a8d6645252543" target='_blank'>
              A Distributed Neural Hybrid System Learning Framework in Modeling Complex Dynamical Systems.
              </a>
            </td>
          <td>
            Yejiang Yang, Tao Wang, Weiming Xiang
          </td>
          <td>2024-06-28</td>
          <td>IEEE transactions on neural networks and learning systems</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Measles is an important infectious disease system both for its burden on public health and as an opportunity for studying nonlinear spatio-temporal disease dynamics. Traditional mechanistic models often struggle to fully capture the complex nonlinear spatio-temporal dynamics inherent in measles outbreaks. In this paper, we first develop a high-dimensional feed-forward neural network model with spatial features (SFNN) to forecast endemic measles outbreaks and systematically compare its predictive power with that of a classical mechanistic model (TSIR). We illustrate the utility of our model using England and Wales measles data from 1944-1965. These data present multiple modeling challenges due to the interplay between metapopulations, seasonal trends, and nonlinear dynamics related to demographic changes. Our results show that, while the TSIR model yields more accurate very short-term (1 to 2 biweeks ahead) forecasts for highly populous cities, overall, our neural network model (SFNN) outperforms the TSIR in other forecasting windows. Furthermore, we show that our spatial-feature neural network model, without imposing mechanistic assumptions a priori, can uncover gravity-model-like spatial hierarchy of measles spread in which major cities play an important role in driving regional outbreaks. We then turn our attention to integrative approaches that combine mechanistic and machine learning models. Specifically, we investigate how the TSIR can be utilized to improve a state-of-the-art approach known as Physics-Informed-Neural-Networks (PINN) which explicitly combines compartmental models and neural networks. Our results show that the TSIR can facilitate the reconstruction of latent susceptible dynamics, improving both forecasts and parameter inference of measles dynamics within the PINN. In summary, our results show that appropriately designed neural network-based models can outperform traditional mechanistic models for short to long-term forecasts, while simultaneously providing mechanistic interpretability. Our work also provides valuable insights into more effectively integrating machine learning models with mechanistic models to enhance public health responses to measles and similar infectious disease systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ead0a3afaa03e20bf2cc5c98fefe3fd05bdd26ba" target='_blank'>
              Neural networks for endemic measles dynamics: comparative analysis and integration with mechanistic models
              </a>
            </td>
          <td>
            Wyatt G. Madden, Wei Jin, Benjamin Lopman, Andreas Zufle, Benjamin Dalziel, C. Jessica, E. Metcalf, B. T. Grenfell, Max S. Y. Lau
          </td>
          <td>2024-05-31</td>
          <td>None</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e559aa95c11591c1701e716bb52a8e5a7008b92" target='_blank'>
              Developing a novel causal inference algorithm for personalized biomedical causal graph learning using meta machine learning
              </a>
            </td>
          <td>
            Hang Wu, Wenqi Shi, May D. Wang
          </td>
          <td>2024-05-27</td>
          <td>BMC Medical Informatics and Decision Making</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Abstract Motivation In recent years, applying computational modeling to systems biology has caused a substantial surge in both discovery and practical applications and a significant shift in our understanding of the complexity inherent in biological systems. Results In this perspective article, we briefly overview computational modeling in biology, highlighting recent advancements such as multi-scale modeling due to the omics revolution, single-cell technology, and integration of artificial intelligence and machine learning approaches. We also discuss the primary challenges faced: integration, standardization, model complexity, scalability, and interdisciplinary collaboration. Lastly, we highlight the contribution made by the Computational Modeling of Biological Systems (SysMod) Community of Special Interest (COSI) associated with the International Society of Computational Biology (ISCB) in driving progress within this rapidly evolving field through community engagement (via both in person and virtual meetings, social media interactions), webinars, and conferences. Availability and implementation Additional information about SysMod is available at https://sysmod.info.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1b174dea12ab92a7f7139cb9122517aefde98aa6" target='_blank'>
              Perspectives on computational modeling of biological systems and the significance of the SysMod community
              </a>
            </td>
          <td>
            B. L. Puniya, Meghna Verma, Chiara Damiani, Shaimaa Bakr, Andreas Dräger
          </td>
          <td>2024-06-26</td>
          <td>Bioinformatics Advances</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="RNA-targeting drug discovery is undergoing an unprecedented revolution. Despite recent advances in this field, developing data-driven deep learning models remains challenging due to the limited availability of validated RNA-small molecule interactions and the scarcity of known RNA structures. In this context, we introduce RNAsmol, a novel sequence-based deep learning framework that incorporates data perturbation with augmentation, graph-based molecular feature representation and attention-based feature fusion modules to predict RNA-small molecule interactions. RNAsmol employs perturbation strategies to balance the bias between true negative and unknown interaction space thereby elucidating the intrinsic binding patterns between RNA and small molecules. The resulting model demonstrates accurate predictions of the binding between RNA and small molecules, outperforming other methods with average improvements of ∼8% (AUROC) in 10-fold cross-validation, ∼16% (AUROC) in cold evaluation (on unseen datasets), and ∼30% (ranking score) in decoy evaluation. Moreover, we use case studies to validate molecular binding hotspots in the prediction of RNAsmol, proving the model’s interpretability. In particular, we demonstrate that RNAsmol, without requiring structural input, can generate reliable predictions and be adapted to many RNA-targeting drug design scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/40adf2258ba8dff762ea347712f4628df142c400" target='_blank'>
              RNA-ligand interaction scoring via data perturbation and augmentation modeling
              </a>
            </td>
          <td>
            Hongli Ma, Letian Gao, Yun-fei Jin, Yilan Bai, Xiaofan Liu, Pengfei Bao, Ke Liu, Zhenjiang Zech Xu, Z. Lu
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Abstract Recent advances in microfluidics and sequencing technologies allow researchers to explore cellular heterogeneity at single-cell resolution. In recent years, deep learning frameworks, such as generative models, have brought great changes to the analysis of transcriptomic data. Nevertheless, relying on the potential space of these generative models alone is insufficient to generate biological explanations. In addition, most of the previous work based on generative models is limited to shallow neural networks with one to three layers of latent variables, which may limit the capabilities of the models. Here, we propose a deep interpretable generative model called d-scIGM for single-cell data analysis. d-scIGM combines sawtooth connectivity techniques and residual networks, thereby constructing a deep generative framework. In addition, d-scIGM incorporates hierarchical prior knowledge of biological domains to enhance the interpretability of the model. We show that d-scIGM achieves excellent performance in a variety of fundamental tasks, including clustering, visualization, and pseudo-temporal inference. Through topic pathway studies, we found that d-scIGM-learned topics are better enriched for biologically meaningful pathways compared to the baseline models. Furthermore, the analysis of drug response data shows that d-scIGM can capture drug response patterns in large-scale experiments, which provides a promising way to elucidate the underlying biological mechanisms. Lastly, in the melanoma dataset, d-scIGM accurately identified different cell types and revealed multiple melanin-related driver genes and key pathways, which are critical for understanding disease mechanisms and drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/39b1eecd23692d90bedf7211e73433834e2697ed" target='_blank'>
              Comprehensive single-cell RNA-seq analysis using deep interpretable generative modeling guided by biological hierarchy knowledge
              </a>
            </td>
          <td>
            Hegang Chen, Yuyin Lu, Zhiming Dai, Yuedong Yang, Qing Li, Yanghui Rao
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Graph convolutional neural networks (GCNs) have shown promising performance in modeling graph data, particularly for small-scale molecules. Message-passing neural networks (MPNNs) are an important form of GCN variant. They excel at gathering and integrating particular information about molecules via several repetitions of message transmission. This capability has resulted in major advances in molecular modeling and property prediction. By combining the self-attention mechanism with MPNNs, there is potential to improve molecular representation while using Transformers' proven efficacy in other artificial intelligence disciplines. This research introduces a transformer-based message-passing neural network (T-MPNN) that is intended to improve the process of embedding molecular representations for property prediction. Our technique incorporates attention processes into MPNNs' message-passing and readout phases, resulting in molecular representations that are seamlessly integrated. The experimental results from three datasets show that T-MPNN outperforms or equals cutting-edge baseline models in tasks involving quantitative structure-property connections. By studying case studies of SARS-COV growth inhibitors, we demonstrate our model's ability to graphically depict attention at the atomic level. This enables us to pinpoint individual chemical atoms or functional groups linked with desirable biological properties. The model we propose improves the interpretability of classic MPNNs and is a useful tool for investigating the impact of self-attention on chemical substructures and functional groups in molecular representation learning. This leads to a better understanding of medication modes of action.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ec7d2dd1e7e05d519e360d3155e47ae5036abea0" target='_blank'>
              Antiviral Medication Prediction Using A Deep Learning Model of Drug-Target Interaction for The Coronavirus SARS-COV
              </a>
            </td>
          <td>
            Renaldy Fredyan
          </td>
          <td>2024-05-31</td>
          <td>Engineering, MAthematics and Computer Science Journal (EMACS)</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Drug-target interaction (DTI) prediction is crucial for identifying new therapeutics and detecting mechanisms of action. While structure-based methods accurately model physical interactions between a drug and its protein target, cell-based assays such as Cell Painting can better capture complex DTI interactions. This paper introduces MOTI$\mathcal{VE}$, a Morphological cOmpound Target Interaction Graph dataset that comprises Cell Painting features for $11,000$ genes and $3,600$ compounds along with their relationships extracted from seven publicly available databases. We provide random, cold-source (new drugs), and cold-target (new genes) data splits to enable rigorous evaluation under realistic use cases. Our benchmark results show that graph neural networks that use Cell Painting features consistently outperform those that learn from graph structure alone, feature-based models, and topological heuristics. MOTI$\mathcal{VE}$ accelerates both graph ML research and drug discovery by promoting the development of more reliable DTI prediction models. MOTI$\mathcal{VE}$ resources are available at https://github.com/carpenter-singh-lab/motive.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/56b24c679485788a7715a3c1a1a3499ab88dd2f0" target='_blank'>
              MOTIVE: A Drug-Target Interaction Graph For Inductive Link Prediction
              </a>
            </td>
          <td>
            John Arevalo, Ellen Su, A. Carpenter, Shantanu Singh
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Accurate identification of protein binding sites is pivotal for understanding molecular interactions and facilitating drug discovery efforts. However, the dynamic nature of proteinligand interactions presents a formidable challenge, necessitating innovative approaches to bridge the gap between theoretical predictions and experimental realities. This review explores the challenges and recent advancements in protein binding site prediction. Specifically, we highlight the integration of molecular dynamics simulations, machine learning, and deep learning techniques to capture the dynamic and complex nature of protein-ligand interactions. Additionally, we discuss the importance of integrating experimental data, such as structural information and biochemical assays, to enhance prediction accuracy and reliability. By navigating the intersection of classical and the onset of machine learning and deep learning approaches, we aim to provide insights into current state-of-the-art techniques and chart a course for future protein binding site prediction advancements. Ultimately, these efforts could unravel the mysteries of protein-ligand interactions and accelerate drug discovery endeavors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0d0a3e8aa8dbf3f452f7812a1199ea004cd73a50" target='_blank'>
              Machine Learning in Enhancing Protein Binding Sites Predictions - What Has Changed Since Then?
              </a>
            </td>
          <td>
            Oluwayimika E Ibitoye, Mahmoud Soliman
          </td>
          <td>2024-06-11</td>
          <td>Combinatorial chemistry & high throughput screening</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mammalian cells are commonly used as hosts in cell culture for biologics production in the pharmaceutical industry. Structured mechanistic models of metabolism have been used to capture complex cellular mechanisms that contribute to varying metabolic shifts in different cell lines. However, little research has focused on the impact of temporal changes in enzyme abundance and activity on the modeling of cell metabolism. In this work, we present a framework for constructing mechanistic models of metabolism that integrate growth-signaling control of enzyme activity and transcript dynamics. The proposed approach is applied to build models for three Chinese hamster ovary (CHO) cell lines using fed-batch culture data and time-series transcript profiles. Leveraging information from the transcriptome data, we develop a parameter estimation approach based on multi-cell-line (MCL) learning, which combines data sets from different cell lines and trains the individual cell-line models jointly to improve model accuracy. The computational results demonstrate the important role of growth signaling and transcript variability in metabolic models as well as the virtue of the MCL approach for constructing cell-line models with a limited amount of data. The resulting models exhibit a high level of accuracy in predicting distinct metabolic behaviors in the different cell lines; these models can potentially be used to accelerate the process and cell-line development for the biomanufacturing of new protein therapeutics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e85b0ee10ff62351e38806eafe8cf7db9748e226" target='_blank'>
              Multi-cell-line learning for the data-driven construction of mechanistic metabolic models.
              </a>
            </td>
          <td>
            Yen-An Lu, Meghan G. McCann, Wei-Shou Hu, Qi Zhang
          </td>
          <td>2024-06-03</td>
          <td>Biotechnology and bioengineering</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hybrid dynamical systems are prevalent in science and engineering to express complex systems with continuous and discrete states. To learn the laws of systems, all previous methods for equation discovery in hybrid systems follow a two-stage paradigm, i.e. they first group time series into small cluster fragments and then discover equations in each fragment separately through methods in non-hybrid systems. Although effective, these methods do not fully take advantage of the commonalities in the shared dynamics of multiple fragments that are driven by the same equations. Besides, the two-stage paradigm breaks the interdependence between categorizing and representing dynamics that jointly form hybrid systems. In this paper, we reformulate the problem and propose an end-to-end learning framework, i.e. Amortized Equation Discovery (AMORE), to jointly categorize modes and discover equations characterizing the dynamics of each mode by all segments of the mode. Experiments on four hybrid and six non-hybrid systems show that our method outperforms previous methods on equation discovery, segmentation, and forecasting.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40610f6f31195a8a68f3802b6cc12d6503f9bae" target='_blank'>
              Amortized Equation Discovery in Hybrid Dynamical Systems
              </a>
            </td>
          <td>
            Yongtuo Liu, Sara Magliacane, Miltiadis Kofinas, E. Gavves
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Amyotrophic Lateral Sclerosis (ALS) is characterized as a rapidly progressive neurodegenerative disease that presents individuals with limited treatment options in the realm of medical interventions and therapies. The disease showcases a diverse range of onset patterns and progression trajectories, emphasizing the critical importance of early detection of functional decline to enable tailored care strategies and timely therapeutic interventions. The present investigation, spearheaded by the iDPP@CLEF 2024 challenge, focuses on utilizing sensor-derived data obtained through an app. This data is used to construct various machine learning models specifically designed to forecast the advancement of the ALS Functional Rating Scale-Revised (ALSFRS-R) score, leveraging the dataset provided by the organizers. In our analysis, multiple predictive models were evaluated to determine their efficacy in handling ALS sensor data. The temporal aspect of the sensor data was compressed and amalgamated using statistical methods, thereby augmenting the interpretability and applicability of the gathered information for predictive modeling objectives. The models that demonstrated optimal performance were a naive baseline and ElasticNet regression. The naive model achieved a Mean Absolute Error (MAE) of 0.20 and a Root Mean Square Error (RMSE) of 0.49, slightly outperforming the ElasticNet model, which recorded an MAE of 0.22 and an RMSE of 0.50. Our comparative analysis suggests that while the naive approach yielded marginally better predictive accuracy, the ElasticNet model provides a robust framework for understanding feature contributions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30fccfafda3f00e1b2a74286e909924acfbf3c09" target='_blank'>
              Machine Learning for ALSFRS-R Score Prediction: Making Sense of the Sensor Data
              </a>
            </td>
          <td>
            Ritesh Mehta, Aleksandar Pramov, Shashank Verma
          </td>
          <td>2024-07-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Systems biology tackles the challenge of understanding the high complexity in the internal regulation of homeostasis in the human body through mathematical modelling. These models can aid in the discovery of disease mechanisms and potential drug targets. However, on one hand the development and validation of knowledge-based mechanistic models is time-consuming and does not scale well with increasing features in medical data. On the other hand, more data-driven approaches such as machine learning models require large volumes of data to produce generalizable models. The integration of neural networks and mechanistic models, forming universal differential equation (UDE) models, enables the automated learning of unknown model terms with less data than the neural network alone. Nevertheless, estimating parameters for these hybrid models remains difficult with sparse data and limited sampling durations that are common in biological applications. In this work, we propose the use of physiology-informed regularization, penalizing biologically implausible model behavior to guide the UDE towards more physiologically plausible regions of the solution space. In a simulation study we show that physiology-informed regularization not only results in a more accurate forecasting of model behaviour, but also supports training with less data. We also applied this technique to learn a representation of the rate of glucose appearance in the glucose minimal model using meal response data measured in healthy people. In that case, the inclusion of regularization reduces variability between UDE-embedded neural networks that were trained from different initial parameter guesses. Author summary Systems biology concerns the modelling and analysis of biological processes, by viewing these as interconnected systems. Modelling is typically done either using mechanistic differential equations that are derived from experiments and known biology, or using machine learning on large biological datasets. While mathematical modelling from biological experiments can provide useful insights with limited data, building and validating these models takes a long time and often requires highly invasive measurements in humans. Efforts to combine this classical technique with machine learning have resulted in a framework termed universal differential equations, where the model equations contain a neural network to describe unknown biological interactions. While these methods have shown success in numerous fields, applications in biology are more challenging due to limited data-availability, high data sparsity. In this work, we have introduced physiology-informed regularization to overcome these instabilities and to constrain the model to biologically plausible behavior. Our results show that by using physiology-informed regularization, we can accurately predict future unseen observations in a simulated example, with much more limited data than a similar model without regularization. Additionally, we show an application of this technique on human data, applying a neural network to learn the appearance of glucose in the blood plasma after a meal.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ddf0dda21d3a8fbe599f98eed4fe943aa342f41" target='_blank'>
              Physiology-informed regularization enables training of universal differential equation systems for biological applications
              </a>
            </td>
          <td>
            Max de Rooij, Balázs Erdős, N. V. van Riel, Shauna D. O’Donovan
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Modeling complex physical dynamics is a fundamental task in science and engineering. Traditional physics-based models are first-principled, explainable, and sample-efficient. However, they often rely on strong modeling assumptions and expensive numerical integration, requiring significant computational resources and domain expertise. While deep learning (DL) provides efficient alternatives for modeling complex dynamics, they require a large amount of labeled training data. Furthermore, its predictions may disobey the governing physical laws and are difficult to interpret. Physics-guided DL aims to integrate first-principled physical knowledge into data-driven methods. It has the best of both worlds and is well equipped to better solve scientific problems. Recently, this field has gained great progress and has drawn considerable interest across discipline Here, we introduce the framework of physics-guided DL with a special emphasis on learning dynamical systems. We describe the learning pipeline and categorize state-of-the-art methods under this framework. We also offer our perspectives on the open challenges and emerging opportunities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60d721e89c2f9c549241a4982b77f9c752b34460" target='_blank'>
              Learning dynamical systems from data: An introduction to physics-guided deep learning
              </a>
            </td>
          <td>
            Rose Yu, Rui Wang
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Multi-omic characterization and integration remains a challenge due to data complexity and sparsity. Addressing this, our study introduces an unsupervised deep learning model, MOVE (Multi-Omic Variational Encoder), specifically designed to integrate and augment the Cancer Dependency Map (DepMap). Harnessing orthogonal multi-omic information, this model successfully generates molecular and phenotypic profiles, resulting in an increase of 32.7% in the number of multi-omic profiles and thereby generating a complete DepMap for 1,523 cancer cell lines. The synthetically enhanced data increases statistical power, uncovering less studied mechanisms associated with drug resistance, and refines the identification of genetic associations and clustering of cancer cell lines. By applying SHAP for model interpretation, MOVE reveals multi-omic features essential for cell clustering and biomarker identification related to drug and gene dependencies. This understanding is crucial for the development of much-needed, effective strategies in prioritizing cancer targets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d5f98740294adae572cbbfc27a5b1f5503d6c2b3" target='_blank'>
              Synthetic augmentation of cancer cell line multi-omic datasets using unsupervised deep learning
              </a>
            </td>
          <td>
            Zhaoxiang Cai, Sofia Apolinário, A. R. Baião, Clare Pacini, Miguel D. Sousa, Susana Vinga, R. Reddel, Phillip J Robinson, Mathew J. Garnett, Qing Zhong, Emanuel Gonçalves
          </td>
          <td>2024-06-29</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Predicting drug-target interaction (DTI) is critical in the drug discovery process. Despite remarkable advances in recent DTI models through the integration of representations from diverse drug and target encoders, such models often struggle to capture the fine-grained interactions between drugs and protein, i.e. the binding of specific drug atoms (or substructures) and key amino acids of proteins, which is crucial for understanding the binding mechanisms and optimising drug design. To address this issue, this paper introduces a novel model, called FusionDTI, which uses a token-level Fusion module to effectively learn fine-grained information for Drug-Target Interaction. In particular, our FusionDTI model uses the SELFIES representation of drugs to mitigate sequence fragment invalidation and incorporates the structure-aware (SA) vocabulary of target proteins to address the limitation of amino acid sequences in structural information, additionally leveraging pre-trained language models extensively trained on large-scale biomedical datasets as encoders to capture the complex information of drugs and targets. Experiments on three well-known benchmark datasets show that our proposed FusionDTI model achieves the best performance in DTI prediction compared with seven existing state-of-the-art baselines. Furthermore, our case study indicates that FusionDTI could highlight the potential binding sites, enhancing the explainability of the DTI prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da80d50b034f767911675f205cca0f7842894ced" target='_blank'>
              FusionDTI: Fine-grained Binding Discovery with Token-level Fusion for Drug-Target Interaction
              </a>
            </td>
          <td>
            Zhaohan Meng, Zaiqiao Meng, I. Ounis
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Motivation Anti-cancer drug response prediction (DRP) using cancer cell lines plays a vital role in stratified medicine and drug discovery. Recently there has been a surge of new deep learning (DL) models for DRP that show promising performance improvements. However, different models use different input data modalities and neural network architectures making it hard to find the source of these improvements. Results We consider three DL DRP models with reported state-of-the-art performance and use genomics or transcriptomics profiles of cell lines and chemical profiles of drugs as input data. We also construct null hypothesis benchmarks that simply leverage patterns in truth values and thus do not use omics or chemical profiles. By experimenting with these DL models and comparing them with our null hypothesis benchmarks we show that for multiple testing types, much or all of the current reported performance is a property of the training truth values. Furthermore, we show that for cancer blind testing (testing on unseen cell lines) no performance comes from drug features, instead, performance is due to the transcriptomics cell line profiles.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10c5705a239471a8f4b914d01d1c42bcefb5e083" target='_blank'>
              Understanding the Sources of Performance in Deep Learning Drug Response Prediction Models
              </a>
            </td>
          <td>
            Nikhil Branson, Pedro R. Cutillas, Conrad Besseant
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The evaluation of drug-induced Torsades de pointes (TdP) risks is crucial in drug safety assessment. In this study, we discuss machine learning approaches in the prediction of drug-induced TdP risks using preclinical data. Specifically, a random forest model was trained on the dataset generated by the rabbit ventricular wedge assay. The model prediction performance was measured on 28 drugs from the Comprehensive In Vitro Proarrhythmia Assay initiative. Leave-one-drug-out cross-validation provided an unbiased estimation of model performance. Stratified bootstrap revealed the uncertainty in the asymptotic model prediction. Our study validated the utility of machine learning approaches in predicting drug-induced TdP risks from preclinical data. Our methods can be extended to other preclinical protocols and serve as a supplementary evaluation in drug safety assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/22aff64c32acf59cf216b66c0fc2cd3375c21ab4" target='_blank'>
              Drug safety assessment by machine learning models.
              </a>
            </td>
          <td>
            Nan Miles Xi, D. Huang
          </td>
          <td>2024-06-18</td>
          <td>Journal of biopharmaceutical statistics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Background/Aim: Despite the advances in oncology and cancer treatment over the past decades, cancer remains one of the deadliest diseases. This study focuses on further understanding the complex nature of cancer by using mathematical tumor modeling to understand, capture as best as possible, and describe its complex dynamics under chemotherapy treatment. Materials and Methods: Focusing on autoregressive with exogenous inputs, i.e., ARX, and adaptive neuro-fuzzy inference system, i.e., ANFIS, models, this work investigates tumor growth dynamics under both single and combination anticancer agent chemotherapy treatments using chemotherapy treatment data on xenografted mice. Results: Four ARX and ANFIS models for tumor growth inhibition were developed, estimated, and evaluated, demonstrating a strong correlation with tumor weight data, with ANFIS models showing superior performance in handling the multi-agent tumor growth complexities. These findings suggest potential clinical applications of the ANFIS models through further testing. Both types of models were also tested for their prediction capabilities across different chemotherapy schedules, with accurate forecasting of tumor growth up to five days in advance. The use of adaptive prediction and sliding (moving) data window techniques allowed for continuous model updating, ensuring more robust predictive capabilities. However, long-term forecasting remains a challenge, with accuracy declining over longer prediction horizons. Conclusion: While ANFIS models showed greater reliability in predictions, the simplicity and rapid deployment of ARX models offer advantages in situations requiring immediate approximations. Future research with larger, more diverse datasets and by exploring varying model complexities is recommended to improve the models’ reliability and applicability in clinical decision-making, thereby aiding the development of personalized chemotherapy regimens.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57e4427fcccf3edcdd11ca0d2dc97f640c7df5d5" target='_blank'>
              A Comparative Analysis of ARX and ANFIS Models for Tumor Growth Prediction Under Single and Multi-agent Chemotherapy
              </a>
            </td>
          <td>
            S. Liliopoulos, G. Stavrakakis, Konstantinos S Dimas
          </td>
          <td>2024-05-31</td>
          <td>AntiCancer Research</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The identification of gene regulatory networks is important for understanding the mechanisms of various biological phenomena. Many methods have been proposed to infer networks from time-series gene expression data obtained by high-throughput next-generation sequencings. Such methods can effectively infer gene regulatory networks for in silico data, but inferring the networks accurately from in vivo data remiains a challenge because of the large noise and low time sampling rate. Here, we proposed a novel unsupervised learning method, Multi-view attention Long-short term memory for Network inference (MaLoN). It can infer gene regulatory networks with temporal changes in gene regulation using the multi-view attention Long Short-term memory model. Using in vivo benchmark datasets in Saccharomyces cerevisiae and Escherichia coli, we showed that MaLoN can infer gene regulatory networks more accurately than existing methods. The ablated models indicated that the multi-view attention mechanism suppressed false positives. The order of activation of gene regulations inferred by MaLoN was consistent with existing knowledge.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08b480f00fdffa622301c36ab4291cce3493b065" target='_blank'>
              Inference of gene regulatory networks for overcoming low performance in real-world data
              </a>
            </td>
          <td>
            Yusuke Hiki, Y. Tokuoka, , Akira Funahashi
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Abstract Summary Phenotype-based drug screening emerges as a powerful approach for identifying compounds that actively interact with cells. Transcriptional and proteomic profiling of cell lines and individual cells provide insights into the cellular state alterations that occur at the molecular level in response to external perturbations, such as drugs or genetic manipulations. In this paper, we propose cycleCDR, a novel deep learning framework to predict cellular response to external perturbations. We leverage the autoencoder to map the unperturbed cellular states to a latent space, in which we postulate the effects of drug perturbations on cellular states follow a linear additive model. Next, we introduce the cycle consistency constraints to ensure that unperturbed cellular state subjected to drug perturbation in the latent space would produces the perturbed cellular state through the decoder. Conversely, removal of perturbations from the perturbed cellular states can restore the unperturbed cellular state. The cycle consistency constraints and linear modeling in the latent space enable to learn transferable representations of external perturbations, so that our model can generalize well to unseen drugs during training stage. We validate our model on four different types of datasets, including bulk transcriptional responses, bulk proteomic responses, and single-cell transcriptional responses to drug/gene perturbations. The experimental results demonstrate that our model consistently outperforms existing state-of-the-art methods, indicating our method is highly versatile and applicable to a wide range of scenarios. Availability and implementation The source code is available at: https://github.com/hliulab/cycleCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1c2b23f3deb29435692d8627c8b4e0b406ec968" target='_blank'>
              Predicting single-cell cellular responses to perturbations using cycle consistency learning
              </a>
            </td>
          <td>
            Wei Huang, Hui Liu
          </td>
          <td>2024-06-28</td>
          <td>Bioinformatics</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Dose-response prediction in cancer is an active application field in machine learning. Using large libraries of \textit{in-vitro} drug sensitivity screens, the goal is to develop accurate predictive models that can be used to guide experimental design or inform treatment decisions. Building on previous work that makes use of permutation invariant multi-output Gaussian Processes in the context of dose-response prediction for drug combinations, we develop a variational approximation to these models. The variational approximation enables a more scalable model that provides uncertainty quantification and naturally handles missing data. Furthermore, we propose using a deep generative model to encode the chemical space in a continuous manner, enabling prediction for new drugs and new combinations. We demonstrate the performance of our model in a simple setting using a high-throughput dataset and show that the model is able to efficiently borrow information across outputs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/54716d9548a4c571e30130186ea85e7d8aec3712" target='_blank'>
              Permutation invariant multi-output Gaussian Processes for drug combination prediction in cancer
              </a>
            </td>
          <td>
            Leiv Rønneberg, V. Lalchand, Paul D. W. Kirk
          </td>
          <td>2024-06-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Targeted monotherapies for cancer often fail due to inherent or acquired drug resistance. By aiming at multiple targets simultaneously, drug combinations can produce synergistic interactions that increase drug effectiveness and reduce resistance. Computational models based on the integration of omics data have been used to identify synergistic combinations, but predicting drug synergy remains a challenge. Here, we introduce DIPx, an algorithm for personalized prediction of drug synergy based on biologically motivated tumor- and drug-specific pathway activation scores (PASs). We trained and validated DIPx in the AstraZeneca-Sanger (AZS) DREAM Challenge dataset using two separate test sets: Test Set 1 comprised the combinations already present in the training set, while Test Set 2 contained combinations absent from the training set, thus indicating the model’s ability to handle novel combinations. The Spearman correlation coefficients between predicted and observed drug synergy were 0.50 (95% CI: 0.47–0.53) in Test Set 1 and 0.26 (95% CI: 0.22–0.30) in Test Set 2, compared to 0.38 (95% CI: 0.34–0.42) and 0.18 (95% CI: 0.16–0.20), respectively, for the best performing method in the Challenge. We show evidence that higher synergy is associated with higher functional interaction between the drug targets, and this functional interaction information is captured by PAS. We illustrate the use of PAS to provide a potential biological explanation in terms of activated pathways that mediate the synergistic effects of combined drugs. In summary, DIPx can be a useful tool for personalized prediction of drug synergy and exploration of activated pathways related to the effects of combined drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/967bc228620e27010cd037753de4c0665790671f" target='_blank'>
              Pathway activation model for personalized prediction of drug synergy
              </a>
            </td>
          <td>
            Quang Thinh Trac, Yue Huang, Tom Erkers, P. Östling, Anna Bohlin, A. Österroos, Mattias Vesterlund, R. Jafari, I. Siavelis, Helena Bäckvall, Santeri Kiviluoto, L. Orre, Mattias Rantalainen, Janne Lehtiö, Sören Lehmann, Olli-P. Kallioniemi, Y. Pawitan, T. Vu
          </td>
          <td>2024-06-07</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>58</td>
        </tr>

        <tr id="Analyzing the motion of multiple biological agents, be it cells or individual animals, is pivotal for the understanding of complex collective behaviors. With the advent of advanced microscopy, detailed images of complex tissue formations involving multiple cell types have become more accessible in recent years. However, deciphering the underlying rules that govern cell movements is far from trivial. Here, we present a novel deep learning framework to estimate the underlying equations of motion from observed trajectories, a pivotal step in decoding such complex dynamics. Our framework integrates graph neural networks with neural differential equations, enabling effective prediction of two-body interactions based on the states of the interacting entities. We demonstrate the efficacy of our approach through two numerical experiments. First, we used a simulated data from a toy model to tune the hyperparameters. Based on the obtained hyperparameters, we then applied this approach to a more complex model that describes interacting cells of cellular slime molds. Our results show that the proposed method can accurately estimate the function of two-body interactions, thereby precisely replicating both individual and collective behaviors within these systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0bbd03a34ea3ecbe6332287f4b5f0d43638d30f4" target='_blank'>
              Integrating GNN and Neural ODEs for Estimating Two-Body Interactions in Mixed-Species Collective Motion
              </a>
            </td>
          <td>
            Masahito Uwamichi, S. Schnyder, Tetsuya J. Kobayashi, Satoshi Sawai
          </td>
          <td>2024-05-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Graph neural networks (GNNs) have gained significant attention in diverse domains, ranging from urban planning to pandemic management. Ensuring both accuracy and robustness in GNNs remains a challenge due to insufficient quality data that contains sufficient features. With sufficient training data where all spatiotemporal patterns are well-represented, existing GNN models can make reasonably accurate predictions. However, existing methods fail when the training data are drawn from different circumstances (e.g., traffic patterns on regular days) than test data (e.g., traffic patterns after a natural disaster). Such challenges are usually classified under domain generalization. In this work, we show that one way to address this challenge in the context of spatiotemporal prediction is by incorporating domain differential equations into graph convolutional networks (GCNs). We theoretically derive conditions where GCNs incorporating such domain differential equations are robust to mismatched training and testing data compared to baseline domain agnostic models. To support our theory, we propose two domain-differential-equation-informed networks: Reaction-Diffusion Graph Convolutional Network (RDGCN), which incorporates differential equations for traffic speed evolution, and the Susceptible-Infectious-Recovered Graph Convolutional Network (SIRGCN), which incorporates a disease propagation model. Both RDGCN and SIRGCN are based on reliable and interpretable domain differential equations that allow the models to generalize to unseen patterns. We experimentally show that RDGCN and SIRGCN are more robust with mismatched testing data than state-of-the-art deep learning methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ce74ab023edbff0d9a6fad7e1a98a69c76287cb" target='_blank'>
              On the generalization discrepancy of spatiotemporal dynamics-informed graph convolutional networks
              </a>
            </td>
          <td>
            Yue Sun, Chao Chen, Yuesheng Xu, Sihong Xie, Rick S. Blum, Parv Venkitasubramaniam
          </td>
          <td>2024-07-12</td>
          <td>Frontiers in Mechanical Engineering</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Inspired by the advancements in pre-trained Large Language Models, there has been a surge of studies in the Life Sciences focusing on constructing foundation models with large scale single-cell RNA-seq data. These studies typically involve pre-training a transformer model on large-scale single-cell sequencing data, followed by fine-tuning for a variety of downstream tasks, achieving notable performance. However, these models all share a common short-coming: to utilize the transformer architecture, originally designed for textual data, they artificially impose a sequential structure on genes within cells, simplifying the complex interactions between genes. Furthermore, they focus solely on transcriptomic data, neglecting other relevant biological information. To address these issues, here we introduce Cell-Graph Compass (CGC), the first foundational model that leverages graph structures to model single cells and describes cells from multiple perspectives, including transcriptional profiles, gene text summaries, transcription factor regulatory networks, gene co-expression patterns, and gene positional relationships. By incorporating self-attention mechanisms, we pretrained the model on 50 million human single-cell sequencing data, resulting in a robust digital representation of cells. Extensive downstream experiments demonstrate that our approach can capture meaningful biological knowledge and achieve superior results in various problem scenarios, achieving the state-of-the-art (SOTA).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50e1d6e564faa2c3f5e89eca6ef5223dc3de6e56" target='_blank'>
              Cell-Graph Compass: Modeling Single Cells with Graph Structure Foundation Model
              </a>
            </td>
          <td>
            Chen Fang, Zhilong Hu, Shaole Chang, Qingqing Long, Wentao Cui, Wenhao Liu, Cong Li, Yana Liu, Pengfei Wang, Zhen Meng, Jia Pan, Yuanchun Zhou, Guihai Feng, Linghui Chen, Xin Li
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Human cancers present a significant public health challenge and require the discovery of novel drugs through translational research. Transcriptomics profiling data that describes molecular activities in tumors and cancer cell lines are widely utilized for predicting anti-cancer drug responses. However, existing AI models face challenges due to noise in transcriptomics data and lack of biological interpretability. To overcome these limitations, we introduce VETE (Variational and Explanatory Transcriptomics Encoder), a novel neural network framework that incorporates a variational component to mitigate noise effects and integrates traceable gene ontology into the neural network architecture for encoding cancer transcriptomics data. Key innovations include a local interpretability-guided method for identifying ontology paths, a visualization tool to elucidate biological mechanisms of drug responses, and the application of centralized large scale hyperparameter optimization. VETE demonstrated robust accuracy in cancer cell line classification and drug response prediction. Additionally, it provided traceable biological explanations for both tasks and offers insights into the mechanisms underlying its predictions. VETE bridges the gap between AI-driven predictions and biologically meaningful insights in cancer research, which represents a promising advancement in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/accd87735bc6a210dee2ce7fbed45af6ac443286" target='_blank'>
              Variational and Explanatory Neural Networks for Encoding Cancer Profiles and Predicting Drug Responses
              </a>
            </td>
          <td>
            Tianshu Feng, Rohan D Gnanaolivu, Abolfazl Safikhani, Yuanhang Liu, Jun Jiang, Nicholas Chia, A. Partin, Priyanka Vasanthakumari, Yitan Zhu, Chen Wang
          </td>
          <td>2024-07-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In the complex landscape of hematologic samples such as peripheral blood or bone marrow derived from flow cytometry (FC) data, cell-level prediction presents profound challenges. This work explores injecting hierarchical prior knowledge into graph neural networks (GNNs) for single-cell multi-class classification of tabular cellular data. By representing the data as graphs and encoding hierarchical relationships between classes, we propose our hierarchical plug-in method to be applied to several GNN models, namely, FCHC-GNN, and effectively designed to capture neighborhood information crucial for single-cell FC domain. Extensive experiments on our cohort of 19 distinct patients, demonstrate that incorporating hierarchical biological constraints boosts performance significantly across multiple metrics compared to baseline GNNs without such priors. The proposed approach highlights the importance of structured inductive biases for gaining improved generalization in complex biological prediction tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/12e0fdaea8a065eda5545038bbdb830b51b0c573" target='_blank'>
              Injecting Hierarchical Biological Priors into Graph Neural Networks for Flow Cytometry Prediction
              </a>
            </td>
          <td>
            Fatemeh Nassajian Mojarrad, Lorenzo Bini, Thomas Matthes, Stéphane Marchand-Maillet
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Identifying how a given chemical of interest exerts its impact on biological systems is a critical step in developing new medicines and chemical products. The mechanism of a query compound of interest can sometimes be identified when its image-based morphological profile matches a compound in a library of well-annotated compound profiles. In this study, we demonstrate a significant improvement in classification performance by incorporating side information: gene representations. We generate these representations using the morphological profiles of cells where the level of a single gene’s expression has been artificially increased or decreased. The genes are selected as those encoding known protein targets of annotated compounds in the library. A transformer model is trained to classify gene-compound pairs, where each pair represents a potential interaction between a gene and a compound, as true or false. Subsequently, the model generates a ranked list of likely target genes for a previously unseen query compound. Although the strategy exhibits high performance only for compounds that target previously encountered genes – likely due to the limited size of our training dataset – the performance increase demonstrates a notable improvement over simply matching compound profiles directly to compound profiles or to gene profiles. Larger datasets may improve the prediction capabilities of this approach, enabling the prediction of gene targets for novel compounds, which can then be experimentally validated.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9924f439a76927ce3b7feb27a61ec80c26600f34" target='_blank'>
              Cell morphological representations of genes enhance prediction of drug targets
              </a>
            </td>
          <td>
            Niveditha S. Iyer, Daniel J. Michael, S-Y Gordon Chi, John Arevalo, S. Chandrasekaran, A. Carpenter, P. Rajpurkar, Shantanu Singh
          </td>
          <td>2024-06-10</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>26</td>
        </tr>

        <tr id="Numerous natural and human-made systems exhibit critical transitions whereby slow changes in environmental conditions spark abrupt shifts to a qualitatively distinct state. These shifts very often entail severe consequences; therefore, it is imperative to devise robust and informative approaches for anticipating the onset of critical transitions. Real-world complex systems can comprise hundreds or thousands of interacting entities, and implementing prevention or management strategies for critical transitions requires knowledge of the exact condition in which they will manifest. However, most research so far has focused on low-dimensional systems and small networks containing fewer than ten nodes or has not provided an estimate of the location where the transition will occur. We address these weaknesses by developing a deep-learning framework which can predict the specific location where critical transitions happen in networked systems with size up to hundreds of nodes. These predictions do not rely on the network topology, the edge weights, or the knowledge of system dynamics. We validate the effectiveness of our machine-learning-based framework by considering a diverse selection of systems representing both smooth (second-order) and explosive (first-order) transitions: the synchronization transition in coupled Kuramoto oscillators; the sharp decline in the resource biomass present in an ecosystem; and the abrupt collapse of a Wilson-Cowan neuronal system. We show that our method provides accurate predictions for the onset of critical transitions well in advance of their occurrences, is robust to noise and transient data, and relies only on observations of a small fraction of nodes. Finally, we demonstrate the applicability of our approach to real-world systems by considering empirical vegetated ecosystems in Africa.




 Published by the American Physical Society
 2024


">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b2d1e225302c12329f67e7dcbe05d29d014a6b7" target='_blank'>
              Early Predictor for the Onset of Critical Transitions in Networked Dynamical Systems
              </a>
            </td>
          <td>
            Zijia Liu, Xiaozhu Zhang, Xiaolei Ru, Tingting Gao, Jack Murdoch Moore, Gang Yan
          </td>
          <td>2024-07-15</td>
          <td>Physical Review X</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Classical compartmental models in epidemiology often struggle to accurately capture real-world dynamics due to their inability to address the inherent heterogeneity of populations. In this paper, we introduce a novel approach that incorporates heterogeneity through a mobility variable, transforming the traditional ODE system into a system of integro-differential equations that describe the dynamics of population densities across different compartments. Our results show that, for the same basic reproduction number, our mobility-based model predicts a smaller final pandemic size compared to classic compartmental models, whose population densities are represented as Dirac delta functions in our density-based framework. This addresses the overestimation issue common in many classical models. Additionally, we demonstrate that the time series of the infected population is sufficient to uniquely identify the mobility distribution. We reconstruct this distribution using a machine-learning-based framework, providing both theoretical and algorithmic support to effectively constrain the mobility-based model with real-world data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c03a14a48a310a736f8149b03d7cfacc5775bf6" target='_blank'>
              Modeling, Inference, and Prediction in Mobility-Based Compartmental Models for Epidemiology
              </a>
            </td>
          <td>
            Ning Jiang, Weiqi Chu, Yao Li
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="Since the introduction of the first immune checkpoint inhibitor (ICI), immunotherapy has changed the landscape of molecular therapeutics for cancers. However, ICIs do not work equally well on all cancers and for all patients. There has been a growing interest in using mathematical and computational models to optimize clinical responses. Ordinary differential equations (ODEs) have been widely used for mechanistic modeling in immuno-oncology and immunotherapy because they allow rapid simulations of temporal changes in the cellular and molecular populations involved. Nonetheless, ODEs cannot describe the spatial structure in the tumor microenvironment or quantify the influence of spatially-dependent characteristics of tumor-immune dynamics. For these reasons, agent-based models (ABMs) have gained popularity because they can model more detailed phenotypic and spatial heterogeneity that better reflect the complexity seen in vivo. In the context of anti-PD-1 ICIs, we compare treatment outcomes simulated from an ODE model and an ABM to show the importance of including spatial components in computational models of cancer immunotherapy. We consider tumor cells of high and low antigenicity and two distinct cytotoxic T lymphocyte (CTL) killing mechanisms. The preferred mechanism differs based on the antigenicity of tumor cells. Our ABM reveals varied phenotypic shifts within the tumor and spatial organization of tumor and CTLs, despite similarities in key immune parameters, initial conditions of simulation, and early temporal trajectories of the cell populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a38a51133539694b984ad15aa283204f18aa1920" target='_blank'>
              Agent-Based Modeling of Virtual Tumors Reveals the Critical Influence of Microenvironmental Complexity on Immunotherapy Efficacy
              </a>
            </td>
          <td>
            Yixuan Wang, Daniel R. Bergman, Erica Trujillo, Anthony A. Fernald, Lie Li, Alexander T. Pearson, Randy F. Sweis, Trachette Jackson
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="In recent years, the escalating environmental challenges have contributed to a rising incidence of cancer. The precise anticipation of cancer incidence and mortality rates has emerged as a pivotal focus in scientific inquiry, exerting a profound impact on the formulation of public health policies. This investigation adopts a pioneering machine learning framework to address this critical issue, utilizing a dataset encompassing 72,591 comprehensive records that include essential variables such as age, case count, population size, race, gender, site, and year of diagnosis. Diverse machine learning algorithms, including decision trees, random forests, logistic regression, support vector machines, and neural networks, were employed in this study. The ensuing analysis revealed testing accuracies of 62.17%, 61.92%, 54.53%, 55.72%, and 62.30% for the respective models. This state-of-the-art model not only enhances our understanding of cancer dynamics but also equips researchers and policymakers with the capability of making meticulous projections concerning forthcoming cancer incidence and mortality rates. Considering sustainability, the application of this advanced machine learning framework emphasizes the importance of judiciously utilizing extensive and intricate databases. By doing so, it facilitates a more sustainable approach to healthcare planning, allowing for informed decision-making that takes into account the long-term ecological and societal impacts of cancer-related policies. This integrative perspective underscores the broader commitment to sustainable practices in both health research and public policy formulation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4560de26fc63a5fa7d9e33f55b5a31acaa915143" target='_blank'>
              Machine Learning Model Construction and Testing: Anticipating Cancer Incidence and Mortality
              </a>
            </td>
          <td>
            Yuanzhao Ding
          </td>
          <td>2024-06-30</td>
          <td>Diseases</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/73fbc8395840f2621fd3a1b982cf278edaa1f8cb" target='_blank'>
              A review of machine learning methods for cancer characterization from microbiome data
              </a>
            </td>
          <td>
            Marco Teixeira, Francisco Silva, R. Ferreira, Tânia Pereira, Ceu Figueiredo, Hélder P. Oliveira
          </td>
          <td>2024-05-30</td>
          <td>NPJ Precision Oncology</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b55d2566d48623ecfc68341b1562423b2d582647" target='_blank'>
              Deep learning-based approach for COVID-19 spread prediction
              </a>
            </td>
          <td>
            S. Cumbane, Gyözö Gidófalvi
          </td>
          <td>2024-06-10</td>
          <td>International Journal of Data Science and Analytics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="A Marked Temporal Point Process (MTPP) is a stochastic process whose realization is a set of event-time data. MTPP is often used to understand complex dynamics of asynchronous temporal events such as money transaction, social media, healthcare, etc. Recent studies have utilized deep neural networks to capture complex temporal dependencies of events and generate embedding that aptly represent the observed events. While most previous studies focus on the inter-event dependencies and their representations, how individual events influence the overall dynamics over time has been under-explored. In this regime, we propose a Decoupled MTPP framework that disentangles characterization of a stochastic process into a set of evolving influences from different events. Our approach employs Neural Ordinary Differential Equations (Neural ODEs) to learn flexible continuous dynamics of these influences while simultaneously addressing multiple inference problems, such as density estimation and survival rate computation. We emphasize the significance of disentangling the influences by comparing our framework with state-of-the-art methods on real-life datasets, and provide analysis on the model behavior for potential applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01528db95c399e0822e827b108710fdf76fd5f21" target='_blank'>
              Decoupled Marked Temporal Point Process using Neural Ordinary Differential Equations
              </a>
            </td>
          <td>
            Yujee Song, Donghyun Lee, Rui Meng, Won Hwa Kim
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mechanistic models of dynamic, interacting cell populations have yielded many insights into the growth and resolution of immune responses. Historically these models have described the behavior of pre-defined cell types based on small numbers of phenotypic markers. The ubiquity of deep phenotyping therefore presents a new challenge; how do we confront tractable and interpretable mathematical models with high-dimensional data? To tackle this problem, we studied the development and persistence of lung-resident memory CD4 and CD8 T cells (TRM) in mice infected with influenza virus. We developed an approach in which dynamical model parameters and the population structure are inferred simultaneously. This method uses deep learning and stochastic variational inference and is trained on the single-cell flow-cytometry data directly, rather than on the kinetics of pre-identified clusters. We show that during the resolution phase of the immune response, memory CD4 and CD8 T cells within the lung are phenotypically diverse, with subsets exhibiting highly distinct and time-dependent dynamics. TRM heterogeneity is maintained long-term by ongoing differentiation of relatively persistent Bcl-2hi CD4 and CD8 TRM subsets which resolve into distinct functional populations. Our approach yields new insights into the dynamics of tissue-localized immune memory, and is a novel basis for interpreting time series of high-dimensional data, broadly applicable to diverse biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/de98a7caad311fa21b6e5ae069bafe928ed0e792" target='_blank'>
              A variational deep-learning approach to modeling memory T cell dynamics
              </a>
            </td>
          <td>
            C. V. van Dorp, Joshua I. Gray, Daniel H. Paik, Donna L. Farber, Andrew J. Yates
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="In this study, we introduce the Framework for Optimized Customizable User-Informed Synthesis (FOCUS), a generative machine learning model tailored for drug discovery. FOCUS integrates domain expertise and uses Proximal Policy Optimization (PPO) to guide Monte Carlo Tree Search (MCTS) to efficiently explore chemical space. It generates SMILES representations of potential drug candidates, optimizing for druggability and binding efficacy to NOD2, PEP, and MCT1 receptors. The model is highly interpretive, allowing for user-feedback and expert-driven adjustments based on detailed cycle reports. Employing tools like SHAP and LIME, FOCUS provides a transparent analysis of decision-making processes, emphasizing features such as docking scores and interaction fingerprints. Comparative studies with Muramyl Dipeptide (MDP) demonstrate improved interaction profiles. FOCUS merges advanced machine learning with expert insight, accelerating the drug discovery pipeline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5a3d974c18b8711815827ec4a3fbb71dde29bc6" target='_blank'>
              FOCUS on NOD2: Advancing IBD Drug Discovery with a User-Informed Machine Learning Framework.
              </a>
            </td>
          <td>
            Ruhi Choudhary, R. Mahadevan
          </td>
          <td>2024-06-06</td>
          <td>ACS medicinal chemistry letters</td>
          <td>0</td>
          <td>47</td>
        </tr>

        <tr id="Mathematical models are indispensable to the system biology toolkit for studying the structure and behavior of intracellular signaling networks. A common approach to modeling is to develop a system of equations that encode the known biology using approximations and simplifying assumptions. As a result, the same signaling pathway can be represented by multiple models, each with its set of underlying assumptions, which opens up challenges for model selection and decreases certainty in model predictions. Here, we use Bayesian multimodel inference to develop a framework to increase certainty in systems biology models. Using models of the extracellular regulated kinase (ERK) pathway, we first show that multimodel inference increases predictive certainty and yields predictors that are robust to changes in the set of available models. We then show that predictions made with multi-model inference are robust to data uncertainties introduced by decreasing the measurement duration and reducing the sample size. Finally, we use multimodel inference to identify a new model to explain experimentally measured sub-cellular location-specific ERK activity dynamics. In summary, our framework highlights multimodel inference as a disciplined approach to increasing the certainty of intracellular signaling activity predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/492e6724b6c05e22b755836f73b25e0ce04843d4" target='_blank'>
              Increasing certainty in systems biology models using Bayesian multimodel inference
              </a>
            </td>
          <td>
            Nathaniel Linden-Santangeli, Jin Zhang, Boris Kramer, P. Rangamani
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Machine-learning (ML) and deep-learning (DL) approaches to predict the molecular properties of small molecules are increasingly deployed within the design-make-test-analyze (DMTA) drug design cycle to predict molecular properties of interest. Despite this uptake, there are only a few automated packages to aid their development and deployment that also support uncertainty estimation, model explainability, and other key aspects of model usage. This represents a key unmet need within the field, and the large number of molecular representations and algorithms (and associated parameters) means it is nontrivial to robustly optimize, evaluate, reproduce, and deploy models. Here, we present QSARtuna, a molecule property prediction modeling pipeline, written in Python and utilizing the Optuna, Scikit-learn, RDKit, and ChemProp packages, which enables the efficient and automated comparison between molecular representations and machine learning models. The platform was developed by considering the increasingly important aspect of model uncertainty quantification and explainability by design. We provide details for our framework and provide illustrative examples to demonstrate the capability of the software when applied to simple molecular property, reaction/reactivity prediction, and DNA encoded library enrichment classification. We hope that the release of QSARtuna will further spur innovation in automatic ML modeling and provide a platform for education of best practices in molecular property modeling. The code for the QSARtuna framework is made freely available via GitHub.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d79165e7710fcacddea5e1ab595a6e3520735bc2" target='_blank'>
              QSARtuna: An Automated QSAR Modeling Platform for Molecular Property Prediction in Drug Design.
              </a>
            </td>
          <td>
            Lewis Mervin, Alexey Voronov, Mikhail Kabeshov, O. Engkvist
          </td>
          <td>2024-07-01</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Multi-factor screenings are commonly used in diverse applications in medicine and bioengineering, including optimizing combination drug treatments and microbiome engineering. Despite the advances in high-throughput technologies, large-scale experiments typically remain prohibitively expensive. Here we introduce a machine learning platform, structure-augmented regression (SAR), that exploits the intrinsic structure of each biological system to learn a high-accuracy model with minimal data requirement. Under different environmental perturbations, each biological system exhibits a unique, structured phenotypic response. This structure can be learned based on limited data and once learned, can constrain subsequent quantitative predictions. We demonstrate that SAR requires significantly fewer data comparing to other existing machine-learning methods to achieve a high prediction accuracy, first on simulated data, then on experimental data of various systems and input dimensions. We then show how a learned structure can guide effective design of new experiments. Our approach has implications for predictive control of biological systems and an integration of machine learning prediction and experimental design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c960a00bc91e7ef861762c26cbaf5fb088878ab0" target='_blank'>
              Data-driven learning of structure augments quantitative prediction of biological responses
              </a>
            </td>
          <td>
            Yuanchi Ha, Helena R. Ma, Feilun Wu, Andrea Weiss, Katherine Duncker, Helen Xu, Jia Lu, Max Golovsky, Daniel Reker, Lingchong You
          </td>
          <td>2024-06-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Predicting drug efficacy and safety in vivo requires information on biological responses (e.g., cell morphology and gene expression) to small molecule perturbations. However, current molecular representation learning methods do not provide a comprehensive view of cell states under these perturbations and struggle to remove noise, hindering model generalization. We introduce the Information Alignment (InfoAlign) approach to learn molecular representations through the information bottleneck method in cells. We integrate molecules and cellular response data as nodes into a context graph, connecting them with weighted edges based on chemical, biological, and computational criteria. For each molecule in a training batch, InfoAlign optimizes the encoder’s latent representation with a minimality objective to discard redundant structural information. A sufficiency objective decodes the representation to align with different feature spaces from the molecule’s neighborhood in the context graph. We demonstrate that the proposed sufficiency objective for alignment is tighter than existing encoder-based contrastive methods. Empirically, we validate representations from InfoAlign in two downstream tasks: molecular property prediction against up to 19 baseline methods across four datasets, plus zero-shot molecule-morphology matching.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6db33b94c598876b05b7af83901a3a69d60eb47d" target='_blank'>
              Learning Molecular Representation in a Cell
              </a>
            </td>
          <td>
            Gang Liu, Srijit Seal, John Arevalo, Zhenwen Liang, A. Carpenter, Meng Jiang, Shantanu Singh
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In pharmaceutical research, the strategy of drug repurposing accelerates the development of new therapies while reducing R&D costs. Network pharmacology lays the theoretical groundwork for identifying new drug indications, and deep graph models have become essential for their precision in mapping complex biological networks. Our study introduces an advanced graph model that utilizes graph convolutional networks and tensor decomposition to effectively predict signed chemical-gene interactions. This model demonstrates superior predictive performance, especially in handling the polar relations in biological networks. Our research opens new avenues for drug discovery and repurposing, especially in understanding the mechanism of actions of drugs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32a7ff5d9100b5d43430996784e6c64556d581f2" target='_blank'>
              A deep graph model for the signed interaction prediction in biological network
              </a>
            </td>
          <td>
            Shuyi Jin, Mengji Zhang, Meijie Wang, Lun Yu
          </td>
          <td>2024-07-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Generative modeling of single-cell RNA-seq data has shown invaluable potential in community-driven tasks such as trajectory inference, batch effect removal and gene expression generation. However, most recent deep models generating synthetic single cells from noise operate on pre-processed continuous gene expression approximations, ignoring the inherently discrete and over-dispersed nature of single-cell data, which limits downstream applications and hinders the incorporation of robust noise models. Moreover, crucial aspects of deep-learning-based synthetic single-cell generation remain underexplored, such as controllable multi-modal and multi-label generation and its role in the performance enhancement of downstream tasks. This work presents Cell Flow for Generation (CFGen), a flow-based conditional generative model for multi-modal single-cell counts, which explicitly accounts for the discrete nature of the data. Our results suggest improved recovery of crucial biological data characteristics while accounting for novel generative tasks such as conditioning on multiple attributes and boosting rare cell type classification via data augmentation. By showcasing CFGen on a diverse set of biological datasets and settings, we provide evidence of its value to the fields of computational biology and deep generative models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fc736fadd20f3db8b214e7b20b42c62b707a2cb5" target='_blank'>
              Generating Multi-Modal and Multi-Attribute Single-Cell Counts with CFGen
              </a>
            </td>
          <td>
            Alessandro Palma, Till Richter, Hanyi Zhang, Manuel Lubetzki, Alexander Tong, Andrea Dittadi, Fabian J Theis
          </td>
          <td>2024-07-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Parameter estimation is one of the central problems in computational modeling of biological systems. Typically, scientists must fully specify the mathematical structure of the model, often expressed as a system of ordinary differential equations, to estimate the parameters. This process poses significant challenges due to the necessity for a detailed understanding of the underlying biological mechanisms. In this paper, we present an approach for estimating model parameters and assessing their identifiability in situations where only partial knowledge of the system structure is available. The partially known model is extended into a system of Hybrid Neural Ordinary Differential Equations, which captures the unknown portions of the system using neural networks. Integrating neural networks into the model structure introduces two primary challenges for parameter estimation: the need to globally explore the search space while employing gradient-based optimization, and the assessment of parameter identifiability, which may be hindered by the expressive nature of neural networks. To overcome the first issue, we treat biological parameters as hyperparameters in the extended model, exploring the parameter search space during hyperparameter tuning. The second issue is then addressed by an a posteriori analysis of parameter identifiability, computed by introducing a variant of a well-established approach for mechanistic models. These two components are integrated into an end-to-end pipeline that is thoroughly described in the paper. We assess the effectiveness of the proposed workflow on test cases derived from three different benchmark models. These test cases have been designed to mimic real-world conditions, including the presence of noise in the training data and various levels of data availability for the system variables. Author summary Parameter estimation is a central challenge in modeling biological systems. Typically, scientists calibrate the parameters by aligning model predictions with measured data once the model structure is defined. Our paper introduces a workflow that leverages the integration between mechanistic modeling and machine learning to estimate model parameters when the model structure is not fully known. We focus mainly on analyzing the identifiability of the model parameters, which measures how confident we can be in the parameter estimates given the available experimental data and partial mechanistic understanding of the system. We assessed the effectiveness of our approach in various in silico scenarios. Our workflow represents a first step to adapting traditional methods used in fully mechanistic models to the scenario of hybrid modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a064c473effb10441bd8fe1c6bde24ed0162e1e3" target='_blank'>
              Robust parameter estimation and identifiability analysis with Hybrid Neural Ordinary Differential Equations in Computational Biology
              </a>
            </td>
          <td>
            Stefano Giampiccolo, Federico Reali, Anna Fochesato, Giovanni Iacca, Luca Marchetti
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="Dynamic Positron Emission Tomography (dPET) imaging and Time-Activity Curve (TAC) analyses are essential for understanding and quantifying the biodistribution of radiopharmaceuticals over time and space. Traditional compartmental modeling, while foundational, commonly struggles to fully capture the complexities of biological systems, including non-linear dynamics and variability. This study introduces an innovative data-driven neural network-based framework, inspired by Reaction Diffusion systems, designed to address these limitations. Our approach, which adaptively fits TACs from dPET, enables the direct calibration of diffusion coefficients and reaction terms from observed data, offering significant improvements in predictive accuracy and robustness over traditional methods, especially in complex biological scenarios. By more accurately modeling the spatio-temporal dynamics of radiopharmaceuticals, our method advances modeling of pharmacokinetic and pharmacodynamic processes, enabling new possibilities in quantitative nuclear medicine.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db7e97f4a77c199435d67bf7917712e6bcfb5bc2" target='_blank'>
              Beyond Conventional Parametric Modeling: Data-Driven Framework for Estimation and Prediction of Time Activity Curves in Dynamic PET Imaging
              </a>
            </td>
          <td>
            N. Zakariaei, Arman Rahmim, Eldad Haber
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Quantitative Systems Pharmacology (QSP) has become a powerful tool in the drug development landscape. To facilitate its continued implementation and to further enhance its applicability, a symbiotic approach in which QSP is combined with artificial intelligence (AI) and machine learning (ML) seems key. This manuscript presents four case examples where the application of a symbiotic approach could unlock new insights from multidimensional data, including real-world data, potentially leading to breakthroughs in drug development. Besides the remarkable benefits (gAIns) that the symbiosis can offer, it does also carry potential challenges (pAIns) such as how to assess and quantify uncertainty, bias and error. Hence, to ensure a successful implementation, arising pAIns need to be acknowledged and carefully addressed. Successful implementation of the symbiotic QSP and ML/AI approach has the potential to serve as a catalyst, paving the way for a paradigm shift in drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/988f6febd6f71531fa4453a2990837684fe2ccd4" target='_blank'>
              Coupling quantitative systems pharmacology modelling to machine learning and artificial intelligence for drug development: its pAIns and gAIns
              </a>
            </td>
          <td>
            Núria Folguera-Blasco, Florencia A. T. Boshier, Aydar Uatay, C. Pichardo-Almarza, Massimo Lai, Jacopo Biasetti, Richard Dearden, Megan Gibbs, Holly Kimko
          </td>
          <td>2024-07-12</td>
          <td>Frontiers in Systems Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="In drug candidate design, clearance is one of the most crucial pharmacokinetic parameters to consider. Recent advancements in machine learning techniques coupled with the growing accumulation of drug data have paved the way for the construction of computational models to predict drug clearance. However, concerns persist regarding the reliability of data collected from public sources, and a majority of current in silico quantitative structure-property relationship models tend to neglect the influence of molecular chirality. In this study, we meticulously examined human liver microsome (HLM) data from public databases and constructed two distinct data sets with varying HLM data quantity and quality. Two baseline models (RF and DNN) and three chirality-focused GNNs (DMPNN, TetraDMPNN, and ChIRo) were proposed, and their performance on HLM data was evaluated and compared with each other. The TetraDMPNN model, which leverages chirality from 2D structure, exhibited the best performance with a test R2 of 0.639 and a test root-mean-squared error of 0.429. The applicability domain of the model was also defined by using a molecular similarity-based method. Our research indicates that graph neural networks capable of capturing molecular chirality have significant potential for practical application and can deliver superior performance.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/756181f0a57c903cdba2e0c27dab55ec47217215" target='_blank'>
              Prediction of Human Liver Microsome Clearance with Chirality-Focused Graph Neural Networks.
              </a>
            </td>
          <td>
            Chengtao Pu, Lingxi Gu, Yuxuan Hu, Weijie Han, Xiaohe Xu, Haichun Liu, Yadong Chen, Yanmin Zhang
          </td>
          <td>2024-07-08</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Transfer learning has revolutionized fields including natural language understanding and computer vision by leveraging large‐scale general datasets to pretrain models with foundational knowledge that can then be transferred to improve predictions in a vast range of downstream tasks. More recently, there has been a growth in the adoption of transfer learning approaches in biological fields, where models have been pretrained on massive amounts of biological data and employed to make predictions in a broad range of biological applications. However, unlike in natural language where humans are best suited to evaluate models given a clear understanding of the ground truth, biology presents the unique challenge of being in a setting where there are a plethora of unknowns while at the same time needing to abide by real‐world physical constraints. This perspective provides a discussion of some key points we should consider as a field in designing benchmarks for foundation models in network biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4df589ed1b5f62af21501b0da808e5b66f31b8d6" target='_blank'>
              Perspectives on benchmarking foundation models for network biology
              </a>
            </td>
          <td>
            Christina V. Theodoris
          </td>
          <td>2024-07-11</td>
          <td>Quantitative Biology</td>
          <td>1</td>
          <td>0</td>
        </tr>

        <tr id="The prediction modeling of drug-target interactions is crucial to drug discovery and design, which has seen rapid advancements owing to deep learning technologies. Recently developed methods, such as those based on graph neural networks (GNNs) and Transformers, demonstrate exceptional performance across various datasets by effectively extracting structural information. However, the benchmarking of these novel methods often varies significantly in terms of hyperparameter settings and datasets, which limits algorithmic progress. In view of these, we conduct a comprehensive survey and benchmark for drug-target interaction modeling from a structure perspective, via integrating tens of explicit (i.e., GNN-based) and implicit (i.e., Transformer-based) structure learning algorithms. To this end, we first unify the hyperparameter setting within each class of structure learning methods. Moreover, we conduct a macroscopical comparison between these two classes of encoding strategies as well as the different featurization techniques that inform molecules' chemical and physical properties. We then carry out the microscopical comparison between all the integrated models across the six datasets, via comprehensively benchmarking their effectiveness and efficiency. Remarkably, the summarized insights from the benchmark studies lead to the design of model combos. We demonstrate that our combos can achieve new state-of-the-art performance on various datasets associated with cost-effective memory and computation. Our code is available at \hyperlink{https://github.com/justinwjl/GTB-DTI/tree/main}{https://github.com/justinwjl/GTB-DTI/tree/main}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ed38d9516d82a27c843ba4400f8f439b5d95bf9" target='_blank'>
              Benchmark on Drug Target Interaction Modeling from a Structure Perspective
              </a>
            </td>
          <td>
            Xinnan Zhang, Jialin Wu, Junyi Xie, Tianlong Chen, Kaixiong Zhou
          </td>
          <td>2024-07-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Identifying the primary site of origin of metastatic cancer is vital for guiding treatment decisions, especially for patients with cancer of unknown primary (CUP). Despite advanced diagnostic techniques, CUP remains difficult to pinpoint and is responsible for a considerable number of cancer-related fatalities. Understanding its origin is crucial for effective management and potentially improving patient outcomes. This study introduces a machine learning framework, ONCOfind-AI, that leverages transcriptome-based gene set features to enhance the accuracy of predicting the origin of metastatic cancers. We demonstrate its potential to facilitate the integration of RNA sequencing and microarray data by using gene set scores for characterization of transcriptome profiles generated from different platforms. Integrating data from different platforms resulted in improved accuracy of machine learning models for predicting cancer origins. We validated our method using external data from clinical samples collected through the Kangbuk Samsung Medical Center and Gene Expression Omnibus. The external validation results demonstrate a top-1 accuracy ranging from 0.80 to 0.86, with a top-2 accuracy of 0.90. This study highlights that incorporating biological knowledge through curated gene sets can help to merge gene expression data from different platforms, thereby enhancing the compatibility needed to develop more effective machine learning prediction models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9fe702a79951b5bf6948f4448427fc41c3af8b9" target='_blank'>
              Application of Transcriptome-Based Gene Set Featurization for Machine Learning Model to Predict the Origin of Metastatic Cancer
              </a>
            </td>
          <td>
            Y. Jeong, Jinah Chu, Juwon Kang, Seungjun Baek, Jae-Hak Lee, Dong-Sub Jung, Won-Woo Kim, Y. Kim, Jihoon Kang, In-Gu Do
          </td>
          <td>2024-07-09</td>
          <td>Current Issues in Molecular Biology</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8d4bb229f62bcab0916b3d2b4498d9f9dcea2bea" target='_blank'>
              Beyond benchmarking and towards predictive models of dataset-specific single-cell RNA-seq pipeline performance
              </a>
            </td>
          <td>
            Cindy Fang, Alina Selega, Kieran R. Campbell
          </td>
          <td>2024-06-17</td>
          <td>Genome Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d1f54608476ba4ebd37c1c1c7ed98f64403bee2f" target='_blank'>
              A comprehensive review of computational cell cycle models in guiding cancer treatment strategies
              </a>
            </td>
          <td>
            Chenhui Ma, Evren Gurkan-Cavusoglu
          </td>
          <td>2024-07-05</td>
          <td>NPJ Systems Biology and Applications</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The live identification of emergent behavior in complex systems with little a-priori information is a challenging task and existing approaches are either applicable to a small subset of models or do not scale well. In contrast, post-mortem approaches that have a more in-depth understanding of the characteristics of emergent properties often struggle with analyzing a large amount of data to extract relationships between the variables, events, and entities whose interaction eventually leads to emergent behavior. Machine learning approaches have been promoted as potential replacements of existing approaches, due to their ability to analyze large amounts of data without a-priori knowledge of existing relationships. In this paper, we present a first step towards the use of supervised learning approaches to identify and predict emergent behavior. Our hybrid approach unifies live and post-mortem perspectives by relying on a visual inspection of the simulation run and the simulation data set to identify a set of features that are more likely to generate emergent behavior (post-mortem) which are then used by a machine learning module to predict emergent behavior (live). Our analysis shows the potential of such approaches but also highlights challenges and future avenues of research.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/50c22c6b6693586120f80e593dda61dfc8e28ccd" target='_blank'>
              Detecting Emergent Behavior in Complex Systems: A Machine Learning Approach
              </a>
            </td>
          <td>
            Simranjeet Singh Dahia, Claudia Szabo
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the 38th ACM SIGSIM Conference on Principles of Advanced Discrete Simulation</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Kolmogorov-Arnold networks (KANs) as an alternative to multi-layer perceptrons (MLPs) are a recent development demonstrating strong potential for data-driven modeling. This work applies KANs as the backbone of a neural ordinary differential equation (ODE) framework, generalizing their use to the time-dependent and temporal grid-sensitive cases often seen in dynamical systems and scientific machine learning applications. The proposed KAN-ODEs retain the flexible dynamical system modeling framework of Neural ODEs while leveraging the many benefits of KANs compared to MLPs, including higher accuracy and faster neural scaling, stronger interpretability and generalizability, and lower parameter counts. First, we quantitatively demonstrated these improvements in a comprehensive study of the classical Lotka-Volterra predator-prey model. We then showcased the KAN-ODE framework's ability to learn symbolic source terms and complete solution profiles in higher-complexity and data-lean scenarios including wave propagation and shock formation, the complex Schr\"odinger equation, and the Allen-Cahn phase separation equation. The successful training of KAN-ODEs, and their improved performance compared to traditional Neural ODEs, implies significant potential in leveraging this novel network architecture in myriad scientific machine learning applications for discovering hidden physics and predicting dynamic evolution.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efeeddb4162e3eefbe1d974d6ce7cd1d32498a6b" target='_blank'>
              KAN-ODEs: Kolmogorov-Arnold Network Ordinary Differential Equations for Learning Dynamical Systems and Hidden Physics
              </a>
            </td>
          <td>
            Benjamin C. Koenig, Suyong Kim, Sili Deng
          </td>
          <td>2024-07-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Modeling the dynamics of flexible objects has become an emerging topic in the community as these objects become more present in many applications, e.g., soft robotics. Due to the properties of flexible materials, the movements of soft objects are often highly nonlinear and, thus, complex to predict. Data-driven approaches seem promising for modeling those complex dynamics but often neglect basic physical principles, which consequently makes them untrustworthy and limits generalization. To address this problem, we propose a physics-constrained learning method that combines powerful learning tools and reliable physical models. Our method leverages the data collected from observations by sending them into a Gaussian process that is physically constrained by a distributed Port-Hamiltonian model. Based on the Bayesian nature of the Gaussian process, we not only learn the dynamics of the system, but also enable uncertainty quantification. Furthermore, the proposed approach preserves the compositional nature of Port-Hamiltonian systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/020e63b8b17cdec54c056d8a1edc98770c8fa7ab" target='_blank'>
              Physics-constrained learning for PDE systems with uncertainty quantified port-Hamiltonian models
              </a>
            </td>
          <td>
            Kaiyuan Tan, Peilun Li, Thomas Beckers
          </td>
          <td>2024-06-17</td>
          <td>ArXiv, DBLP</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In a landscape where scientific discovery is increasingly driven by data, the integration of machine learning (ML) with traditional scientific methodologies has emerged as a transformative approach. This paper introduces a novel, data-driven framework that synergizes physics-based priors with advanced ML techniques to address the computational and practical limitations inherent in first-principle-based methods and brute-force machine learning methods. Our framework showcases four algorithms, each embedding a specific physics-based prior tailored to a particular class of nonlinear systems, including separable and nonseparable Hamiltonian systems, hyperbolic partial differential equations, and incompressible fluid dynamics. The intrinsic incorporation of physical laws preserves the system's intrinsic symmetries and conservation laws, ensuring solutions are physically plausible and computationally efficient. The integration of these priors also enhances the expressive power of neural networks, enabling them to capture complex patterns typical in physical phenomena that conventional methods often miss. As a result, our models outperform existing data-driven techniques in terms of prediction accuracy, robustness, and predictive capability, particularly in recognizing features absent from the training set, despite relying on small datasets, short training periods, and small sample sizes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6ba424643899decf4b48792c380cc0c66abe63c7" target='_blank'>
              Data-Driven Computing Methods for Nonlinear Physics Systems with Geometric Constraints
              </a>
            </td>
          <td>
            Yunjin Tong
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Deep learning models have emerged as rapid, accurate, and effective approaches for clinical decisions. Through a combination of drug screening and deep learning models, drugs that may benefit patients before and after surgery can be discovered to reduce the risk of complications or speed recovery. However, most existing drug prediction methods have high data requirements and lack interpretability, which has a limited role in adjuvant surgical treatment. To address these limitations, we propose the attention-based convolution transpositional interfusion network (ACTIN) for flexible and efficient drug discovery. ACTIN leverages the graph convolution and the transformer mechanism, utilizing drug and transcriptome data to assess the impact of chemical pharmacophores containing certain elements on gene expression. Remarkably, just with only 393 training instances, only one-tenth of the other models, ACTIN achieves state-of-the-art performance, demonstrating its effectiveness even with limited data. By incorporating chemical element embedding disparity and attention mechanism-based parameter analysis, it identifies the possible pharmacophore containing certain elements that could interfere with specific cell lines, which is particularly valuable for screening useful pharmacophores for new drugs tailored to adjuvant surgical treatment. To validate its reliability, we conducted comprehensive examinations by utilizing transcriptome data from the lung tissue of fatal COVID-19 patients as additional input for ACTIN, we generated novel lead chemicals that align with clinical evidence. In summary, ACTIN offers insights into the perturbation biases of elements within pharmacophore on gene expression, which holds the potential for guiding the development of new drugs that benefit surgical treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/646d725ac5d98017c5f5f349fd91039af403bb8f" target='_blank'>
              A versatile attention-based neural network for chemical perturbation analysis and its potential to aid surgical treatment: A experimental study.
              </a>
            </td>
          <td>
            Zheqi Fan, Houming Zhao, Jingcheng Zhou, Dingchang Li, Yunlong Fan, Yiming Bi, Shuaifei Ji
          </td>
          <td>2024-06-13</td>
          <td>International journal of surgery</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="ADME (Absorption, Distribution, Metabolism, Excretion) properties are key parameters to judge whether a drug candidate exhibits a desired pharmacokinetic (PK) profile. In this study, we tested multi-task machine learning (ML) models to predict ADME and animal PK endpoints trained on in-house data generated at Boehringer Ingelheim. Models were evaluated both at the design stage of a compound (i. e., no experimental data of test compounds available) and at testing stage when a particular assay would be conducted (i. e., experimental data of earlier conducted assays may be available). Using realistic time-splits, we found a clear benefit in performance of multi-task graph-based neural network models over single-task model, which was even stronger when experimental data of earlier assays is available. In an attempt to explain the success of multi-task models, we found that especially endpoints with the largest numbers of data points (physicochemical endpoints, clearance in microsomes) are responsible for increased predictivity in more complex ADME and PK endpoints. In summary, our study provides insight into how data for multiple ADME/PK endpoints in a pharmaceutical company can be best leveraged to optimize predictivity of ML models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48ebc7a80c92373f24ac9292ebc2570251dd17d0" target='_blank'>
              Multi-Task ADME/PK prediction at industrial scale: leveraging large and diverse experimentaldatasets.
              </a>
            </td>
          <td>
            Moritz Walter, J. M. Borghardt, Lina Humbeck, Miha Škalič
          </td>
          <td>2024-07-08</td>
          <td>Molecular informatics</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Most approaches for assessing causality in complex dynamical systems fail when the interactions between variables are inherently non-linear and non-stationary. Here we introduce Temporal Autoencoders for Causal Inference (TACI), a methodology that combines a new surrogate data metric for assessing causal interactions with a novel two-headed machine learning architecture to identify and measure the direction and strength of time-varying causal interactions. Through tests on both synthetic and real-world datasets, we demonstrate TACI's ability to accurately quantify dynamic causal interactions across a variety of systems. Our findings display the method's effectiveness compared to existing approaches and also highlight our approach's potential to build a deeper understanding of the mechanisms that underlie time-varying interactions in physical and biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/82d22ad3492ed41cdaeed3d49e84841e11089e47" target='_blank'>
              Inferring the time-varying coupling of dynamical systems with temporal convolutional autoencoders
              </a>
            </td>
          <td>
            Josuan Calderon, Gordon J. Berman
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Purpose: Artificial Intelligence (AI), which mimics the human brain structure and operation, simulates intelligence. The aim of Machine Learning (ML), which is a branch of artificial intelligence, is to create models by analyzing data. Another type of artificial intelligence, Deep Learning (DL), depicts geometric changes using several layers of model representations. Since DL broke the computational analysis record, AI has advanced in many areas. 
Materials and Methods: Contrary to the widespread use of conventional ML methodologies, there is still a need to promote the use and popularity of DL for pharmaceutical research and development. Drug discovery and design have been enhanced by ML and DL in major research projects. To fully realize its potential, drug design must overcome many challenges and issues. Various aspects of medication design must be considered to successfully address these concerns and challenges. This review article explains DL's significance both in technological breakthroughs and in effective medications. 
Results: There are numerous barriers and substantial challenges associated with drug design associated with DL architectures and key application domains. The article discusses several elements of medication development that have been influenced by existing research. Two widely used and efficient Neural Network (NN) designs are discussed in this article: Convolutional Neural Networks (CNNs) and Recurrent Neural Networks (RNNs). 
Conclusion: It is described how these tools can be utilized to design and discover small molecules for drug discovery. They are also given an overview of the history of DL approaches, as well as a discussion of some of their drawbacks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2b0fdca353e21ebc2e1d16a9b8425165c55ad670" target='_blank'>
              Deep Learning in Drug Design—Progress, Methods, and Challenges
              </a>
            </td>
          <td>
            Seyyed Saeid Masoomkhah, Khosro Rezaee, Mojtaba Ansari, Hossein Eslami
          </td>
          <td>2024-06-24</td>
          <td>Frontiers in Biomedical Technologies</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The rapid evolution of single-cell sequencing technologies has facilitated precise transcriptomics profiling at the single-cell level, shedding light on the intricate heterogeneity within cellular populations. Despite these advances, the inherent diversity of cells and data challenges such as noise, batch effects, and sparsity, underscores the pressing need for a unified model to learn and represent cellular states effectively. Single-cell Large Language Models (LLMs) have been crafted to bridge this gap yet exhibit limited performance on human cells. This short-fall may stem from the confounding effects of training data from diverse species, partly because of limited cells for the single species. Here, we have compiled a dataset of approximately 100 million human cells sequenced by multiple technolo-gies from human single-cell datasets with various file types deposited in public databases and websites. Leveraging these extensive data cohorts, we developed CellFM, a robust single-cell foundation model with an impressive 800 million parameters, marking an eight-fold increase over the current largest single-species model. To ensure the training of CellFM on the MindSpore AI framework from Huawei, we have integrated RetNet, a Transformer architecture variant with lin-ear complexity for a balance between efficiency and performance, serving as the backbone of our model. Our comprehensive experiments have shown that CellFM outperforms existing models across diverse applications, such as cell annotation, perturbation prediction, and gene function prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bd45aed312f5393a6aaf2e612c46c292cadd436e" target='_blank'>
              CellFM: a large-scale foundation model pre-trained on transcriptomics of 100 million human cells
              </a>
            </td>
          <td>
            Yuansong Zeng, Jiancong Xie, Zhuoyi Wei, Yun Su, Ningyuan Shangguan, Shuangyu Yang, Chengyang Zhang, Wenbing Li, Jinbo Zhang, Nan Fang, Hongyu Zhang, Huiying Zhao, Yutong Lu, Jue Fan, Weijiang Yu, Yuedong Yang
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Spatiotemporal processes are a fundamental tool for modeling dynamics across various domains, from heat propagation in materials to oceanic and atmospheric flows. However, currently available neural network-based modeling approaches fall short when faced with data collected randomly over time and space, as is often the case with sensor networks in real-world applications like crowdsourced earthquake detection or pollution monitoring. In response, we developed a new spatiotemporal method that effectively handles such randomly sampled data. Our model integrates techniques from amortized variational inference, neural differential equations, neural point processes, and implicit neural representations to predict both the dynamics of the system and the probabilistic locations and timings of future observations. It outperforms existing methods on challenging spatiotemporal datasets by offering substantial improvements in predictive accuracy and computational efficiency, making it a useful tool for modeling and understanding complex dynamical systems observed under realistic, unconstrained conditions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3672c7dfec49ffe06ab53ce52945c37c38e785c8" target='_blank'>
              Modeling Randomly Observed Spatiotemporal Dynamical Systems
              </a>
            </td>
          <td>
            V. Iakovlev, Harri Lähdesmäki
          </td>
          <td>2024-06-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Deep learning approaches have demonstrated remarkable potential in predicting cancer drug responses (CDRs), using cell line and drug features. However, existing methods predominantly rely on single-omics data of cell lines, potentially overlooking the complex biological mechanisms governing cell line responses. This paper introduces DeepFusionCDR, a novel approach employing unsupervised contrastive learning to amalgamate multi-omics features, including mutation, transcriptome, methylome, and copy number variation data, from cell lines. Furthermore, we incorporate molecular SMILES-specific transformers to derive drug features from their chemical structures. The unified multi-omics and drug signatures are combined, and a multi-layer perceptron (MLP) is applied to predict IC50 values for cell line-drug pairs. Moreover, this MLP can discern whether a cell line is resistant or sensitive to a particular drug. We assessed DeepFusionCDR's performance on the GDSC dataset and juxtaposed it against cutting-edge methods, demonstrating its superior performance in regression and classification tasks. We also conducted ablation studies and case analyses to exhibit the effectiveness and versatility of our proposed approach. Our results underscore the potential of DeepFusionCDR to enhance CDR predictions by harnessing the power of multi-omics fusion and molecular-specific transformers. The prediction of DeepFusionCDR on TCGA patient data and case study highlight the practical application scenarios of DeepFusionCDR in real-world environments. Source code and datasets can be available on https://github.com/altriavin/DeepFusionCDR.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/55237e128fd3db51c88289d302cb3a3283ce281e" target='_blank'>
              DeepFusionCDR: Employing Multi-Omics Integration and Molecule-Specific Transformers for Enhanced Prediction of Cancer Drug Responses.
              </a>
            </td>
          <td>
            Xiaowen Hu, Pan Zhang, Jiaxuan Zhang, Lei Deng
          </td>
          <td>2024-06-27</td>
          <td>IEEE journal of biomedical and health informatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Lung adenocarcinoma (LUAD) is a significant planetary health challenge with its high morbidity and mortality rate, not to mention the marked interindividual variability in treatment outcomes and side effects. There is an urgent need for robust systems biomarkers that can help with early cancer diagnosis, prediction of treatment outcomes, and design of precision/personalized medicines for LUAD. The present study aimed at systems biomarkers of LUAD and deployed integrative bioinformatics and machine learning tools to harness gene expression data. Predictive models were developed to stratify patients based on prognostic outcomes. Importantly, we report here several potential key genes, for example, PMEL and BRIP1, and pathways implicated in the progression and prognosis of LUAD that could potentially be targeted for precision/personalized medicine in the future. Our drug repurposing analysis and molecular docking simulations suggested eight drug candidates for LUAD such as heat shock protein 90 inhibitors, cardiac glycosides, an antipsychotic agent (trifluoperazine), and a calcium ionophore (ionomycin). In summary, this study identifies several promising leads on systems biomarkers and drug candidates for LUAD. The findings also attest to the importance of integrative bioinformatics, structural biology and machine learning techniques in biomarker discovery, and precision oncology research and development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/27385f3cdf2b3fd850f3630dcaa0634963dceebe" target='_blank'>
              Lung Adenocarcinoma Systems Biomarker and Drug Candidates Identified by Machine Learning, Gene Expression Data, and Integrative Bioinformatics Pipeline.
              </a>
            </td>
          <td>
            Semra Melis Soyer, Pemra Ozbek, Ceyda Kasavi
          </td>
          <td>2024-07-09</td>
          <td>Omics : a journal of integrative biology</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Today, the field of biomedical engineering spans numerous areas of scientific research that grapple with the challenges of intelligent analysis of small datasets. Analyzing such datasets with existing artificial intelligence tools is a complex task, often complicated by issues like overfitting and other challenges inherent to machine learning methods and artificial neural networks. These challenges impose significant constraints on the practical application of these tools to the problem at hand. While data augmentation can offer some mitigation, existing methods often introduce their own set of limitations, reducing their overall effectiveness in solving the problem. In this paper, the authors present an improved neural network-based technology for predicting outcomes when analyzing small and extremely small datasets. This approach builds on the input doubling method, leveraging response surface linearization principles to improve performance. Detailed flowcharts of the improved technology’s operations are provided, alongside descriptions of new preparation and application algorithms for the proposed solution. The modeling, conducted using two biomedical datasets with optimal parameters selected via differential evolution, demonstrated high prediction accuracy. A comparison with several existing methods revealed a significant reduction in various errors, underscoring the advantages of the improved neural network technology, which does not require training, for the analysis of extremely small biomedical datasets.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7b8f4413319cc1d449f1821e560daf53d471c3fe" target='_blank'>
              Improvement of the ANN-Based Prediction Technology for Extremely Small Biomedical Data Analysis
              </a>
            </td>
          <td>
            I. Izonin, R.O. Tkachenko, O. Berezsky, Iurii Krak, Michal Kováč, M. Fedorchuk
          </td>
          <td>2024-07-12</td>
          <td>Technologies</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="e13603 Background: Oncology therapeutic development has high failure rates with substantial costs. Advances in technology, including molecular characterisations of cancer and computational power, provide the opportunity to better model therapeutic response for existing and emerging therapies. Methods: Concr adapted Bayesian statistical principles used by astrophysicists to effectively integrate multi-dimensional data (genomic & transcriptomic) for accurate modelling of cancer biology. Our proprietary computational framework was applied to predict clinical response and survival through construction of digital twins and in silico simulations of clinical trials. Predicted log odds ratios (LOR) were generated for drug response and compared to published data from 8 suitable historical clinical trials. Three distinct models underpin this framework: Drug Efficacy, Treatment Response & Overall Survival (OS). Three publicly available datasets were used for training: Cancer Therapeutic Response Portal ( in vitro dose-response data), Cancer Cell Line Encyclopaedia and TCGA. Results: For all 8 clinical studies, the digital twin model accurately simulated both trial arms, compared drug efficacy across arms and predicted which treatment was most active. Unblinded evaluation: Our model accurately predicted gemcitabine would have greater clinical benefit than 5-FU (LOR -0.10, P <0.0001) in metastatic pancreatic cancer. In advanced breast cancer, the model predicted docetaxel had a higher response rate than doxorubicin, and carboplatin response rates were similar to docetaxel in BRCA wild-type patients. In platinum-sensitive recurrent ovarian cancer, the model predicted cisplatin-based combination had a higher response rate than paclitaxel. Blinded evaluation: Using data for paclitaxel, the model correctly predicted that nab-paclitaxel+gemcitabine response rates were higher than gemcitabine in metastatic pancreatic cancer (predicted LOR -0.090, p = <0.001). In all 3 early breast cancer studies, predictions mirrored clinical trial results, adjuvant ECF > CMF, adjuvant TC > AC and adjuvant tamoxifen was superior to capecitabine. The underlying pan-tumour Overall Survival model used a Random Survival Forest approach, trained on 23 cancers with predictive accuracy of 0.78 AUC ROC and C-index 0.71. To enable cohort enrichment for drug response, the model segmented patient cohorts into treatment responder and non-responder. Using this approach, the log odds increased for drug response rate for 16 of the 19 different cancers and 14 of the different 17 drugs. Conclusions: Our digital twin model represents transformative technology for clinical trial simulation with strong potential for accelerating and de-risking clinical drug development, and utility for synthetic control arms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/78a659d949c4ba645ce412729470f4959a538035" target='_blank'>
              Prediction of therapeutic response and cancer outcomes in solid tumours via in silico clinical trials.
              </a>
            </td>
          <td>
            Uzma Asghar, M. Griffiths, Amanzhol Kubeyev, Jordan Laurie, Andrea Giorni, Luiz A Zillmann da Silva, Prabu Sivasubramaniam, Matthew Foster, Andrew Biankin
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Understanding the pharmacokinetics, safety and efficacy of candidate drugs is crucial for their success. One key aspect is the characterization of absorption, distribution, metabolism, excretion and toxicity (ADMET) properties, which require early assessment in the drug discovery and development process. This study aims to present an innovative approach for predicting ADMET properties using attention-based graph neural networks (GNNs). The model utilizes a graph-based representation of molecules directly derived from Simplified Molecular Input Line Entry System (SMILE) notation. Information is processed sequentially, from substructures to the whole molecule, employing a bottom-up approach. The developed GNN is tested and compared with existing approaches using six benchmark datasets and by encompassing regression (lipophilicity and aqueous solubility) and classification (CYP2C9, CYP2C19, CYP2D6 and CYP3A4 inhibition) tasks. Results show the effectiveness of our model, which bypasses the computationally expensive retrieval and selection of molecular descriptors. This approach provides a valuable tool for high-throughput screening, facilitating early assessment of ADMET properties and enhancing the likelihood of drug success in the development pipeline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/01453cd5518b0593e0b01cf8fcaabf43951b2ae4" target='_blank'>
              Predicting ADMET Properties from Molecule SMILE: A Bottom-Up Approach Using Attention-Based Graph Neural Networks
              </a>
            </td>
          <td>
            Alessandro De Carlo, Davide Ronchi, Marco Piastra, E. Tosca, P. Magni
          </td>
          <td>2024-06-01</td>
          <td>Pharmaceutics</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="The search for prognostic biomarkers capable of predicting patient outcomes, by analyzing gene expression in tissue samples and other molecular profiles, remains largely on single-gene-based or global-gene-search approaches. Gene-centric approaches, while foundational, fail to capture the higher-order dependencies that reflect the activities of co-regulated processes, pathway alterations, and regulatory networks, all of which are crucial in determining the patient outcomes in complex diseases like cancer. Here, we introduce GPS-Net, a computational framework that fills the gap in efficiently identifying prognostic modules by incorporating the holistic pathway structures and the network of gene interactions. By innovatively incorporating advanced multiple kernel learning techniques and network-based regularization, the proposed method not only enhances the accuracy of biomarker and pathway identification but also significantly reduces computational complexity, as demonstrated by extensive simulation studies. Applying GPS-Net, we identified key pathways that are predictive of patient outcomes in a cancer immunotherapy study. Overall, our approach provides a novel framework that renders genome-wide pathway-level prognostic analysis both feasible and scalable, synergizing both mechanism-driven and data-driven for precision genomics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f73acfd1579a45da99bac9453e7272a9df579aca" target='_blank'>
              GPS-Net: discovering prognostic pathway modules based on network regularized kernel learning
              </a>
            </td>
          <td>
            Sijie Yao, Kaiqiao Li, Tingyi Li, Xiaoqing Yu, P. Kuan, Xuefeng Wang
          </td>
          <td>2024-07-18</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Predicting protein function from protein sequence, structure, interaction, and other relevant information is important for generating hypotheses for biological experiments and studying biological systems, and therefore has been a major challenge in protein bioinformatics. Numerous computational methods had been developed to advance protein function prediction gradually in the last two decades. Particularly, in the recent years, leveraging the revolutionary advances in artificial intelligence (AI), more and more deep learning methods have been developed to improve protein function prediction at a faster pace. Here, we provide an in-depth review of the recent developments of deep learning methods for protein function prediction. We summarize the significant advances in the field, identify several remaining major challenges to be tackled, and suggest some potential directions to explore. The data sources and evaluation metrics widely used in protein function prediction are also discussed to assist the machine learning, AI, and bioinformatics communities to develop more cutting-edge methods to advance protein function prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d58b5b064d13eaffd68fec855854732decafd97b" target='_blank'>
              Deep learning methods for protein function prediction.
              </a>
            </td>
          <td>
            Frimpong Boadu, Ahhyun Lee, Jianlin Cheng
          </td>
          <td>2024-07-12</td>
          <td>Proteomics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Paired single-cell sequencing technologies enable the simultaneous measurement of complementary modalities of molecular data at single-cell resolution. Along with the advances in these technologies, many methods based on variational autoencoders have been developed to integrate these data. However, these methods do not explicitly incorporate prior biological relationships between the data modalities, which could significantly enhance modeling and interpretation. We propose a novel probabilistic learning framework that explicitly incorporates conditional independence relationships between multi-modal data as a directed acyclic graph using a generalized hierarchical variational autoencoder. We demonstrate the versatility of our framework across various applications pertinent to single-cell multi-omics data integration. These include the isolation of common and distinct information from different modalities, modality-specific differential analysis, and integrated cell clustering. We anticipate that the proposed framework can facilitate the construction of highly flexible graphical models that can capture the complexities of biological hypotheses and unravel the connections between different biological data types, such as different modalities of paired single-cell multi-omics data. The implementation of the proposed framework can be found in the repository https://github.com/kuijjerlab/CAVACHON.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1d5edcd7da413e72fd7c1ca0c333150327939b18" target='_blank'>
              CAVACHON: a hierarchical variational autoencoder to integrate multi-modal single-cell data
              </a>
            </td>
          <td>
            Ping-Han Hsieh, Ru-Xiu Hsiao, Katalin Ferenc, Anthony Mathelier, R. Burkholz, Chien-Yu Chen, G. K. Sandve, T. Belova, M. Kuijjer
          </td>
          <td>2024-05-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/84a5d0b1c26573809df096903d64fc1da14e7531" target='_blank'>
              Advancing cancer drug development with mechanistic mathematical modeling: bridging the gap between theory and practice.
              </a>
            </td>
          <td>
            Alexander Kulesza, C. Couty, Paul Lemarre, Craig J Thalhauser, Yanguang Cao
          </td>
          <td>2024-06-21</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8771c0911641ab60255a369cee70970d99ac457" target='_blank'>
              A variational expectation-maximization framework for balanced multi-scale learning of protein and drug interactions.
              </a>
            </td>
          <td>
            Jiahua Rao, Jiancong Xie, Qianmu Yuan, Deqin Liu, Zhen Wang, Yutong Lu, Shuangjia Zheng, Yuedong Yang
          </td>
          <td>2024-05-25</td>
          <td>Nature communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="After the coronavirus disease 2019 (COVID-19) outbreak erupted, it swiftly spread globally and triggered a severe public health crisis in 2019. To contain the virus’s spread, several countries implemented various lockdown measures. As the governments faced this unprecedented challenge, understanding the impact of lockdown policies became paramount. The goal of addressing the pandemic crisis is to devise prudent policies that strike a balance between safeguarding lives and maintaining economic stability. Traditional mathematical and statistical models for studying virus transmission only offer macro-level predictions of epidemic development and often overlook individual variations’ impact, therefore failing to reflect the role of government decisions. To address this challenge, we propose an integrated framework that combines agent-based modeling (ABM) and deep Q-network (DQN) techniques. This framework enables a more comprehensive analysis and optimization of epidemic control strategies while considering real human behavior. We construct a pandemic simulator based on the ABM method, accurately simulating agents’ daily activities, interactions, and the dynamic spread of the virus. Additionally, we employ a data-driven approach and adjust the model through real statistical data to enhance its effectiveness. Subsequently, we integrated ABM into a decision-making framework using reinforcement learning techniques to explore the most effective strategies. In experiments, we validated the model’s effectiveness by simulating virus transmission across different countries globally. In this model, we obtained decision outcomes when governments focused on various factors. Our research findings indicate that our model serves as a valuable tool for decision-makers, enabling them to formulate prudent and rational policies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1ce211bae4071f08f917efe2eb6cd613c89e2263" target='_blank'>
              A Data-Driven Pandemic Simulator with Reinforcement Learning
              </a>
            </td>
          <td>
            Yuting Zhang, Biyang Ma, Langcai Cao, Yanyu Liu
          </td>
          <td>2024-06-27</td>
          <td>Electronics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="One of the core concepts in science, and something that happens intuitively in every-day dynamic systems modeling, is the combination of models or methods. Especially in dynamical systems modeling, often two or more structures are combined to obtain a more powerful or efficient architecture regarding a specific application (area). Further, even physical simulations are combined with machine learning architectures, to increase prediction accuracy or optimize the computational performance. In this work, we shortly discuss, which types of models are usually combined and propose a model interface that is capable of expressing a width variety of mixed algebraic, discrete and differential equation based models. Further, we examine different established, as well as new ways of combining these models from a system theoretical point of view and highlight two challenges - algebraic loops and local event affect functions in discontinuous models - that require a special approach. Finally, we propose a new wildcard topology, that is capable of describing the generic connection between two combined models in an easy to interpret fashion that can be learned as part of a gradient based optimization procedure. The contributions of this paper are highlighted at a proof of concept: Different connection topologies between two models are learned, interpreted and compared applying the proposed methodology and software implementation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe81ab9d1a688154b1e40c6ca68ccf95c122d0c7" target='_blank'>
              Learnable & Interpretable Model Combination in Dynamic Systems Modeling
              </a>
            </td>
          <td>
            Tobias Thummerer, Lars Mikelsons
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="An active area of MGI (Materials Genome Initiative)/MGE (Materials Genome Engineering) is to accelerate the development of new materials by means of active learning and “digital trial‐error” using a prediction model of material property. Machine learning methods have widely been employed for predicting crystalline materials properties with crystal graph neural networks (CGNN). The prediction accuracy of the state‐of‐the‐art (SOTA) CGNN models based on big models and big data is generally higher. However, for the development of some classes of materials, the datasets obtained by experiments are usually lacking due to costly experiments and measurement costs. The lack of datasets will impact the accuracy of CGNN models and may result in overfitting during training models. This paper proposes a simplified crystal graph convolutional neural network (S‐CGCNN) which possesses higher prediction accuracy while reducing the vast amount of train datasets and computation costs. The S‐CGCNN model has successfully predicted properties of crystalline materials, such as piezoelectric materials and dielectric materials, and increased the prediction accuracy up to 12%–20% than existing SOTA CGNN models. Furthermore, the distribution map between properties and compositions of materials has been built to screen the latent space of candidate materials efficiently by principal component analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc52450c274e7725d3d1d8634944dcd802abc243" target='_blank'>
              A machine learning‐based crystal graph network and its application in development of functional materials
              </a>
            </td>
          <td>
            Gang Xu, You Xue, Xiaoxiao Geng, Xinmei Hou, Jinwu Xu
          </td>
          <td>2024-06-04</td>
          <td>Materials Genome Engineering Advances</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="OBJECTIVE
The aim of this project was to create time-aware, individual-level risk score models for adverse drug events related to multiple sclerosis disease-modifying therapy and to provide interpretable explanations for model prediction behavior.


MATERIALS AND METHODS
We used temporal sequences of observational medical outcomes partnership common data model (OMOP CDM) concepts derived from an electronic health record as model features. Each concept was assigned an embedding representation that was learned from a graph convolution network trained on a knowledge graph (KG) of OMOP concept relationships. Concept embeddings were fed into long short-term memory networks for 1-year adverse event prediction following drug exposure. Finally, we implemented a novel extension of the local interpretable model agnostic explanation (LIME) method, knowledge graph LIME (KG-LIME) to leverage the KG and explain individual predictions of each model.


RESULTS
For a set of 4859 patients, we found that our model was effective at predicting 32 out of 56 adverse event types (P < .05) when compared to demographics and past diagnosis as variables. We also assessed discrimination in the form of area under the curve (AUC = 0.77 ± 0.15) and area under the precision-recall curve (AUC-PR = 0.31 ± 0.27) and assessed calibration in the form of Brier score (BS = 0.04 ± 0.04). Additionally, KG-LIME generated interpretable literature-validated lists of relevant medical concepts used for prediction.


DISCUSSION AND CONCLUSION
Many of our risk models demonstrated high calibration and discrimination for adverse event prediction. Furthermore, our novel KG-LIME method was able to utilize the knowledge graph to highlight concepts that were important to prediction. Future work will be required to further explore the temporal window of adverse event occurrence beyond the generic 1-year window used here, particularly for short-term inpatient adverse events and long-term severe adverse events.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1966744286d710728ee88bd61e935e315ec74e5" target='_blank'>
              KG-LIME: predicting individualized risk of adverse drug events for multiple sclerosis disease-modifying therapy.
              </a>
            </td>
          <td>
            Jason Patterson, Nicholas Tatonetti
          </td>
          <td>2024-07-04</td>
          <td>Journal of the American Medical Informatics Association : JAMIA</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Graph structures offer a versatile framework for representing diverse patterns in nature and complex systems, applicable across domains like molecular chemistry, social networks, and transportation systems. While diffusion models have excelled in generating various objects, generating graphs remains challenging. This thesis explores the potential of score-based generative models in generating such objects through a modelization as combinatorial complexes, which are powerful topological structures that encompass higher-order relationships. In this thesis, we propose a unified framework by employing stochastic differential equations. We not only generalize the generation of complex objects such as graphs and hypergraphs, but we also unify existing generative modelling approaches such as Score Matching with Langevin dynamics and Denoising Diffusion Probabilistic Models. This innovation overcomes limitations in existing frameworks that focus solely on graph generation, opening up new possibilities in generative AI. The experiment results showed that our framework could generate these complex objects, and could also compete against state-of-the-art approaches for mere graph and molecule generation tasks.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b2cf98fd14928741718024c4281fad73ff5c04e4" target='_blank'>
              Combinatorial Complex Score-based Diffusion Modelling through Stochastic Differential Equations
              </a>
            </td>
          <td>
            Adrien Carrel
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Deep learning (DL) and classical machine learning (ML) models are compared and contrasted in this study, which offers a complete overview of the differences and technological improvements between the two types of models. Through an analysis of a diverse range of research publications, the study draws attention to the distinct advantages and uses of both techniques. Deep learning, which is characterized by its use of neural networks with several layers, is particularly effective at managing massive datasets that are not organized. It has also made great progress in the areas of image and audio recognition, natural language processing, and complicated pattern identification exercises. On the other hand, classic machine learning models, which are based on the extraction of features and simpler methods, continue to be quite successful in structured data situations such as classification, regression, and clustering challenges. By concentrating on aspects such as data quantity, computing resources, and unique application needs, the survey sheds light on the parameters that should be considered when selecting between deep learning and machine learning. In addition to this, it addresses the ever-changing environment of hybrid models, which combine methods from both deep learning and machine learning in order to capitalize on the advantages of both approaches. This study highlights the significance of contextual awareness in the fast-developing area of artificial intelligence by providing researchers and practitioners with useful insights that can be used to deploy the AI models that are the most appropriate for their particular requirements.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb0715ffbf9f174de0794365b6ebb28e44718f8" target='_blank'>
              Systematic Analysis of Deep Learning Models vs. Machine Learning
              </a>
            </td>
          <td>
            Dr. Sheshang Degadwala, Dhairya Vyas
          </td>
          <td>2024-07-09</td>
          <td>International Journal of Scientific Research in Computer Science, Engineering and Information Technology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="EpiLearn is a Python toolkit developed for modeling, simulating, and analyzing epidemic data. Although there exist several packages that also deal with epidemic modeling, they are often restricted to mechanistic models or traditional statistical tools. As machine learning continues to shape the world, the gap between these packages and the latest models has become larger. To bridge the gap and inspire innovative research in epidemic modeling, EpiLearn not only provides support for evaluating epidemic models based on machine learning, but also incorporates comprehensive tools for analyzing epidemic data, such as simulation, visualization, transformations, etc. For the convenience of both epidemiologists and data scientists, we provide a unified framework for training and evaluation of epidemic models on two tasks: Forecasting and Source Detection. To facilitate the development of new models, EpiLearn follows a modular design, making it flexible and easy to use. In addition, an interactive web application is also developed to visualize the real-world or simulated epidemic data. Our package is available at https://github.com/Emory-Melody/EpiLearn.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1e0a4bf71b4cf7906c2ed42050b45a3e4b7e895" target='_blank'>
              EpiLearn: A Python Library for Machine Learning in Epidemic Modeling
              </a>
            </td>
          <td>
            Zewen Liu, Yunxiao Li, Mingyang Wei, Guancheng Wan, Max S. Y. Lau, Wei Jin
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="During developmental processes such as embryogenesis, how a group of cells fold into specific structures, is a central question in biology that defines how living organisms form. Establishing tissue-level morphology critically relies on how every single cell decides to position itself relative to its neighboring cells. Despite its importance, it remains a major challenge to understand and predict the behavior of every cell within the living tissue over time during such intricate processes. To tackle this question, we propose a geometric deep learning model that can predict multicellular folding and embryogenesis, accurately capturing the highly convoluted spatial interactions among cells. We demonstrate that multicellular data can be represented with both granular and foam-like physical pictures through a unified graph data structure, considering both cellular interactions and cell junction networks. We successfully use our model to achieve two important tasks, interpretable 4-D morphological sequence alignment, and predicting local cell rearrangements before they occur at single-cell resolution. Furthermore, using an activation map and ablation studies, we demonstrate that cell geometries and cell junction networks together regulate local cell rearrangement which is critical for embryo morphogenesis. This approach provides a novel paradigm to study morphogenesis, highlighting a unified data structure and harnessing the power of geometric deep learning to accurately model the mechanisms and behaviors of cells during development. It offers a pathway toward creating a unified dynamic morphological atlas for a variety of developmental processes such as embryogenesis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce21f827f84dd8500ab0740daa37b4f870f6a662" target='_blank'>
              Multicell-Fold: geometric learning in folding multicellular life
              </a>
            </td>
          <td>
            Haiqian Yang, Anh Q. Nguyen, Dapeng Bi, Markus J. Buehler, Ming Guo
          </td>
          <td>2024-07-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>6</td>
        </tr>

        <tr id="ABSTRACT: The Integration of Machine Learning (ML) techniques holds significant promise in addressing challenges across various sectors, particularly within healthcare and biomedical fields. In this study, we focus on leveraging ML methodologies to address the longstanding issues surrounding the prediction and treatment of Parkinson's Disease (PD). PD prediction has historically suffered from inaccuracies and inconsistent treatments. Our research aims to mitigate these challenges by developing a predictive system tailored specifically to PD datasets. To achieve this, we systematically explore various ML algorithms for binary classification tasks, comparing their efficacy in predicting PD. By analyzing and comparing the performance of these algorithms, we aim to establish a robust pathway for accurately examining and diagnosing PD, thereby reducing discrepancies and associated risks. Our findings underscore the importance of employing ML techniques in developing effective decision support systems for PD prediction. By synthesizing results from multiple algorithms, our study not only contributes to filling existing research gaps but also provides actionable insights for the development of advanced medical applications. Overall, this research offers a comprehensive evaluation of ML approaches in the context of PD prediction, highlighting their potential to revolutionize diagnostic processes and improve patient outcomes. Our work not only enhances our understanding of PD but also underscores the transformative impact of ML in addressing complex medical challenges.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/05f6ba64e35cc29c27ab0f002522fa4646818ccc" target='_blank'>
              Exploring Machine Learning Methods for Developing a Predictive System for Parkinson's Disease
              </a>
            </td>
          <td>
            Sumit Das, Tanusree Saha, Ira Nath, Dipansu Mondal
          </td>
          <td>2024-07-01</td>
          <td>Biosciences Biotechnology Research Asia</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="As precision medicine such as targeted therapy and immunotherapy often have limited accessibility, low response rate, and evolved resistance, it is urgent to develop simple, low-cost, and quick-turnaround personalized diagnostic technologies for drug response prediction with high sensitivity, speed, and accuracy. The major challenges of drug response prediction strategies employing digital database modeling are the scarcity of labeled clinical data, applicability only to a few classes of drugs, and losing the resolution at the individual patient level. Although these challenges have been partially addressed by large-scale cancer cell line datasets and more patient-relevant cell-based systems, the integration of different data types and data translation from pre-clinical to clinical utilities are still far-fetched. To overcome the current limitations of precision medicine with a clinically proven drug response prediction assay, we have developed an innovative and proprietary technology based on in vitro patient testing and in silico data analytics. First, a patient-derived gene expression signature was established via the transcriptomic profiling of cell-free mRNA (cfmRNA) from the patient’s blood. Second, a gene-to-drug data fusion and overlaying mechanism to transfer data were performed. Finally, a semi-supervised method was used for the database searching, matching, annotation, and ranking of drug efficacies from a pool of ~700 approved, investigational, or clinical trial drug candidates. A personalized drug response report can be delivered to inform clinical decisions within a week. The PGA (patient-derived gene expression-informed anticancer drug efficacy) test has significantly improved patient outcomes when compared to the treatment plans without PGA support. The implementation of PGA, which combines patient-unique cfmRNA fingerprints with drug mapping power, has the potential to identify treatment options when patients are no longer responding to therapy and when standard-of-care is exhausted.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6a4d56cd8aea96d0d0aeb07e90e09da931c495cc" target='_blank'>
              A Transformative Technology Linking Patient’s mRNA Expression Profile to Anticancer Drug Efficacy
              </a>
            </td>
          <td>
            Chen Yeh, Shu-Ti Lin, Hung-Chih Lai
          </td>
          <td>2024-07-14</td>
          <td>Onco</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract The use of deep learning methods for fluvial flood forecasting is rapidly gaining traction, offering a promising solution to the challenges associated with accurate yet time-consuming numerical models. This paper presents two physics-inspired deep learning approaches specifically designed for fluvial flood forecasting, each embracing different learning principles: centralized and federated learning. The centralized model utilizes an Encoder-Decoder technique to handle input data of varying types and scales, while the federated model is based on a node-link graph with a seq2seq internal model. Both models are enhanced with a probabilistic forecasting head to account for the inherent uncertainty in streamflow forecasts. The objective of these approaches is to address the limitations of traditional numerical models while leveraging the potential of deep learning to improve the speed, accuracy, and scalability of flood forecasting. To validate their effectiveness, the models were tested across different use cases. The findings from the federated learning approach emphasize the importance of catchment clustering before model utilization and demonstrate the models’ ability to generalize effectively in catchments with similar properties. On the other hand, the results of the centralized method highlight the model’s reliance on the test set falling within the data range of the training set (Average NSE and KGE for the sixth hour ahead of 0.88 and 0.78, respectively). To address this limitation, the paper suggests the development of a method for the future, such as leveraging a numerical model or using Generative Adversarial Networks, to generate highly extreme events, particularly in the context of a changing climate. The models are implemented in a flexible operational framework based on open standards, ensuring their adaptability and usability in various settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5608b6c47d77fd9de30f710f51dadaa5013bc0c0" target='_blank'>
              Probabilistic multi-step ahead streamflow forecast based on deep learning
              </a>
            </td>
          <td>
            D. Karimanzira, Lucas Richter, Desiree Hilbring, Michaela Lödige, Jonathan Vogl
          </td>
          <td>2024-06-01</td>
          <td>at - Automatisierungstechnik</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Model-informed approaches provide a quantitative framework to integrate all available nonclinical and clinical data, thus furnishing a totality of evidence approach to drug development and regulatory evaluation. Maximizing the use of all available data and information about the drug enables a more robust characterization of the risk-benefit profile and reduces uncertainty in both technical and regulatory success. This offers the potential to transform rare diseases drug development, where conducting large well-controlled clinical trials is impractical and/or unethical due to a small patient population, a significant portion of which could be children. Additionally, the totality of evidence generated by model-informed approaches can provide confirmatory evidence for regulatory approval without the need for additional clinical data. In the article, applications of novel quantitative approaches such as quantitative systems pharmacology, disease progression modeling, artificial intelligence, machine learning, modeling of real-world data using model-based meta-analysis and strategies such as external control and patient-reported outcomes as well as clinical trial simulations to optimize trials and sample collection are discussed. Specific case studies of these modeling approaches in rare diseases are provided to showcase applications in drug development and regulatory review. Finally, perspectives are shared on the future state of these modeling approaches in rare diseases drug development along with challenges and opportunities for incorporating such tools in the rational development of drug products.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/622fb9f7f42b553823de73706d785a94402ad18f" target='_blank'>
              New Horizons of Model Informed Drug Development in Rare Diseases Drug Development.
              </a>
            </td>
          <td>
            Amitava Mitra, Nessy Tania, Mariam A Ahmed, Noha Rayad, Rajesh Krishna, Salwa Albusaysi, Rana B Bakhaidar, Elizabeth Shang, Maria Burian, Michelle Martin-Pozo, Islam R Younis
          </td>
          <td>2024-07-11</td>
          <td>Clinical pharmacology and therapeutics</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="By using omics, we can now examine all components of biological systems simultaneously. Deep learning-based drug prediction methods have shown promise by integrating cancer-related multi-omics data. However, the complex interaction between genes poses challenges in accurately projecting multi-omics data. In this research, we present a predictive model for drug response that incorporates diverse types of omics data, comprising genetic mutation, copy number variation, methylation, and gene expression data. This study proposes latent alignment for information mismatch in integration, which is achieved through an attention module capturing interactions among diverse types of omics data. The latent alignment and attention modules significantly improve predictions, outperforming the baseline model, with MSE = 1.1333, F1-score = 0.5342, and AUROC = 0.5776. High accuracy was achieved in predicting drug responses for piplartine and tenovin-6, while the accuracy was comparatively lower for mitomycin-C and obatoclax. The latent alignment module exclusively outperforms the baseline model, enhancing the MSE by 0.2375, the F1-score by 4.84%, and the AUROC by 6.1%. Similarly, the attention module only improves these metrics by 0.1899, 2.88%, and 2.84%, respectively. In the interpretability case study, panobinostat exhibited the most effective predicted response, with a value of −4.895. We provide reliable insights for drug selection in personalized medicine by identifying crucial genetic factors influencing drug response.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f40ba043b018c0ffb309d5df62d65dac2ff38ebf" target='_blank'>
              An Innovative Multi-Omics Model Integrating Latent Alignment and Attention Mechanism for Drug Response Prediction
              </a>
            </td>
          <td>
            Hui-O Chen, Yuan-Chi Cui, Peng-Chan Lin, Jung-Hsien Chiang
          </td>
          <td>2024-06-27</td>
          <td>Journal of Personalized Medicine</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="BACKGROUND
Cardiotoxicity is a major cause of drug withdrawal. The hERG channel, regulating ion flow, is pivotal for heart and nervous system function. Its blockade is a concern in drug development. Predicting hERG blockade is essential for identifying cardiac safety issues. Various QSAR models exist, but their performance varies. Ongoing improvements show promise, necessitating continued efforts to enhance accuracy using emerging deep learning algorithms in predicting potential hERG blockade.


STUDY DESIGN AND METHOD
Using a large training dataset, six individual QSAR models were developed. Additionally, three ensemble models were constructed. All models were evaluated using 10-fold cross-validations and two external datasets.


RESULTS
The 10-fold cross-validations resulted in Mathews correlation coefficient (MCC) values from 0.682 to 0.730, surpassing the best-reported model on the same dataset (0.689). External validations yielded MCC values from 0.520 to 0.715 for the first dataset, exceeding those of previously reported models (0 - 0.599). For the second dataset, MCC values fell between 0.025 and 0.215, aligning with those of reported models (0.112 - 0.220).


CONCLUSIONS
The developed models can assist the pharmaceutical industry and regulatory agencies in predicting hERG blockage activity, thereby enhancing safety assessments and reducing the risk of adverse cardiac events associated with new drug candidates.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38a44a5f72bc076ae255decea155cfb06c06a618" target='_blank'>
              Machine learning and deep learning approaches for enhanced prediction of hERG blockade: a comprehensive QSAR modeling study.
              </a>
            </td>
          <td>
            Jie Liu, Md Kamrul Hasan Khan, Wenjing Guo, Fan Dong, W. Ge, Chaoyang Zhang, Ping Gong, T. Patterson, Huixiao Hong
          </td>
          <td>2024-07-05</td>
          <td>Expert opinion on drug metabolism & toxicology</td>
          <td>0</td>
          <td>31</td>
        </tr>

        <tr id="Abstract Reconstructing the topology of gene regulatory network from gene expression data has been extensively studied. With the abundance functional transcriptomic data available, it is now feasible to systematically decipher regulatory interaction dynamics in a logic form such as a Boolean network (BN) framework, which qualitatively indicates how multiple regulators aggregated to affect a common target gene. However, inferring both the network topology and gene interaction dynamics simultaneously is still a challenging problem since gene expression data are typically noisy and data discretization is prone to information loss. We propose a new method for BN inference from time-series transcriptional profiles, called LogicGep. LogicGep formulates the identification of Boolean functions as a symbolic regression problem that learns the Boolean function expression and solve it efficiently through multi-objective optimization using an improved gene expression programming algorithm. To avoid overly emphasizing dynamic characteristics at the expense of topology structure ones, as traditional methods often do, a set of promising Boolean formulas for each target gene is evolved firstly, and a feed-forward neural network trained with continuous expression data is subsequently employed to pick out the final solution. We validated the efficacy of LogicGep using multiple datasets including both synthetic and real-world experimental data. The results elucidate that LogicGep adeptly infers accurate BN models, outperforming other representative BN inference algorithms in both network topology reconstruction and the identification of Boolean functions. Moreover, the execution of LogicGep is hundreds of times faster than other methods, especially in the case of large network inference.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/81189774cd4d14cf52c8efb5374bdb7e2ca817d4" target='_blank'>
              LogicGep: Boolean networks inference using symbolic regression from time-series transcriptomic profiling data
              </a>
            </td>
          <td>
            Dezhen Zhang, Shuhua Gao, Zhiping Liu, Rui Gao
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Many real-world systems can be represented as sets of interacting components. Examples of such systems include computational systems such as query processors, natural systems such as cells, and social systems such as families. Many approaches have been proposed in traditional (associational) machine learning to model such structured systems, including statistical relational models and graph neural networks. Despite this prior work, existing approaches to estimating causal effects typically treat such systems as single units, represent them with a fixed set of variables and assume a homogeneous data-generating process. We study a compositional approach for estimating individual treatment effects (ITE) in structured systems, where each unit is represented by the composition of multiple heterogeneous components. This approach uses a modular architecture to model potential outcomes at each component and aggregates component-level potential outcomes to obtain the unit-level potential outcomes. We discover novel benefits of the compositional approach in causal inference - systematic generalization to estimate counterfactual outcomes of unseen combinations of components and improved overlap guarantees between treatment and control groups compared to the classical methods for causal effect estimation. We also introduce a set of novel environments for empirically evaluating the compositional approach and demonstrate the effectiveness of our approach using both simulated and real-world data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e838efcfff37782c83e16c5dda1430df92f3f650" target='_blank'>
              Compositional Models for Estimating Causal Effects
              </a>
            </td>
          <td>
            Purva Pruthi, David Jensen
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="10061 Background: Genomics precision medicine, deployed via tumor panel sequencing, now assists in deploying targeted therapies to cancer patients. Numerous clinical trials have investigated the utility and benefit of genomics precision medicine in multiple cancer indications. Current large-scale studies report actionability rates from ~35% to ~60%, although clinical benefit rates have been shown to be closer to 10%. While this has positively impacted patients in need, the gap between actionability and benefit remains a clinical challenge attributed to multiple factors including the complex, multi-factorial relationship between molecular status and response to therapy. These differences go beyond simple disease states and may be reflective of multiple clinically relevant features including age, sex, and race/ethnicity. Methods: We implemented a functional precision medicine (FPM) program where patients with advanced pediatric cancers were prospectively profiled via high-throughput drug sensitivity testing (DST) of FDA-approved agents on patient-derived tumor cells as well as genomics testing. The objective was to investigate the clinical utility and benefit of FPM guidance in the treatment of pediatric cancer and elucidate the relationship between molecular status of patients’ diseases and treatment responses. We generated DST data (n = 21 patients) and genomic profiling data (n = 20 patients) on pediatric cancer patients in Miami, FL, as well as post-hoc whole exome and transcriptome sequencing data (n = 13 patients) and investigated three specific relationships. Results: We analyzed the relationship between racial/ethnic background and functional response to anti-cancer agents, determining potential differences in response to therapeutic classes. Next, we examined relationships between functional response and cancer type, identifying an unanticipated lack of clustering between disease indications in patients with advanced pediatric cancers. Finally, we applied an explainable machine learning (xML) framework to the functional genomic dataset to develop multi-omics biomarker hypotheses for the chemotherapy agent idarubicin, pinpointing a potential multi-cancer relationship between response to idarubicin and known disease mechanisms in acute myeloid leukemia (AML), the sole indication where idarubicin is approved. We further present additional proof-of-concept studies generating biomarker hypotheses via xML, demonstrating a framework for development of multi-omics biomarkers. Conclusions: We are now expanding our pan-pediatric cancer functional genomics dataset through an NIMHD-funded expansion cohort (NCT05857969, n = 65 patients) to further investigate multi-omics relationships between functional and molecular characteristics and understand the role of race/ethnicity in the complex relationship.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/786847ce4af81d26fccb1bc7ad4a99896c1c5825" target='_blank'>
              Biomarker development from functional precision medicine datasets via explainable machine learning.
              </a>
            </td>
          <td>
            Noah Berlow, Arlet M Acanda De La Rocha, Maggie Fader, E. Coats, Cima Saghira, P. Espinal, Jeanette Galano, Z. Khatib, Haneen Y. Abdella, O. Maher, Cristina M. Andrade-Feraud, Baylee Holcomb, Yasmin Ghurani, Lilliam Rimblas, Tom á s R. Guilarte, Nan Hu, Daria Salyakina, Diana J Azzam
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="This study aims to enhance the prediction of COVID-19 vaccination trends using a novel integrated forecasting model, facilitating better public health decision-making and resource allocation during the pandemic. As the COVID-19 pandemic continues to impact global health, accurately forecasting vaccination trends is critical for effective public health response and strategy development. Traditional forecasting models often fail to capture the complex dynamics of pandemic-driven vaccination rates. The analysis utilizes a comprehensive dataset comprising over 68,487 entries, detailing daily vaccination statistics across various demographics and geographic locations. This dataset provides a robust foundation for modeling and forecasting efforts. It utilizes advanced time series analysis techniques and machine learning algorithms to accurately predict future vaccination patterns based on the Hybrid Harvest model, which combines the strengths of ARIMA and Prophet models. Hybrid Harvest exhibits superior performance, with mean-square errors (MSEs) of 0.1323, and root-mean-square errors (RMSEs) of 0.0305. Based on these results, the model is significantly more accurate than traditional forecasting methods when predicting vaccination trends. It offers significant advances in forecasting COVID-19 vaccination trends through integration of ARIMA and Prophet models. The model serves as a powerful tool for policymakers to plan vaccination campaigns efficiently and effectively.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fec88d0d91b9ec7834291f814df1c6be0ffe81e3" target='_blank'>
              Hybrid Time Series Model for Advanced Predictive Analysis in COVID-19 Vaccination
              </a>
            </td>
          <td>
            Amna Khalil, Mazhar Javed Awan, Awais Yasin, Tanzeela Kousar, Abdur Rahman, Mohamed Sebaie Youssef
          </td>
          <td>2024-06-24</td>
          <td>Electronics</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Cells must adopt flexible regulatory strategies to make decisions regarding their fate, including differentiation, apoptosis, or survival in the face of various external stimuli. One key cellular strategy that enables these functions is stochastic gene expression programs. However, understanding how transcriptional bursting, and consequently, cell fate, responds to DNA damage on a genome-wide scale poses a challenge. In this study, we propose an interpretable and scalable inference framework, DeepTX, that leverages deep learning methods to connect mechanistic models and scRNA-seq data, thereby revealing genome-wide transcriptional burst kinetics. This framework enables rapid and accurate solutions to transcription models and the inference of transcriptional burst kinetics from scRNA-seq data. Applying this framework to several scRNA-seq datasets of DNA-damaging drug treatments, we observed that fluctuations in transcriptional bursting induced by different drugs could lead to distinct fate decisions: IdU treatment induces differentiation in mouse embryonic stem cells by increasing the burst size of gene expression, while 5FU treatment with low and high dose increases the burst frequency of gene expression to induce cell apoptosis and survival in human colon cancer cells. Together, these results show that DeepTX can be used to analyze single-cell transcriptomics data and can provide mechanistic insights into cell fate decisions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d08e79a79f41500c7b1baabfdd652bb03e9ecc3e" target='_blank'>
              Deep learning linking mechanistic models to single-cell transcriptomics data reveals transcriptional bursting in response to DNA damage
              </a>
            </td>
          <td>
            Zhiwei Huang, Songhao Luo, Zihao Wang, Zhenquan Zhang, Benyuan Jiang, Qing Nie, Jiajun Zhang
          </td>
          <td>2024-07-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Causal structure learning (CSL) refers to the task of learning causal relationships from data. Advances in CSL now allow learning of causal graphs in diverse application domains, which has the potential to facilitate data-driven causal decision-making. Real-world CSL performance depends on a number of $\textit{context-specific}$ factors, including context-specific data distributions and non-linear dependencies, that are important in practical use-cases. However, our understanding of how to assess and select CSL methods in specific contexts remains limited. To address this gap, we present $\textit{CausalRegNet}$, a multiplicative effect structural causal model that allows for generating observational and interventional data incorporating context-specific properties, with a focus on the setting of gene perturbation experiments. Using real-world gene perturbation data, we show that CausalRegNet generates accurate distributions and scales far better than current simulation frameworks. We illustrate the use of CausalRegNet in assessing CSL methods in the context of interventional experiments in biology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dccde4b2105ed3bb6508dcc81706fd29a96de4c0" target='_blank'>
              Simulation-based Benchmarking for Causal Structure Learning in Gene Perturbation Experiments
              </a>
            </td>
          <td>
            Luka Kovavcevi'c, Izzy Newsham, Sach Mukherjee, John Whittaker
          </td>
          <td>2024-07-08</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Machine learning has been proposed as an alternative to theoretical modeling when dealing with complex problems in biological physics. However, in this perspective, we argue that a more successful approach is a proper combination of these two methodologies. We discuss how ideas coming from physical modeling neuronal processing led to early formulations of computational neural networks, e.g., Hopfield networks. We then show how modern learning approaches like Potts models, Boltzmann machines, and the transformer architecture are related to each other, specifically, through a shared energy representation. We summarize recent efforts to establish these connections and provide examples on how each of these formulations integrating physical modeling and machine learning have been successful in tackling recent problems in biomolecular structure, dynamics, function, evolution, and design. Instances include protein structure prediction; improvement in computational complexity and accuracy of molecular dynamics simulations; better inference of the effects of mutations in proteins leading to improved evolutionary modeling and finally how machine learning is revolutionizing protein engineering and design. Going beyond naturally existing protein sequences, a connection to protein design is discussed where synthetic sequences are able to fold to naturally occurring motifs driven by a model rooted in physical principles. We show that this model is “learnable” and propose its future use in the generation of unique sequences that can fold into a target structure.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/857c85e6a44f8c49c7d41155c041eb3e9361ca60" target='_blank'>
              Machine learning in biological physics: From biomolecular prediction to design
              </a>
            </td>
          <td>
            Jonathan Martin, Marcos Lequerica Mateos, J. Onuchic, Ivan Coluzza, F. Morcos
          </td>
          <td>2024-06-24</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>1</td>
          <td>95</td>
        </tr>

        <tr id="Drug-target relationships may now be predicted computationally using bioinformatics data, which is a valuable tool for understanding pharmacological effects, enhancing drug development efficiency, and advancing related research. A number of structure-based, ligand-based and network-based approaches have now emerged. Furthermore, the integration of graph attention networks with intricate drug target studies is an application area of growing interest. In our work, we formulate a model called MKDTI by extracting kernel information from various layer embeddings of a graph attention network. This combination improves the prediction ability with respect to novel drug-target relationships. We first build a drug-target heterogeneous network using heterogeneous data of drugs and targets, and then use a self-enhanced multi-head graph attention network to extract potential features in each layer. Next, we utilize embeddings of each layer to computationally extract kernel matrices and fuse multiple kernel matrices. Finally, we use a Dual Laplacian Regularized Least Squares framework to forecast novel drug-target entity connections. This prediction can be facilitated by integrating the kernel matrix associated with the drug-target. We measured our model's efficacy using AUPR and AUC. Compared to the benchmark algorithms, our model outperforms them in the prediction outcomes. In addition, we conducted an experiment on kernel selection. The results show that the multi-kernel fusion approach combined with the kernel matrix generated by the graph attention network provides complementary insights into the model. The fusion of this information helps to enhance the accuracy of the predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/824e3d4abd4c6dd5d7628886474532ee9726ded2" target='_blank'>
              MKDTI: Predicting drug-target interactions via multiple kernel fusion on graph attention network
              </a>
            </td>
          <td>
            Yuhuan Zhou, Yulin Wu, Weiwei Yuan, Xuan Wang, Junyi Li
          </td>
          <td>2024-07-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ee1cbb205fe12bffab9e43a3407fc271c7810b96" target='_blank'>
              An end-to-end method for predicting compound-protein interactions based on simplified homogeneous graph convolutional network and pre-trained language model
              </a>
            </td>
          <td>
            Yufang Zhang, Jiayi Li, Shenggeng Lin, Jianwei Zhao, Yi Xiong, Dong-Qing Wei
          </td>
          <td>2024-06-07</td>
          <td>Journal of Cheminformatics</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Physics-informed deep learning has been developed as a novel paradigm for learning physical dynamics recently. While general physics-informed deep learning methods have shown early promise in learning fluid dynamics, they are difficult to generalize in arbitrary time instants in real-world scenario, where the fluid motion can be considered as a time-variant trajectory involved large-scale particles. Inspired by the advantage of diffusion model in learning the distribution of data, we first propose Pi-fusion, a physics-informed diffusion model for predicting the temporal evolution of velocity and pressure field in fluid dynamics. Physics-informed guidance sampling is proposed in the inference procedure of Pi-fusion to improve the accuracy and interpretability of learning fluid dynamics. Furthermore, we introduce a training strategy based on reciprocal learning to learn the quasiperiodical pattern of fluid motion and thus improve the generalizability of the model. The proposed approach are then evaluated on both synthetic and real-world dataset, by comparing it with state-of-the-art physics-informed deep learning methods. Experimental results show that the proposed approach significantly outperforms existing methods for predicting temporal evolution of velocity and pressure field, confirming its strong generalization by drawing probabilistic inference of forward process and physics-informed guidance sampling. The proposed Pi-fusion can also be generalized in learning other physical dynamics governed by partial differential equations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30d533bf917f7e4f8a81a52e26c206fadbb3f08b" target='_blank'>
              Pi-fusion: Physics-informed diffusion model for learning fluid dynamics
              </a>
            </td>
          <td>
            Jing Qiu, Jiancheng Huang, Xiangdong Zhang, Zeng Lin, Minglei Pan, Zengding Liu, F. Miao
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>20</td>
        </tr>

        <tr id="This study presents a novel approach that leverages Neural Ordinary Differential Equations (Neural ODEs) to unravel the intricate relationships between inputs and outputs in Large Language Models (LLMs), and employs robust control to fine-tune outputs to meet predefined standards. Central to our methodology is the transformation of LLM inputs and outputs into a lower-dimensional latent space, facilitating a detailed examination of the information processing pathways within LLMs. Neural ODEs play a pivotal role in this investigation by providing a dynamic model that captures the continuous evolution of data within the LLMs. Additionally, robust control mechanisms are applied to strategically adjust the model's outputs, ensuring they not only maintain high quality and reliability but also adhere to specific performance criteria. This fusion of Neural ODEs and robust control represents a significant advancement in LLM interpretability, offering a comprehensive framework that elucidates the previously opaque mechanisms of these complex models. Our empirical results validate the effectiveness of this integrated approach, making a substantial contribution to the field of explainable AI by merging advanced machine learning techniques with the critical need for transparency and control in AI outputs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e11388eb181a61495a9fb75dd0509dda44b5e3b3" target='_blank'>
              Unveiling LLM Mechanisms Through Neural ODEs and Control Theory
              </a>
            </td>
          <td>
            Yukun Zhang
          </td>
          <td>2024-06-23</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c1cb4a657370a405d2d68807497b1ada06ceb78e" target='_blank'>
              Deep learning framework with multi-perspective social behaviors for vaccine hesitation
              </a>
            </td>
          <td>
            Raed Alharbi, Sylvia Chan-Olmsted, Huan Chen, My T. Thai
          </td>
          <td>2024-07-21</td>
          <td>Social Network Analysis and Mining</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Introduction The identification of microbe–drug associations can greatly facilitate drug research and development. Traditional methods for screening microbe-drug associations are time-consuming, manpower-intensive, and costly to conduct, so computational methods are a good alternative. However, most of them ignore the combination of abundant sequence, structural information, and microbe-drug network topology. Methods In this study, we developed a computational framework based on a modified graph attention variational autoencoder (MGAVAEMDA) to infer potential microbedrug associations by combining biological information with the variational autoencoder. In MGAVAEMDA, we first used multiple databases, which include microbial sequences, drug structures, and microbe-drug association databases, to establish two comprehensive feature matrices of microbes and drugs after multiple similarity computations, fusion, smoothing, and thresholding. Then, we employed a combination of variational autoencoder and graph attention to extract low-dimensional feature representations of microbes and drugs. Finally, the lowdimensional feature representation and graphical adjacency matrix were input into the random forest classifier to obtain the microbe–drug association score to identify the potential microbe-drug association. Moreover, in order to correct the model complexity and redundant calculation to improve efficiency, we introduced a modified graph convolutional neural network embedded into the variational autoencoder for computing low dimensional features. Results The experiment results demonstrate that the prediction performance of MGAVAEMDA is better than the five state-of-the-art methods. For the major measurements (AUC =0.9357, AUPR =0.9378), the relative improvements of MGAVAEMDA compared to the suboptimal methods are 1.76 and 1.47%, respectively. Discussion We conducted case studies on two drugs and found that more than 85% of the predicted associations have been reported in PubMed. The comprehensive experimental results validated the reliability of our models in accurately inferring potential microbe-drug associations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/47263880de820913b7e76a7587f6021c0f385799" target='_blank'>
              Prediction of microbe–drug associations based on a modified graph attention variational autoencoder and random forest
              </a>
            </td>
          <td>
            Bo Wang, Fangjian Ma, Xiao-xin Du, Guangda Zhang, Jingyou Li
          </td>
          <td>2024-05-31</td>
          <td>Frontiers in Microbiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Every time new variants of COVID-19, back to the pandemic, could bring massive loss to human society and traditional models have failed to catch the complexities of COVID-19. Thus, to handle the unpredictable challenges posed by the dynamic nature of COVID-19, Long Short-Term Memory (LSTM) models were utilized to make accurate short and long-term predictions about different variants and prepare the model for another variant or virus similar to COVID-19 by learning the data of different COVID-19 variants. The dataset, sourced from owid-covid-data, is cleaned and divided into original, alpha, delta, and mixed variants in the United States from March 1, 2020, to April 30, 2022. MSE, RMSE, MAE, and R2 are used to compare the difference between real and predicted values to evaluate how accurately the model performs. Results demonstrate the model excels in long-term and short-term predicting COVID-19 cases and deaths for various variants, and mixed variants even promote accuracy. Thus, the proposed LSTM model shows promise for infectious disease forecasting, providing a foundation for anticipating future outbreaks to help policymakers make better decisions or early prevention.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/539cd072a0ce8edae05a9611a8edaa824a3b32b5" target='_blank'>
              Long short-term memory model for COVID-19 forecasting
              </a>
            </td>
          <td>
            Pengfei Lou
          </td>
          <td>2024-05-28</td>
          <td>Theoretical and Natural Science</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The headway and process steps are in innovation and appropriate strategies are set out a freedom for progress in numerous logical fields and different businesses. AI has become significant apparatus for drug plans and disclosure with the accessibility of spot information from enormous data sets. This article discuss about dissect ML and DL procedures it helps Pharmacy company’s in major related phases in medication disclosure they incorporates approval, future clinical events, and medical preliminaries. This application can be developed using machine learning techniques and using some important methods. This approach is very helpful in medicine industry. In present COVID situation vaccine development plays a major role to find the best medicine for that we have to test the medicine multiple times to test the results, if they did manual process they will take lot of time to update the details for that we are introducing this method. KEYWORDS: pharmaceutical synthesis, machine learning ML, Reinforcement learning RL, Medicine industry, supervised Machine Learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d6f1641a36b7144edabbb4e9793feb87a3610604" target='_blank'>
              Pharmaceutical Synthesis and Discloser Approach Using Naïve Bayesian Method in Machine Learning a.
              </a>
            </td>
          <td>

          </td>
          <td>2024-06-13</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Optimal control of multiple heating, ventilation, and air-conditioning units in an open-plan space demands fast and accurate thermodynamic modeling. While computational fluid dynamics is precise, it is computationally impractical for real-time control. Lumped thermal models divide the room into cells and explicitly model inter-cell air-mixing. However, solving the coupled differential equations is still computationally challenging. Data-driven methods like long-short term memory may handle non-linear thermal dynamics but face limitations in observability, adherence to physical constraints, and generalization. Physics-informed neural networks (PINN) impose physical constraints even while learning from data. However, PINNs still lack the scalability required for effective control in large open-plan offices, primarily due to air-mixing interactions Our approach, PhyGICS, combines graph neural networks with physics-informed learning (PI-GNN) to overcome these challenges. Specifically, we model the thermodynamic interactions as edges between nodes that represent cells. Further, the modeling approach allows explicit modeling of wall and window surface temperatures that are commonly ignored. PhyGICS, utilizing PI-GNN as a state-estimator, employs a receding-horizon approach for optimal HVAC control. We adapt PI-GNNs for building HVAC control by incorporating a time-resetting strategy to handle time-dependent ambient conditions and therefore set-points. Evaluations in a simulated environment with real-world data show PhyGICS outperforms a regular PINN model and other baseline control strategies on: 1) thermal model accuracy; 2) computation time; 3) energy consumption; and 4) user comfort.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/77b9dc12e24efd410665c6606c578c428451e250" target='_blank'>
              PhyGICS – A Physics-informed Graph Neural Network-based Intelligent HVAC Controller for Open-plan Spaces
              </a>
            </td>
          <td>
            S. Nagarathinam, Arunchandar Vasan
          </td>
          <td>2024-05-31</td>
          <td>Proceedings of the 15th ACM International Conference on Future and Sustainable Energy Systems</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Molecular simulations enable the prediction of physicochemical properties of mixtures based on pair-interaction models of the pure components and combining rules to describe the unlike interactions. However, if no adjustment to experimental data is made, the existing combining rules often do not yield sufficiently accurate predictions of mixture data. To address this problem, adjustable binary parameters ξij describing the pair interactions in mixtures (i + j) are used. In this work, we present the first method for predicting ξij for unstudied mixtures based on a matrix completion method (MCM) from machine learning (ML). Considering molecular simulations of Henry's law constants as an example, we demonstrate that ξij for unstudied mixtures can be predicted with high accuracy. Using the predicted ξij significantly increases the accuracy of the Henry's law constant predictions compared to using the default ξij = 1. Our approach is generic and can be transferred to molecular simulations of other mixture properties and even to combining rules in equations of state, granting predictive access to the description of unlike intermolecular interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/014978f89a70a7a9fa83037f07dcdb4ad1e66950" target='_blank'>
              Prediction of pair interactions in mixtures by matrix completion.
              </a>
            </td>
          <td>
            Marco Hoffmann, N. Hayer, M. Kohns, F. Jirasek, Hans Hasse
          </td>
          <td>2024-07-08</td>
          <td>Physical chemistry chemical physics : PCCP</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Molecular analytics increasingly utilize machine learning (ML) for predictive modeling based on data acquired through molecular profiling technologies. However, developing robust models that accurately capture physiological phenotypes is challenged by a multitude of factors. These include the dynamics inherent to biological systems, variability stemming from analytical procedures, and the resource-intensive nature of obtaining sufficiently representative datasets. Here, we propose and evaluate a new method: Contextual Out-of-Distribution Integration (CODI). Based on experimental observations, CODI generates synthetic data that integrate unrepresented sources of variation encountered in real-world applications into a given molecular fingerprint dataset. By augmenting a dataset with out-of-distribution variance, CODI enables an ML model to better generalize to samples beyond the initial training data. Using three independent longitudinal clinical studies and a case-control study, we demonstrate CODI’s application to several classification scenarios involving vibrational spectroscopy of human blood. We showcase our approach’s ability to enable personalized fingerprinting for multi-year longitudinal molecular monitoring and enhance the robustness of trained ML models for improved disease detection. Our comparative analyses revealed that incorporating CODI into the classification workflow consistently led to significantly improved classification accuracy while minimizing the requirement of collecting extensive experimental observations. SIGNIFICANCE STATEMENT Analyzing molecular fingerprint data is challenging due to multiple sources of biological and analytical variability. This variability hinders the capacity to collect sufficiently large and representative datasets that encompass realistic data distributions. Consequently, the development of machine learning models that generalize to unseen, independently collected samples is often compromised. Here, we introduce CODI, a versatile framework that enhances traditional classifier training methodologies. CODI is a general framework that incorporates information about possible out-of-distribution variations into a given training dataset, augmenting it with simulated samples that better capture the true distribution of the data. This allows the classification to achieve improved predictive performance on samples beyond the original distribution of the training data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8f5a765a1c0f589cfa9eccf1b65dd12320bb5114" target='_blank'>
              CODI: Enhancing machine learning-based molecular profiling through contextual out-of-distribution integration
              </a>
            </td>
          <td>
            Tarek Eissa, Marinus Huber, Barbara Obermayer-Pietsch, Birgit Linkohr, A. Peters, Frank Fleischmann, M. Žigman
          </td>
          <td>2024-06-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="This study proposes a new method to improve the efficiency of drug discovery by repurposing existing drugs, aiming to reduce the time and costs associated with traditional drug development processes, which can span 10 to 15 years and cost billions of dollars. Current approaches focus on leveraging heterogeneous data, such as drug-protein and disease-protein interactions, to construct complex networks that link drugs, proteins, and diseases. However, a significant challenge is the imbalance in data, where numerous unconfirmed potential drug-disease interactions (the majority class) outnumber approved drugs (the minority class), severely impacting the predictive performance of machine learning models. Previous attempts to address this issue have shown limited success. This study introduces a novel approach that integrates meta-paths in heterogeneous information networks with data balancing techniques to tackle this imbalance. Experimental results demonstrate that the proposed method enhances model performance and reliability in identifying new relationships between drugs and diseases. This research represents a promising advancement by leveraging network-based strategies and data balancing techniques to facilitate the rediscovery of drug applications, thereby potentially revolutionizing the pharmaceutical industry’s approach to drug development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/25438f7cc83aaed0d4842e6721d7f9054712298b" target='_blank'>
              ENHANCING DRUG DISCOVERY THROUGH A META-PATH BASED OVERSAMPLING APPROACH FOR IMBALANCED DATA
              </a>
            </td>
          <td>
            Manh Hung Le
          </td>
          <td>2024-06-30</td>
          <td>Journal of Science and Technique</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Capturing the time evolution and predicting future kinetic states of physicochemical systems present significant challenges due to the precision and computational effort required. In this study, we demonstrate that the transformer, a machine learning model renowned for machine translation and natural language processing, can be effectively adapted to predict the dynamical state-to-state transition kinetics of biologically relevant physicochemical systems. Specifically, by using sequences of time-discretized states from Molecular Dynamics (MD) simulation trajectories as input, we show that a transformer can learn the complex syntactic and semantic relationships within the trajectory. This enables this generative pre-trained transformer (GPT) to predict kinetically accurate sequences of future states for a diverse set of models and biomolecules of varying complexity. Remarkably, the GPT can predict future states much faster than traditional MD simulations. We show that it is particularly adept at forecasting the time evolution of an out-of-equilibrium active system that do not maintain detailed balance. An analysis of self-attention mechanism inherent in transformers is found to hold crucial role for capturing the long-range correlations necessary for accurate state-to-state transition predictions. Together, our results highlight the ability of transformer based machine learning model in generating future states of physicochemical systems with statistical precision.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3f68df42b37752017ad05438942ecb38a7e24c6e" target='_blank'>
              Predicting Future Kinetic States of Physicochemical Systems Using Generative Pre-trained Transformer
              </a>
            </td>
          <td>
            Palash Bera, Jagannath Mondal
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="When studying ecosystems, hierarchical trees are often used to organize entities based on proximity criteria, such as the taxonomy in microbiology, social classes in geography, or product types in retail businesses, offering valuable insights into entity relationships. Despite their significance, current count-data models do not leverage this structured information. In particular, the widely used Poisson log-normal (PLN) model, known for its ability to model interactions between entities from count data, lacks the possibility to incorporate such hierarchical tree structures, limiting its applicability in domains characterized by such complexities. To address this matter, we introduce the PLN-Tree model as an extension of the PLN model, specifically designed for modeling hierarchical count data. By integrating structured variational inference techniques, we propose an adapted training procedure and establish identifiability results, enhancisng both theoretical foundations and practical interpretability. Additionally, we extend our framework to classification tasks as a preprocessing pipeline, showcasing its versatility. Experimental evaluations on synthetic datasets as well as real-world microbiome data demonstrate the superior performance of the PLN-Tree model in capturing hierarchical dependencies and providing valuable insights into complex data structures, showing the practical interest of knowledge graphs like the taxonomy in ecosystems modeling.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/86dd1a5a64cacf81f6895c4656147dbabe2e7e27" target='_blank'>
              Tree-based variational inference for Poisson log-normal models
              </a>
            </td>
          <td>
            Alexandre Chaussard, Anna Bonnet, Elisabeth Gassiat, S. Corff
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Computational chemistry is an indispensable tool for understanding molecules and predicting chemical properties. However, traditional computational methods face significant challenges due to the difficulty of solving the Schrödinger equations and the increasing computational cost with the size of the molecular system. In response, there has been a surge of interest in leveraging artificial intelligence (AI) and machine learning (ML) techniques to in silico experiments. Integrating AI and ML into computational chemistry increases the scalability and speed of the exploration of chemical space. However, challenges remain, particularly regarding the reproducibility and transferability of ML models. This review highlights the evolution of ML in learning from, complementing, or replacing traditional computational chemistry for energy and property predictions. Starting from models trained entirely on numerical data, a journey set forth toward the ideal model incorporating or learning the physical laws of quantum mechanics. This paper also reviews existing computational methods and ML models and their intertwining, outlines a roadmap for future research, and identifies areas for improvement and innovation. Ultimately, the goal is to develop AI architectures capable of predicting accurate and transferable solutions to the Schrödinger equation, thereby revolutionizing in silico experiments within chemistry and materials science.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0c965a92a4418b12137dd3227bb9be135171b111" target='_blank'>
              In Silico Chemical Experiments in the Age of AI: From Quantum Chemistry to Machine Learning and Back.
              </a>
            </td>
          <td>
            Abdulrahman Aldossary, J. A. Campos-Gonzalez-Angulo, Sergio Pablo-García, Shi Xuan Leong, Ella Miray Rajaonson, Luca Thiede, Gary Tom, Andrew Wang, Davide Avagliano, Alán Aspuru-Guzik
          </td>
          <td>2024-05-25</td>
          <td>Advanced materials</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Molecular machine learning algorithms are becoming increasingly powerful at predicting the potency of potential drug candidates to guide molecular discovery, lead series prioritization, and structural optimization. However, a substantial amount of inhibition data is bounded and inaccessible to traditional regression algorithms. Here, we develop a novel molecular pairing approach to process this data. This creates a new classification task of predicting which one of two paired molecules is more potent. This novel classification task can be accurately solved by various, established molecular machine learning algorithms, including XGBoost and Chemprop. Across 230 ChEMBL IC50 datasets, both tree-based and neural network-based “DeltaClassifiers” show improvements over traditional regression approaches in correctly classifying molecular potency improvements. The Chemprop-based deep DeltaClassifier outperformed all here evaluated regression approaches for paired molecules with shared and with distinct scaffolds, highlighting the promise of this approach for molecular optimization and scaffold-hopping.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fdce6ace9180fc9d7076f2b767bc332d5e680b78" target='_blank'>
              Leveraging bounded datapoints to classify molecular potency improvements
              </a>
            </td>
          <td>
            Zachary Fralish, Paul Skaluba, Daniel Reker
          </td>
          <td>2024-05-31</td>
          <td>RSC Medicinal Chemistry</td>
          <td>1</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0ff6646affa90da3577586b9781963d665063e99" target='_blank'>
              An experimentally validated approach to automated biological evidence generation in drug discovery using knowledge graphs
              </a>
            </td>
          <td>
            Saatviga Sudhahar, Bugra Ozer, Jiakang Chang, Wayne Chadwick, Daniel O'Donovan, Aoife Campbell, Emma Tulip, Neil Thompson, Ian Roberts
          </td>
          <td>2024-07-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Models have always been central to inferring molecular evolution and to reconstructing phylogenetic trees. Their use typically involves the development of a mechanistic framework reflecting our understanding of the underlying biological processes, such as nucleotide substitu- tions, and the estimation of model parameters by maximum likelihood or Bayesian inference. However, deriving and optimizing the likelihood of the data is not always possible under complex evolutionary scenarios or even tractable for large datasets, often leading to unrealistic simplifying assumptions in the fitted models. To overcome this issue, we coupled stochastic simulations of genome evolution with a new supervised deep learning model to infer key parameters of molecular evolution. Our model is designed to directly analyze multiple sequence alignments and estimate per-site evolutionary rates and divergence, without requiring a known phylogenetic tree. The accuracy of our predictions matched that of likelihood-based phylogenetic inference, when rate heterogeneity followed a simple gamma distribution, but it strongly exceeded it under more complex patterns of rate variation, such as codon models. Our approach is highly scalable and can be efficiently applied to genomic data, as we showed on a dataset of 26 million nucleotides from the clownfish clade. Our simulations also showed that the integration of per-site rates obtained by deep learning within a Bayesian framework led to significantly more accu- rate phylogenetic inference, particularly with respect to the estimated branch lengths. We thus propose that future advancements in phylogenetic analysis will benefit from a semi-supervised learning approach that combines deep-learning estimation of substitution rates, which allows for more flexible models of rate variation, and probabilistic inference of the phylogenetic tree, which guarantees interpretability and a rigorous assessment of statistical support.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/02e5726c0265f212865a58fafea78c3fde46d235" target='_blank'>
              Toward a semi-supervised learning approach to phylogenetic estimation.
              </a>
            </td>
          <td>
            Daniele Silvestro, Thibault Latrille, N. Salamin
          </td>
          <td>2024-06-25</td>
          <td>Systematic biology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Identification of critical nodes is a prominent topic in the study of complex networks. Numerous methods have been proposed, yet most exhibit inherent limitations. Traditional approaches primarily analyze specific structural features of the network; however, node influence is typically the result of a combination of multiple factors. Machine learning-based methods struggle to effectively represent the complex characteristics of network structures through suitable embedding techniques and require substantial data for training, rendering them prohibitively costly for large-scale networks. To address these challenges, this paper presents an active learning model based on GraphSAGE and Transformer, named GNNTAL. This model is initially pre-trained on random or synthetic networks and subsequently fine-tuned on real-world networks by selecting a few representative nodes using K-Means clustering and uncertainty sampling. This approach offers two main advantages: (1) it significantly reduces training costs; (2) it simultaneously incorporates both local and global features. A series of comparative experiments conducted on twelve real-world networks demonstrate that GNNTAL achieves superior performance. Additionally, this paper proposes an influence maximization method based on the predictions of the GNNTAL model, which achieves optimal performance without the need for complex computations. Finally, the paper analyses certain limitations of the GNNTAL model and suggests potential solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f2c2569fd1bd2320230e824c982a501344641e9c" target='_blank'>
              GNNTAL:A Novel Model for Identifying Critical Nodes in Complex Networks
              </a>
            </td>
          <td>
            Hao Wang, Ting Luo, Shuang-ping Yang, Ming Jing, Jian Wang, Na Zhao
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Hybrid modeling aims to augment traditional theory-driven models with machine learning components that learn unknown parameters, sub-models or correction terms from data. In this work, we build on FluxRGNN, a recently developed hybrid model of continental-scale bird migration, which combines a movement model inspired by fluid dynamics with recurrent neural networks that capture the complex decision-making processes of birds. While FluxRGNN has been shown to successfully predict key migration patterns, its spatial resolution is constrained by the typically sparse observations obtained from weather radars. Additionally, its trainable components lack explicit incentives to adequately predict take-off and landing events. Both aspects limit our ability to interpret model results ecologically. To address this, we propose two major modifications that allow for more detailed predictions on any desired tessellation while providing control over the interpretability of model components. In experiments on the U.S. weather radar network, the enhanced model effectively leverages the underlying movement model, resulting in strong extrapolation capabilities to unobserved locations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/db8ecd0f823bc56c0fe6b87a31bdeea470304f59" target='_blank'>
              Towards detailed and interpretable hybrid modeling of continental-scale bird migration
              </a>
            </td>
          <td>
            Fiona Lippert, Bart Kranstauber, Patrick Forr'e, E. E. V. Loon
          </td>
          <td>2024-07-14</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Background Alzheimer’s disease (AD), a progressive neurodegenerative disorder, continues to increase in prevalence without any effective treatments to date. In this context, knowledge graphs (KGs) have emerged as a pivotal tool in biomedical research, offering new perspectives on drug repurposing and biomarker discovery by analyzing intricate network structures. Our study seeks to build an AD-specific knowledge graph, highlighting interactions among AD, genes, variants, chemicals, drugs, and other diseases. The goal is to shed light on existing treatments, potential targets, and diagnostic methods for AD, thereby aiding in drug repurposing and the identification of biomarkers. Results We annotated 800 PubMed abstracts and leveraged GPT-4 for text augmentation to enrich our training data for named entity recognition (NER) and relation classification. A comprehensive data mining model, integrating NER and relationship classification, was trained on the annotated corpus. This model was subsequently applied to extract relation triplets from unannotated abstracts. To enhance entity linking, we utilized a suite of reference biomedical databases and refine the linking accuracy through abbreviation resolution. As a result, we successfully identified 3,199,276 entity mentions and 633,733 triplets, elucidating connections between 5,000 unique entities. These connections were pivotal in constructing a comprehensive Alzheimer’s Disease Knowledge Graph (ADKG). We also integrated the ADKG constructed after entity linking with other biomedical databases. The ADKG served as a training ground for Knowledge Graph Embedding models with the high-ranking predicted triplets supported by evidence, underscoring the utility of ADKG in generating testable scientific hypotheses. Further application of ADKG in predictive modeling using the UK Biobank data revealed models based on ADKG outperforming others, as evidenced by higher values in the areas under the receiver operating characteristic (ROC) curves. Conclusion The ADKG is a valuable resource for generating hypotheses and enhancing predictive models, highlighting its potential to advance AD’s disease research and treatment strategies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c8fbfe5663f066385e42e86172396f4c89a633e0" target='_blank'>
              Alzheimer’s Disease Knowledge Graph Enhances Knowledge Discovery and Disease Prediction
              </a>
            </td>
          <td>
            Yue Yang, Kaixian Yu, Shan Gao, Sheng Yu, Di Xiong, Chuanyang Qin, Huiyuan Chen, Jiarui Tang, Niansheng Tang, Hongtu Zhu
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Data-driven modeling methods are studied for turbulent dynamical systems with extreme events under an unambiguous model framework. New neural network architectures are proposed to effectively learn the key dynamical mechanisms including the multiscale coupling and strong instability, and gain robust skill for long-time prediction resistive to the accumulated model errors from the data-driven approximation. The machine learning model overcomes the inherent limitations in traditional long short-time memory networks by exploiting a conditional Gaussian structure informed of the essential physical dynamics. The model performance is demonstrated under a prototype model from idealized geophysical flow and passive tracers, which exhibits analytical solutions with representative statistical features. Many attractive properties are found in the trained model in recovering the hidden dynamics using a limited dataset and sparse observation time, showing uniformly high skill with persistent numerical stability in predicting both the trajectory and statistical solutions among different statistical regimes away from the training regime. The model framework is promising to be applied to a wider class of turbulent systems with complex structures.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbfd6c205ba80b0b4c9e84e5d4fbba3b016ab7b3" target='_blank'>
              Unambiguous Models and Machine Learning Strategies for Anomalous Extreme Events in Turbulent Dynamical System
              </a>
            </td>
          <td>
            D. Qi
          </td>
          <td>2024-06-01</td>
          <td>Entropy</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Scientific Machine Learning is a new class of approaches that integrate physical knowledge and mechanistic models with data-driven techniques for uncovering governing equations of complex processes. Among the available approaches, Universal Differential Equations (UDEs) are used to combine prior knowledge in the form of mechanistic formulations with universal function approximators, like neural networks. Integral to the efficacy of UDEs is the joint estimation of parameters within mechanistic formulations and the universal function approximators using empirical data. The robustness and applicability of resultant models, however, hinge upon the rigorous quantification of uncertainties associated with these parameters, as well as the predictive capabilities of the overall model or its constituent components. With this work, we provide a formalisation of uncertainty quantification (UQ) for UDEs and investigate important frequentist and Bayesian methods. By analysing three synthetic examples of varying complexity, we evaluate the validity and efficiency of ensembles, variational inference and Markov chain Monte Carlo sampling as epistemic UQ methods for UDEs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/320dbbdecad0c6895c5ce6a4b03811b5cb0b85c8" target='_blank'>
              Assessment of Uncertainty Quantification in Universal Differential Equations
              </a>
            </td>
          <td>
            Nina Schmid, David Fernandes del Pozo, Willem Waegeman, Jan Hasenauer
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Mathematical models in ecology and epidemiology must be consistent with observed data in order to generate reliable knowledge and evidence-based policy. Metapopulation systems, which consist of a network of connected sub-populations, pose technical challenges in statistical inference owing to nonlinear, stochastic interactions. Numerical difficulties encountered in conducting inference can obstruct the core scientific questions concerning the link between the mathematical models and the data. Recently, an algorithm has been proposed that enables computationally tractable likelihood-based inference for high-dimensional partially observed stochastic dynamic models of metapopulation systems. We use this algorithm to build a statistically principled data analysis workflow for metapopulation systems. Via a case study of COVID-19, we show how this workflow addresses the limitations of previous approaches. The COVID-19 pandemic provides a situation where mathematical models and their policy implications are widely visible, and we revisit an influential metapopulation model used to inform basic epidemiological understanding early in the pandemic. Our methods support self-critical data analysis, enabling us to identify and address model weaknesses, leading to a new model with substantially improved statistical fit and parameter identifiability. Our results suggest that the lockdown initiated on 23 January 2020 in China was more effective than previously thought.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc0af1d4806696c45152280c39d4ffa45f9e5436" target='_blank'>
              Inference on spatiotemporal dynamics for coupled biological populations.
              </a>
            </td>
          <td>
            Jifan Li, E. Ionides, Aaron A. King, Mercedes Pascual, Ning Ning
          </td>
          <td>2024-07-01</td>
          <td>Journal of the Royal Society, Interface</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Recent advances in high-throughput, multi-condition experiments allow for genome-wide investigation of how perturbations affect transcription and translation in the cell across multiple biological entities or modalities, from chromatin and mRNA information to protein production and spatial morphology. This presents an unprecedented opportunity to unravel how the processes of DNA and RNA regulation direct cell fate determination and disease response. Most methods designed for analyzing large-scale perturbation data focus on the observational outcomes, e.g., expression; however, many potential transcriptional mechanisms, such as transcriptional bursting or splicing dynamics, can underlie these complex and noisy observations. In this analysis, we demonstrate how a stochastic biophysical modeling approach to interpreting high-throughout perturbation data enables deeper investigation of the ‘how’ behind such molecular measurements. Our approach takes advantage of modalities already present in data produced with current technologies, such as nascent and mature mRNA measurements, to illuminate transcriptional dynamics induced by perturbation, predict kinetic behaviors in new perturbation settings, and uncover novel populations of cells with distinct kinetic responses to perturbation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b5f2e4f891098d51ebc75d21062e3cc342d8d190" target='_blank'>
              Stochastic Modeling of Biophysical Responses to Perturbation
              </a>
            </td>
          <td>
            T. Chari, Gennady Gorin, L. Pachter
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>75</td>
        </tr>

        <tr id="Machine learning (ML) algorithms are necessary to efficiently identify potent drug combinations within a large candidate space to combat drug resistance. However, existing ML approaches cannot be applied to emerging and under-studied pathogens with limited training data. To address this, we developed a transfer learning and crowdsourcing framework (TACTIC) to train ML models on data from multiple bacteria. TACTIC was built using 2,965 drug interactions from 12 bacterial strains and outperformed traditional ML models in predicting drug interaction outcomes for species that lack training data. Top TACTIC model features revealed genetic and metabolic factors that influence cross- species and species-specific drug interaction outcomes. Upon analyzing ∼600,000 predicted drug interactions across 9 metabolic environments and 18 bacterial strains, we identified a small set of drug interactions that are selectively synergistic against Gram- negative (e.g., A. baumannii) and non-tuberculous mycobacteria (NTM) pathogens. We experimentally validated synergistic drug combinations containing clarithromycin, ampicillin, and mecillinam against M. abscessus, an emerging pathogen with growing levels of antibiotic resistance. Lastly, we leveraged TACTIC to propose selectively synergistic drug combinations to treat bacterial eye infections (endophthalmitis).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5017bc92dbb38d49a788006dff57d7610ff49b95" target='_blank'>
              Transfer learning predicts species-specific drug interactions in emerging pathogens
              </a>
            </td>
          <td>
            Carolina H. Chung, David C. Chang, Nicole M. Rhoads, Madeline R. Shay, Karthik Srinivasan, M. Okezue, Ashlee D. Brunaugh, Sriram Chandrasekaran
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="Global environmental issues and sustainable development call for new technologies for fine chemical synthesis and waste valorization. Biocatalysis has attracted great attention as the alternative to the traditional organic synthesis. However, it is challenging to navigate the vast sequence space to identify those proteins with admirable biocatalytic functions. The recent development of deep-learning based structure prediction methods such as AlphaFold2 reinforced by different computational simulations or multiscale calculations has largely expanded the 3D structure databases and enabled structure-based design. While structure-based approaches shed light on site-specific enzyme engineering, they are not suitable for large-scale screening of potential biocatalysts. Effective utilization of big data using machine learning techniques opens up a new era for accelerated predictions. Here, we review the approaches and applications of structure-based and machine-learning guided enzyme design. We also provide our view on the challenges and perspectives on effectively employing enzyme design approaches integrating traditional molecular simulations and machine learning, and the importance of database construction and algorithm development in attaining predictive ML models to explore the sequence fitness landscape for the design of admirable biocatalysts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5f8f0df327dacd3f505f7842b1b7a98f6a44dcd1" target='_blank'>
              Navigating the landscape of enzyme design: from molecular simulations to machine learning.
              </a>
            </td>
          <td>
            Jiahui Zhou, Meilan Huang
          </td>
          <td>2024-07-11</td>
          <td>Chemical Society reviews</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Machine learning algorithms identify patterns that would otherwise be difficult to observe in high-dimensional molecular and clinical data. For this reason, machine learning has the potential to have a profound impact on clinical decision making and drug target discovery. Nevertheless, there remain considerable technical challenges in adapting these tools for clinical use. These challenges include clinical feature engineering, model selection, and defining optimal strategies for model training. For cancer care, RNA sequencing of patient tumor biopsies has already proven to be a powerful molecular assay to characterize tumor-intrinsic and -extrinsic phenotypes influencing therapeutic response. To improve the predictive performance of RNA-sequencing data, we developed the tauX machine learning framework to refine gene expression features and improve the performance of machine learning algorithms. The tauX framework uses aggregated ratios of positively and negatively associated predictive genes to simplify the prediction task. We showed a significant improvement in predictive performance using a large database of synthetic gene expression profiles. We also show how the tauX framework can be used to elucidate the mechanisms of response and resistance to checkpoint blockade therapy using data from the Stand Up to Cancer (SU2C) Lung Response Cohort and The Cancer Genome Atlas (TCGA). The tauX strategy achieved superior predictive performance compared to models built upon established feature engineering strategies or widely used cancer gene expression signatures. The tauX framework is available as a freely deployable docker container (https://hub.docker.com/r/pfeiljx/taux).">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/dc34a4fff2c622d191b05555caf0fcd51e742fe5" target='_blank'>
              tauX: A Gene Expression Ratio Strategy to Improve Machine Learning Applications in Precision Medicine
              </a>
            </td>
          <td>
            Jacob Pfeil, Liqian Ma, Hin Ching Lo, Tolga Turan, R. T. McLaughlin, Xu Shi, Severiano Villarruel, Stephen Wilson, Xi Zhao, Josue Samayoa, Kyle Halliwill
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/818ab3acef2b3ecce3e559b8cbeba9611590fb10" target='_blank'>
              Uncovering hidden and complex relations of pandemic dynamics using an AI driven system
              </a>
            </td>
          <td>
            Umit Demirbaga, Navneet Kaur, G. Aujla
          </td>
          <td>2024-07-04</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="The study of effective connectivity (EC) is essential in understanding how the brain integrates and responds to various sensory inputs. Model-driven estimation of EC is a powerful approach that requires estimating global and local parameters of a generative model of neural activity. Insights gathered through this process can be used in various applications, such as studying neurodevelopmental disorders. However, accurately determining EC through generative models remains a significant challenge due to the complexity of brain dynamics and the inherent noise in neural recordings, e.g., in electroencephalography (EEG). Current model-driven methods to study EC are computationally complex and cannot scale to all brain regions as required by whole-brain analyses. To facilitate EC assessment, an inference algorithm must exhibit reliable prediction of parameters in the presence of noise. Further, the relationship between the model parameters and the neural recordings must be learnable. To progress toward these objectives, we benchmarked the performance of a Bi-LSTM model for parameter inference from the Jansen-Rit neural mass model (JR-NMM) simulated EEG under various noise conditions. Additionally, our study explores how the JR-NMM reacts to changes in key biological parameters (i.e., sensitivity analysis) like synaptic gains and time constants, a crucial step in understanding the connection between neural mechanisms and observed brain activity. Our results indicate that we can predict the local JR-NMM parameters from EEG, supporting the feasibility of our deep-learning-based inference approach. In future work, we plan to extend this framework to estimate local and global parameters from real EEG in clinically relevant applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fd046ab8f0278c72f414f856472d80db3160b67" target='_blank'>
              Benchmarking Deep Jansen-Rit Parameter Inference: An in Silico Study
              </a>
            </td>
          <td>
            Deepa Tilwani, Christian O'Reilly
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Abstract Inferring cell type proportions from bulk transcriptome data is crucial in immunology and oncology. Here, we introduce guided LDA deconvolution (GLDADec), a bulk deconvolution method that guides topics using cell type-specific marker gene names to estimate topic distributions for each sample. Through benchmarking using blood-derived datasets, we demonstrate its high estimation performance and robustness. Moreover, we apply GLDADec to heterogeneous tissue bulk data and perform comprehensive cell type analysis in a data-driven manner. We show that GLDADec outperforms existing methods in estimation performance and evaluate its biological interpretability by examining enrichment of biological processes for topics. Finally, we apply GLDADec to The Cancer Genome Atlas tumor samples, enabling subtype stratification and survival analysis based on estimated cell type proportions, thus proving its practical utility in clinical settings. This approach, utilizing marker gene names as partial prior information, can be applied to various scenarios for bulk data deconvolution. GLDADec is available as an open-source Python package at https://github.com/mizuno-group/GLDADec.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bbaa85890721d8dafe5fdcc17cdf0f878d4b54b4" target='_blank'>
              GLDADec: marker-gene guided LDA modeling for bulk gene expression deconvolution
              </a>
            </td>
          <td>
            Iori Azuma, T. Mizuno, Hiroyuki Kusuhara
          </td>
          <td>2024-05-23</td>
          <td>Briefings in Bioinformatics</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Machine Learning (ML) is extensively used for predicting transfer times for general purpose Wide Area Networks (WANs) or public Internet applications, but for Research and Education Networks (RENs) two major gaps exist in literature. First, RENs i.e. networks carrying large data flows have received limited attention by the networking community. RENs behave differently compared to the general purpose Internet applications and other network types. Hence, ML models from other network types cannot be used interchangeably for large data transfers. Second, the ML models are used as blackboxes to train on measured network values and then used to predict transfer times or other runtime network parameters. In this paper, we present a dynamical systems model of the large data transfers typical of RENs in the form of a system of Ordinary Differential Equations (ODEs) inspired by the Lotka-Volterra competition model. We present a transfer time prediction component called Dynamic Transfer Time Predictor (DTTP) which solves the ODEs and predicts the future transfer times. Second we formulate a loss function based on Lyapunov function called Lyapunov Drift Correction (LDC) that self-corrects the transfer time prediction errors dynamically.To design and develop our model, we studied real-world datasets consisting of over 100 million transfer records collected from platforms such as Open Science Grid (OSG), Large Hadron Collider Optical Private Network (LHCOPN), Worldwide LHC Grid (WLCG), as well as the RENs of Internet2 and ESNet. We integrate our model into well-known neural network models and regressors and present evaluation results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4cf5d71336cf90025f7d641a600628cc1d15b0d7" target='_blank'>
              Improving Transfer Time Prediction of ML Models via Auto-correcting Dynamical Systems Modeling
              </a>
            </td>
          <td>
            Venkat Sai Suman Lamba Karanam, B. Ramamurthy
          </td>
          <td>2024-06-24</td>
          <td>2024 IEEE 10th International Conference on Network Softwarization (NetSoft)</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="This review explores the concept of synergy in pharmacology, emphasizing its importance in optimizing treatment outcomes through the combination of drugs with different mechanisms of action. Synergy, defined as an effect greater than the expected additive effect elicited by individual agents according to specific predictive models, offers a promising approach to enhance therapeutic efficacy while minimizing adverse events. The historical evolution of synergy research, from ancient civilizations to modern pharmacology, highlights the ongoing quest to understand and harness synergistic interactions. Key concepts such as concentration-response curves, additive effects, and predictive models are discussed in detail, emphasizing the need for accurate assessment methods throughout translational drug development. While various mathematical models exist for synergy analysis, selecting the appropriate model and software tools remains a challenge, necessitating careful consideration of experimental design and data interpretation. Furthermore, this review addresses practical considerations in synergy assessment, including preclinical and clinical approaches, mechanism of action, and statistical analysis. Optimizing synergy requires attention to concentration/dose ratios, target site localization, and timing of drug administration, ensuring that the benefits of combination therapy detected at bench-side are translatable into clinical practice. Overall, the review advocates for a systematic approach to synergy assessment, incorporating robust statistical analysis, effective and simplified predictive models, and collaborative efforts across pivotal sectors such as academic institutions, pharmaceutical companies, and regulatory agencies. By overcoming critical challenges and maximizing therapeutic potential, effective synergy assessment in drug development holds promise for advancing patient care. Significance Statement Combining drugs with different mechanisms of action for synergistic interactions optimizes treatment efficacy and safety. Accurate interpretation of synergy requires the identification of the expected additive effect. Despite innovative models to predict the additive effect, consensus in drug interaction research is lacking, hindering the bench-to-bedside development of combination therapies. Collaboration among science, industry, and regulation is crucial for advancing combination therapy development, ensuring rigorous application of predictive models in clinical settings.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef37bca029116239fa36d9b1e00dfeb6956e78fd" target='_blank'>
              Drug-drug interactions and synergy: from pharmacological models to clinical application.
              </a>
            </td>
          <td>
            L. Calzetta, Clive P. Page, M. Matera, Mario Cazzola, P. Rogliani
          </td>
          <td>2024-07-15</td>
          <td>Pharmacological reviews</td>
          <td>0</td>
          <td>52</td>
        </tr>

        <tr id="Single-cell time series data frequently display considerable variability across a cell population. The current gold standard for inferring parameter distributions across cell populations is the Global Two Stage (GTS) approach for nonlinear mixed-effects (NLME) models. However, this method is computationally intensive, as it makes repeated use of non-convex optimization that in turn requires numerical integration of the underlying system. Here, we propose the Gradient Matching GTS (GMGTS) method as an efficient alternative to GTS. Gradient matching offers an integration-free approach to parameter estimation that is particularly powerful for dynamical systems that are linear in the unknown parameters, such as biochemical networks modeled by mass action kinetics. Here, we harness the power of gradient matching by integrating it into the GTS framework. To this end, we significantly expand the capabilities of gradient matching via uncertainty propagation calculations and the development of an iterative estimation scheme for partially observed systems. Through comparisons of GMGTS with GTS in different inference setups, we demonstrate that our method provides a significant computational advantage, thereby facilitating the use of complex NLME models in systems biology applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4dced58f5c5737541e67c9ad6ec12731d8224877" target='_blank'>
              Gradient matching accelerates mixed-effects inference for biochemical networks
              </a>
            </td>
          <td>
            Yulan B van Oppen, Andreas Milias-Argeitis
          </td>
          <td>2024-06-12</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Background: Drug safety relies on advanced methods for timely and accurate prediction of side effects. To tackle this requirement, this scoping review examines machine-learning approaches for predicting drug-related side effects with a particular focus on chemical, biological, and phenotypical features. Methods: This was a scoping review in which a comprehensive search was conducted in various databases from 1 January 2013 to 31 December 2023. Results: The results showed the widespread use of Random Forest, k-nearest neighbor, and support vector machine algorithms. Ensemble methods, particularly random forest, emphasized the significance of integrating chemical and biological features in predicting drug-related side effects. Conclusions: This review article emphasized the significance of considering a variety of features, datasets, and machine learning algorithms for predicting drug-related side effects. Ensemble methods and Random Forest showed the best performance and combining chemical and biological features improved prediction. The results suggested that machine learning techniques have some potential to improve drug development and trials. Future work should focus on specific feature types, selection techniques, and graph-based methods for even better prediction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b346d3270a77c7b4d467e797b36b5b738915e328" target='_blank'>
              Machine Learning Techniques for Predicting Drug-Related Side Effects: A Scoping Review
              </a>
            </td>
          <td>
            Esmaeel Toni, H. Ayatollahi, Reza Abbaszadeh, Alireza Fotuhi Siahpirani
          </td>
          <td>2024-06-01</td>
          <td>Pharmaceuticals</td>
          <td>0</td>
          <td>20</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a84901a0ed03e5a47effc5066f6f54e6de724c37" target='_blank'>
              piscesCSM: prediction of anticancer synergistic drug combinations.
              </a>
            </td>
          <td>
            Raghad Aljarf, Carlos H M Rodrigues, Yoochan Myung, D. Pires, David B. Ascher
          </td>
          <td>2024-07-19</td>
          <td>Journal of cheminformatics</td>
          <td>0</td>
          <td>33</td>
        </tr>

        <tr id="Graphs serve as generic tools to encode the underlying relational structure of data. Often this graph is not given, and so the task of inferring it from nodal observations becomes important. Traditional approaches formulate a convex inverse problem with a smoothness promoting objective and rely on iterative methods to obtain a solution. In supervised settings where graph labels are available, one can unroll and truncate these iterations into a deep network that is trained end-to-end. Such a network is parameter efficient and inherits inductive bias from the optimization formulation, an appealing aspect for data constrained settings in, e.g., medicine, finance, and the natural sciences. But typically such settings care equally about uncertainty over edge predictions, not just point estimates. Here we introduce novel iterations with independently interpretable parameters, i.e., parameters whose values - independent of other parameters' settings - proportionally influence characteristics of the estimated graph, such as edge sparsity. After unrolling these iterations, prior knowledge over such graph characteristics shape prior distributions over these independently interpretable network parameters to yield a Bayesian neural network (BNN) capable of graph structure learning (GSL) from smooth signal observations. Fast execution and parameter efficiency allow for high-fidelity posterior approximation via Markov Chain Monte Carlo (MCMC) and thus uncertainty quantification on edge predictions. Synthetic and real data experiments corroborate this model's ability to provide well-calibrated estimates of uncertainty, in test cases that include unveiling economic sector modular structure from S$\&$P$500$ data and recovering pairwise digit similarities from MNIST images. Overall, this framework enables GSL in modest-scale applications where uncertainty on the data structure is paramount.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d4a2a2d1e598080f8f15315512f0885aeba3c17a" target='_blank'>
              Graph Structure Learning with Interpretable Bayesian Neural Networks
              </a>
            </td>
          <td>
            Max Wasserman, Gonzalo Mateos
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/157e8eac3c1eeee1b8294dac26a9c49a1297cc1d" target='_blank'>
              NNICE: a deep quantile neural network algorithm for expression deconvolution
              </a>
            </td>
          <td>
            Y. Jin, Pingzhao Hu, Qian Liu
          </td>
          <td>2024-06-18</td>
          <td>Scientific Reports</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The high-dimensional genomic data presents significant challenges, and traditional analytical methods often struggle to capture the complex, non-linear relationships within these datasets. This study elaborates into the application of machine learning methods for dimensionality reduction and predictive modeling of binary phenotypes using gene expression data. Various dimensionality reduction techniques are explored, including t-distributed stochastic neighbor embedding (t-SNE), Non-negative matrix factorization (NMF), Principal component analysis (PCA), and manifold learning methods. Additionally, various algorithms such as logistic regression, random forests, support vector machines (SVMs)cand naive Bayes models are evaluated for predicting phenotypes. The study employs rigorous cross-validation, permutation testing, and evaluation metrics like the Matthews Correlation Coefficient (MCC) to assess model performance. The study rigorously assesses current genomics strategies, pinpointing their drawbacks and suggesting areas for future investigation, while delving into the potential of machine learning to overcome these hurdles and offer valuable insights in genomics. Keywords- Gene Expression , Dimensionality Reduction, Biomarkers, Cancer Prediction, Epigenetics, Machine Learning, Feature extraction.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/08bb904a8b352ae02dc78261d9490b90e729cf74" target='_blank'>
              Machine Learning Techniques and Hybrid Feature Selection Methods for Efficient Prediction of Cancer
              </a>
            </td>
          <td>
            Remyamol Km
          </td>
          <td>2024-05-28</td>
          <td>INTERANTIONAL JOURNAL OF SCIENTIFIC RESEARCH IN ENGINEERING AND MANAGEMENT</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="As an emerging interpretable technique, Generalized Additive Models (GAMs) adopt neural networks to individually learn non-linear functions for each feature, which are then combined through a linear model for final predictions. Although GAMs can explain deep neural networks (DNNs) at the feature level, they require large numbers of model parameters and are prone to overfitting, making them hard to train and scale. Additionally, in real-world datasets with many features, the interpretability of feature-based explanations diminishes for humans. To tackle these issues, recent research has shifted towards concept-based interpretable methods. These approaches try to integrate concept learning as an intermediate step before making predictions, explaining the predictions in terms of human-understandable concepts. However, these methods require domain experts to extensively label concepts with relevant names and their ground-truth values. In response, we propose CAT, a novel interpretable Concept-bAsed Taylor additive model to simply this process. CAT does not have to require domain experts to annotate concepts and their ground-truth values. Instead, it only requires users to simply categorize input features into broad groups, which can be easily accomplished through a quick metadata review. Specifically, CAT first embeds each group of input features into one-dimensional high-level concept representation, and then feeds the concept representations into a new white-box Taylor Neural Network (TaylorNet). The TaylorNet aims to learn the non-linear relationship between the inputs and outputs using polynomials. Evaluation results across multiple benchmarks demonstrate that CAT can outperform or compete with the baselines while reducing the need of extensive model parameters. Importantly, it can explain model predictions through high-level concepts that human can understand.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d130c609c3474815b03ab0023238576d0a1fe755" target='_blank'>
              CAT: Interpretable Concept-based Taylor Additive Models
              </a>
            </td>
          <td>
            Viet Duong, Qiong Wu, Zhengyi Zhou, Hongjue Zhao, Chenxiang Luo, Eric Zavesky, Huaxiu Yao, Huajie Shao
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We present a mathematical model of the evolutionary dynamics of a metastatic tumour under chemotherapy, comprising non-local partial differential equations for the phenotype-structured cell populations in the primary tumour and its metastasis. These equations are coupled with a physiologically-based pharmacokinetic model of drug delivery, implementing a realistic delivery schedule. The model is carefully calibrated from the literature, focusing on BRAF-mutated melanoma treated with Dabrafenib as a case study. By means of long-time asymptotic analysis, global sensitivity analysis and numerical simulations, we explore the impact of cell migration from the primary to the metastatic site, physiological aspects of the tumour sites and drug dose on the development of drug resistance and treatment efficacy. Our findings provide a possible explanation for empirical evidence indicating that chemotherapy may foster metastatic spread and that metastatic sites may be less impacted by chemotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/75caa1850e37b71799c05e2fad947723dcd9cdaf" target='_blank'>
              The development of drug resistance in metastatic tumours under chemotherapy: an evolutionary perspective
              </a>
            </td>
          <td>
            Federica Padovano, Chiara Villa
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Joint analysis of transcriptomic and T cell receptor (TCR) features at single-cell resolution provides a powerful approach for in-depth T cell immune function research. Here, we introduce a deep learning framework for single-T cell transcriptome and receptor analysis, MIST (Multi-Insight for T cell). MIST features three latent spaces: gene expression, TCR, and a joint latent space. Through analyses of antigen- specific T cells and T cells related to lung cancer immunotherapy, we demonstrate MIST’s interpretability and flexibility. MIST easily and accurately resolves cell function and antigen-specificity by vectorizing and integrating transcriptome and TCR data of T cells. In addition, using MIST, we identified the heterogeneity of CXCL13+ subsets in lung cancer infiltrating CD8+ T cells and their association with immunotherapy, providing additional insights into the functional transition of CXCL13+ T cells related to anti-PD-1 therapy that were not reported in the original study. MIST is available at https://github.com/aapupu/MIST.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e4cb87f8bfc27d222d60016bd4d339891023ace7" target='_blank'>
              MIST: an interpretable and flexible deep learning framework for single-T cell transcriptome and receptor analysis
              </a>
            </td>
          <td>
            Wenpu Lai, Yangqiu Li, Oscar Junhong Luo
          </td>
          <td>2024-07-13</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recently, Conditional Neural Fields (NeFs) have emerged as a powerful modelling paradigm for PDEs, by learning solutions as flows in the latent space of the Conditional NeF. Although benefiting from favourable properties of NeFs such as grid-agnosticity and space-time-continuous dynamics modelling, this approach limits the ability to impose known constraints of the PDE on the solutions -- e.g. symmetries or boundary conditions -- in favour of modelling flexibility. Instead, we propose a space-time continuous NeF-based solving framework that - by preserving geometric information in the latent space - respects known symmetries of the PDE. We show that modelling solutions as flows of pointclouds over the group of interest $G$ improves generalization and data-efficiency. We validated that our framework readily generalizes to unseen spatial and temporal locations, as well as geometric transformations of the initial conditions - where other NeF-based PDE forecasting methods fail - and improve over baselines in a number of challenging geometries.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c8295fdfb6de078027cabc7d49b319df8abb0ee" target='_blank'>
              Space-Time Continuous PDE Forecasting using Equivariant Neural Fields
              </a>
            </td>
          <td>
            David M. Knigge, David R. Wessels, Riccardo Valperga, Samuele Papa, J. Sonke, E. Gavves, E. J. Bekkers
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>56</td>
        </tr>

        <tr id="The obscure drug response continues to be a limiting factor for accurate cures for cancer. Next generation sequencing technologies have propelled the pharmacogenomic studies with characterized large panels of cancer cell line at multi‐omics level. However, the sufficient integration of the multi‐omics data and the efficient prediction for drug response and synergy still remain a challenge. To address these problems, ECFD is designed, an ensemble cascade forest‐based framework that predicts drug response and synergy using five types of omics data. Experimental results show the significant advantages of the ECFD model over existing models. The best integration of feature extraction is determined and the superiorities of robust stability in the face of new and small samples are highlighted. In addition, the methodological framework highlights the explainability of the model, the mechanisms of drug resistance and drug combination treatment strategies based on explainable analyses and biological networks. In sum, ECFD may facilitate the evaluation of drug response and speculation of potential synergy therapies in personalized and precision treatment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/202be07a703e58d34b9d528b6a979074db9400b4" target='_blank'>
              An Ensemble Cascade Forest‐Based Framework for Multi‐Omics Drug Response and Synergy Prediction
              </a>
            </td>
          <td>
            Ruijiang Li, Binsheng Sui, Dongjing Leng, Song He, Kunhong Liu, Xiaochen Bo
          </td>
          <td>2024-06-09</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Learning linear and nonlinear dynamical systems from available data is a timely topic in scientific machine learning. Learning must be performed while enforcing the numerical stability of the learned model, the existing knowledge within an informed or augmented setting, or by taking into account the multiscale dynamics—for both linear and nonlinear dynamics. However, when the final objective of such a learned dynamical system is to be used for control purposes, learning transformed dynamics can be advantageous. Therefore, many alternatives exists, and the present paper focuses on two of them: the first based on the discovery and use of the so-called flat control and the second one based on the use of the Koopman theory. The main contributions when addressing the first is the discovery of the flat output transformation by using an original neural framework. Moreover, when using the Koopman theory, this paper proposes an original procedure for learning parametric dynamics in the latent space, which is of particular interest in control-based engineering applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bde1af4107f9ee8f6740e26e9c5d03b288c163cb" target='_blank'>
              Learning Transformed Dynamics for Efficient Control Purposes
              </a>
            </td>
          <td>
            C. Ghnatios, Joel Mouterde, Jerome Tomezyk, Joaquim Da Silva, Francisco Chinesta
          </td>
          <td>2024-07-19</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Identifying cell types and states remains a time-consuming, error-prone challenge for spatial biology. While deep learning is increasingly used, it is difficult to generalize due to variability at the level of cells, neighborhoods, and niches in health and disease. To address this, we developed TACIT, an unsupervised algorithm for cell annotation using predefined signatures that operates without training data. TACIT uses unbiased thresholding to distinguish positive cells from background, focusing on relevant markers to identify ambiguous cells in multiomic assays. Using five datasets (5,000,000-cells; 51-cell types) from three niches (brain, intestine, gland), TACIT outperformed existing unsupervised methods in accuracy and scalability. Integrating TACIT-identified cell types with a novel Shiny app revealed new phenotypes in two inflammatory gland diseases. Finally, using combined spatial transcriptomics and proteomics, we discovered under- and overrepresented immune cell types and states in regions of interest, suggesting multimodality is essential for translating spatial biology to clinical applications.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cb5295d8599f76ce0d6396791698b01a3cf30271" target='_blank'>
              Spatial Deconvolution of Cell Types and Cell States at Scale Utilizing TACIT
              </a>
            </td>
          <td>
            Jinze Liu, Khoa Huynh, Katarzyna Tyc, B. Matuck, Quinn T. Easter, Aditya Pratapa, Nikhil V. Kumar, Paola Pérez, Rachel Kulchar, T. Pranzatelli, Deiziane Souza, Theresa Weaver, Xufeng Qu, Luiz Soares Junior, Marisa Dolhnokoff, David E. Kleiner, Stephen Hewitt, Luiz Fernando Ferraz da Silva, V. Rocha, B.M. Warner, K. Byrd
          </td>
          <td>2024-06-27</td>
          <td>Research Square</td>
          <td>0</td>
          <td>8</td>
        </tr>

        <tr id="Understanding transcriptional heterogeneity in cancer cells and its implication for treatment response is critical to identify how resistance occurs and may be targeted. Such heterogeneity can be captured by in vitro studies through clonal barcoding methods. We present TraCSED (Transformer-based modeling of Clonal Selection and Expression Dynamics), a dynamic deep learning approach for modeling clonal selection. Using single-cell gene expression and the fitness of barcoded clones, TraCSED identifies interpretable gene programs and the timepoints at which they are associated with clonal selection. When applied to cells treated with either giredestrant, an estrogen receptor (ER) antagonist and degrader, or palbociclib, a CDK4/6 inhibitor, time-dependent resistance pathways are revealed. For example, ER activity is associated with positive selection around day four under palbociclib treatment and this adaptive response can be suppressed by combining the drugs. Yet, in the combination treatment, one clone still emerged. Clustering based on partial least squares regression found that high baseline expression of both SNHG25 and SNCG genes was the primary marker of positive selection to co-treatment and thus potentially associated with innate resistance – an aspect that traditional differential analysis methods missed. In conclusion, TraCSED enables associating pathways with phenotypes in a time-dependent manner from scRNA-seq data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/cdb809968ef629bdd6c8b90e0032f40dbb52452b" target='_blank'>
              Transformer-based modeling of Clonal Selection and Expression Dynamics (TraCSED) reveals resistance signatures in breast cancer
              </a>
            </td>
          <td>
            Nathan Maulding, Jun Zou, Wei Zhou, Ciara Metcalfe, Josh Stuart, Xin Ye, Marc Hafner
          </td>
          <td>2024-06-19</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Virtual Screening (VS) of vast compound libraries guided by Artificial Intelligence (AI) models is a highly productive approach to early drug discovery. Data splitting is crucial for better benchmarking of such AI models. Traditional random data splits produce similar molecules between training and test sets, conflicting with the reality of VS libraries which mostly contain structurally distinct compounds. Scaffold split, grouping molecules by shared core structure, is widely considered to reflect this real-world scenario. However, here we show that the scaffold split also overestimates VS performance. The reason is that molecules with different chemical scaffolds are often similar, which hence introduces unrealistically high similarities between training molecules and test molecules following a scaffold split. Our study examined three representative AI models on 60 NCI-60 datasets, each with approximately 30,000 to 50,000 molecules tested on a different cancer cell line. Each dataset was split with three methods: scaffold, Butina clustering and the more accurate Uniform Manifold Approximation and Projection (UMAP) clustering. Regardless of the model, model performance is much worse with UMAP splits from the results of the 2100 models trained and evaluated for each algorithm and split. These robust results demonstrate the need for more realistic data splits to tune, compare, and select models for VS. For the same reason, avoiding the scaffold split is also recommended for other molecular property prediction problems. The code to reproduce these results is available at https://github.com/ScaffoldSplitsOverestimateVS">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6865e8bd8d18d531ebe4c835b07204534ccf27bc" target='_blank'>
              Scaffold Splits Overestimate Virtual Screening Performance
              </a>
            </td>
          <td>
            Qianrong Guo, Saiveth Hernández-Hernández, Pedro J. Ballester
          </td>
          <td>2024-06-02</td>
          <td>ArXiv</td>
          <td>2</td>
          <td>3</td>
        </tr>

        <tr id="Accumulation models, where a system progressively acquires binary features over time, are common in the study of cancer progression, evolutionary biology, and other fields. Many approaches have been developed to infer the accumulation pathways by which features (for example, mutations) are acquired over time. However, most of these approaches do not support reversibility: the loss of a feature once it has been acquired (for example, the clearing of a mutation from a tumour or population). Here, we demonstrate how the well-established Mk model from evolutionary biology, embedded on a hypercubic transition graph, can be used to infer the dynamics of accumulation processes, including the possibility of reversible transitions, from data which may be uncertain and cross-sectional, longitudinal, or phylogenetically / phylogenomically embedded. Positive and negative interactions between arbitrary sets of features (not limited to pairwise interactions) are supported. We demonstrate this approach with synthetic datasets and real data on bacterial drug resistance and cancer progression. While this implementation is limited in the number of features that can be considered, we discuss how this limitation may be relaxed to deal with larger systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/096bc6a723931c9c16f263760f1c2f885bca8b3c" target='_blank'>
              A hypercubic Mk model framework for capturing reversibility in disease, cancer, and evolutionary accumulation modelling
              </a>
            </td>
          <td>
            Iain G. Johnston, Ramon Diaz-Uriarte
          </td>
          <td>2024-07-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Black box deep learning models trained on genomic sequences excel at predicting the outcomes of different gene regulatory mechanisms. Therefore, interpreting these models may provide novel insights into the underlying biology, supporting downstream biomedical applications. Due to their complexity, interpretable surrogate models can only be built for local explanations (e.g., a single instance). However, accomplishing this requires generating a dataset in the neighborhood of the input, which must maintain syntactic similarity to the original data while introducing semantic variability in the model's predictions. This task is challenging due to the complex sequence-to-function relationship of DNA. We propose using Genetic Programming to generate datasets by evolving perturbations in sequences that contribute to their semantic diversity. Our custom, domain-guided individual representation effectively constrains syntactic similarity, and we provide two alternative fitness functions that promote diversity with no computational effort. Applied to the RNA splicing domain, our approach quickly achieves good diversity and significantly outperforms a random baseline in exploring the search space, as shown by our proof-of-concept, short RNA sequence. Furthermore, we assess its generalizability and demonstrate scalability to larger sequences, resulting in a ~30% improvement over the baseline.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ef43bb7883ab3d6b68f8e2d282f90a2d9dd3b4ff" target='_blank'>
              Semantically Rich Local Dataset Generation for Explainable AI in Genomics
              </a>
            </td>
          <td>
            Pedro Barbosa, Rosina Savisaar, Alcides Fonseca
          </td>
          <td>2024-07-03</td>
          <td>Proceedings of the Genetic and Evolutionary Computation Conference</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Numerous Pre-trained cell foundation models (CFM) have been crafted to encapsulate the comprehensive gene-gene interaction network within cells, leveraging extensive single-cell sequencing data. These models have shown promise in various cell biology applications, including cell type annotation, perturbation inference, and cell state embedding, etc. However, their clinical utility, particularly in cancer diagnosis and prognosis, remains an open question. We introduce the GeneBag model, a novel CFM that represents a cell as “a bag of unordered genes” with continuous expression values and a full-length gene list. Pre-trained on single-cell data and fine-tuned on bulk RNA-seq datasets, GeneBag achieves superior performance across cancer diagnosis and prognosis scenarios. In a zero-shot learning setting, GeneBag can classify cancer and non-cancer tissues with approximately 96.2% accuracy. With fine-tuning, it can annotate 40 different types of cancers and corresponding normal biopsies with an overall accuracy of ∼97.2%. It notably excels in classifying challenging cancers such as bladder (93%) and stomach (90%). Furthermore, GeneBag is capable of cancer staging with 68.5% accuracy and 5-year survival prediction with an AUC of ∼80.4%. This study marks the first to demonstrate the potential of CFMs in RNA-based cancer diagnostics and prognostics, indicating a promising avenue for AI-assisted molecular diagnosis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1eb775112abc591a1a520cde17c597775a658b0b" target='_blank'>
              GeneBag: training a cell foundation model for broad-spectrum cancer diagnosis and prognosis with bulk RNA-seq data
              </a>
            </td>
          <td>
            Yu-ping Liang, Dan Li, Aguix Guohua Xu, Yan Shao, Kun Tang
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The identification of a mathematical dynamics model is a crucial step in the designing process of a controller. However, it is often very difficult to identify the system's governing equations, especially in complex environments that combine physical laws of different disciplines. In this paper, we present a new approach that allows identifying an ordinary differential equation by means of a physics-informed machine learning algorithm. Our method introduces a special neural network that allows exploiting prior human knowledge to a certain degree and extends it autonomously, so that the resulting differential equations describe the system as accurately as possible. We validate the method on a Duffing oscillator with simulation data and, additionally, on a cascaded tank example with real-world data. Subsequently, we use the developed algorithm in a model-based reinforcement learning framework by alternately identifying and controlling a system to a target state. We test the performance by swinging-up an inverted pendulum on a cart.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9fb84f2c4de6cf8d43f1207b6c58095273d711f1" target='_blank'>
              Identifying Ordinary Differential Equations for Data-efficient Model-based Reinforcement Learning
              </a>
            </td>
          <td>
            Tobias Nagel, Marco F. Huber
          </td>
          <td>2024-06-28</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f77989353f88a88cfddf73a76fa518dbebb03ec3" target='_blank'>
              GATSol, an enhanced predictor of protein solubility through the synergy of 3D structure graph and large language modeling
              </a>
            </td>
          <td>
            Bin Li, Dengming Ming
          </td>
          <td>2024-06-01</td>
          <td>BMC Bioinformatics</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Recent advances in machine learning methods for materials science have significantly enhanced accurate predictions of the properties of novel materials. Here, we explore whether these advances can be adapted to drug discovery by addressing the problem of prospective validation - the assessment of the performance of a method on out-of-distribution data. First, we tested whether k-fold n-step forward cross-validation could improve the accuracy of out-of-distribution small molecule bioactivity predictions. We found that it is more helpful than conventional random split cross-validation in describing the accuracy of a model in real-world drug discovery settings. We also analyzed discovery yield and novelty error, finding that these two metrics provide an understanding of the applicability domain of models and an assessment of their ability to predict molecules with desirable bioactivity compared to other small molecules. Based on these results, we recommend incorporating a k-fold n-step forward cross-validation and these metrics when building state-of-the-art models for bioactivity prediction in drug discovery.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/38cd10dafeaecc8755eed9192a85a6e2483eab04" target='_blank'>
              Step Forward Cross Validation for Bioactivity Prediction: Out of Distribution Validation in Drug Discovery
              </a>
            </td>
          <td>
            Udit Surya Saha, Michele Vendruscolo, A. Carpenter, Shantanu Singh, Andreas Bender, Srijit Seal
          </td>
          <td>2024-07-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="With the prevailing efforts to combat the coronavirus disease 2019 (COVID-19) pandemic, there are still uncertainties that are yet to be discovered about its spread, future impact, and resurgence. In this paper, we present a three-stage data-driven approach to distill the hidden information about COVID-19. The first stage employs a Bayesian network structure learning method to identify the causal relationships among COVID-19 symptoms and their intrinsic demographic variables. As a second stage, the output from the Bayesian network structure learning, serves as a useful guide to train an unsupervised machine learning (ML) algorithm that uncovers the similarities in patients' symptoms through clustering. The final stage then leverages the labels obtained from clustering to train a demographic symptom identification (DSID) model which predicts a patient's symptom class and the corresponding demographic probability distribution. We applied our method on the COVID-19 dataset obtained from the Centers for Disease Control and Prevention (CDC) in the United States. Results from the experiments show a testing accuracy of 99.99%, as against the 41.15% accuracy of a heuristic ML method. This strongly reveals the viability of our Bayesian network and ML approach in understanding the relationship between the virus symptoms, and providing insights on patients' stratification towards reducing the severity of the virus.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/262096c542d3af917b7d728e1e38182b018205e8" target='_blank'>
              Bayesian Networks and Machine Learning for COVID-19 Severity Explanation and Demographic Symptom Classification
              </a>
            </td>
          <td>
            O. Ajayi, Yu Cheng
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="The integration of Graph Neural Networks (GNNs) and Neural Ordinary and Partial Differential Equations has been extensively studied in recent years. GNN architectures powered by neural differential equations allow us to reason about their behavior, and develop GNNs with desired properties such as controlled smoothing or energy conservation. In this paper we take inspiration from Turing instabilities in a Reaction Diffusion (RD) system of partial differential equations, and propose a novel family of GNNs based on neural RD systems. We \textcolor{black}{demonstrate} that our RDGNN is powerful for the modeling of various data types, from homophilic, to heterophilic, and spatio-temporal datasets. We discuss the theoretical properties of our RDGNN, its implementation, and show that it improves or offers competitive performance to state-of-the-art methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57254c8083a6fe4f39ea4b61b190c97bb047ab60" target='_blank'>
              Graph Neural Reaction Diffusion Models
              </a>
            </td>
          <td>
            Moshe Eliasof, Eldad Haber, Eran Treister
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>1</td>
          <td>15</td>
        </tr>

        <tr id="Supervised machine learning models rely on training datasets with positive (target class) and negative examples. Therefore, the composition of the training dataset has a direct influence on model performance. Specifically, negative sample selection bias, concerning samples not representing the target class, presents challenges across a range of domains such as text classification and protein-protein interaction prediction. Machine-learning-based immunotherapeutics design is an increasingly important area of research, focusing on designing antibodies or T-cell receptors (TCRs) that can bind to their target molecules with high specificity and affinity. Given the biomedical importance of immunotherapeutics, there is a need to address the unresolved question of how negative training set composition impacts model generalization and biological rule discovery to enable rational and safe drug design. We set out to study this question in the context of the antibody-antigen binding prediction problem by varying the negative class, encompassing a binding affinity gradient. We based our investigation on large synthetic datasets that provide ground truth structure-based antibody-antigen binding data, allowing access to residue-wise binding energy on the binding interface. We found that both out-of-distribution generalization and binding rule discovery depended on the type of negative dataset used. Importantly, we discovered that a model’s capacity to learn the binding rules of the positive dataset is not a trivial correlate of its classification accuracy. We confirmed our findings with real-world relevant experimental data. Our work highlights the importance of considering training dataset composition for achieving optimal out-of-distribution performance and rule learning in machine-learning-based research. Significance Statement The effectiveness of supervised machine learning models hinges on the composition of their training datasets, particularly the inclusion of negative examples. This bias in negative sample selection can greatly impact model performance. As the development of immunotherapeutic agents using machine learning is becoming increasingly crucial in biomedicine, understanding the impact of negative training set composition is imperative. Our study, focused on the antibody-antigen binding prediction problem, reveals that the choice of negative dataset significantly affects both out-of-distribution generalization and binding rule discovery across synthetic and experimental data. These findings underscore the necessity of carefully considering training dataset composition in machine-learning-driven biomedical research for optimal performance, robustness and meaningful rule acquisition.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23c2f1758056f3fe1173a00d561a41e8889e3482" target='_blank'>
              Training data composition determines machine learning generalization and biological rule discovery
              </a>
            </td>
          <td>
            E. Ursu, Aygul Minnegalieva, Puneet Rawat, Maria Chernigovskaya, Robi Tacutu, G. K. Sandve, Philippe A. Robert, Victor Greiff
          </td>
          <td>2024-06-23</td>
          <td>bioRxiv</td>
          <td>1</td>
          <td>28</td>
        </tr>

        <tr id="Cancers are commonly characterised by a complex pathology encompassing genetic, microscopic and macroscopic features, which can be probed individually using imaging and omics technologies. Integrating this data to obtain a full understanding of pathology remains challenging. We introduce a new method called Deep Latent Variable Path Modelling (DLVPM), which combines the representational power of deep learning with the capacity of path modelling to identify relationships between interacting elements in a complex system. To evaluate the capabilities of DLVPM, we initially trained a foundational model to map dependencies between SNV, Methylation, miRNA-Seq, RNA-Seq and Histological data using Breast Cancer data from The Cancer Genome Atlas (TCGA). This method exhibited superior performance in mapping associations between data types compared to classical path modelling. We additionally performed successful applications of the model to: stratify single-cell data, identify synthetic lethal interactions using CRISPR-Cas9 screens derived from cell-lines, and detect histologic-transcriptional associations using spatial transcriptomic data. Results from each of these data types can then be understood with reference to the same holistic model of illness.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36a969a03ce741afe8ca32855f97d3ec725e814d" target='_blank'>
              Integrating Multi-Modal Cancer Data Using Deep Latent Variable Path Modelling
              </a>
            </td>
          <td>
            Alex Ing, Alvaro Andrades, M. R. Cosenza, J. Korbel
          </td>
          <td>2024-06-16</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>95</td>
        </tr>

        <tr id="8614 Background: Advanced lung adenocarcinoma (aLUAD) is divided into multiple molecularly defined subsets, each with specific biological characteristics and responses to targeted therapy. Consequently, conducting practice-changing and cost-effective randomized trials is challenging. In silico clinical trials, utilizing mechanistic computational models, offer a potential solution, yet their ability to predict clinical trial results has not been proven. Methods: We updated a previously published version of the computational mechanistic model of EGFR-mutated aLUAD, notably by incorporating metastases. Disease model was integrated with physiologically based pharmacokinetic (PBPK) models of cisplatin, pemetrexed, and osimertinib, offering a more comprehensive understanding of the disease's dynamics. These models are founded on causal connections between biological entities and phenomena, establishing a link between the specific biological behaviors of EGFR-mutated aLUAD and the evolution of tumor size and progression over time. We have independently and prospectively simulated the outcomes of the FLAURA2 and MARIPOSA randomized trials dedicated to EGFR-mutated aLUAD. This was achieved by replicating the experimental protocols and generating cohorts of virtual twins of patients using publicly available data. Results: In silico results were released before the actual outcomes of the FLAURA2 and MARIPOSA trials were publicly available. The simulations yielded predictions with overlapping confidence intervals, similar hazard ratios, median survivals, and shapes of curves compared to the actual trial data. Bootstrapped weighted log-rank tests were used to compare the simulated and observed Kaplan-Meier curves for the two arms of FLAURA2 and for the osimertinib arm of MARIPOSA. Between 94 and 98% of the tests were statistically non-significant (α=0.05), confirming the accuracy of the predictions. Conclusions: This first-of-its-kind prediction of clinical trial outcomes demonstrates that in silico trials, when based on robust mechanistic models, can be a reliable tool for enhancing the design of actual trials, particularly for EGFR-mutated aLUAD. They offer a promising avenue for overcoming the challenges of comparator arms in single-arm trials and may serve as a new standard for formulating statistical hypotheses in future studies.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c76086ed48f1470fe5129435b0503e60cd9f7700" target='_blank'>
              Prospective validation of in silico clinical trials prediction using an EGFR-mutated NSCLC mechanistic computational disease model.
              </a>
            </td>
          <td>
            Michaël Duruisseaux, P. Masson, Arnaud Nativel, Nicolas Girard, J. Cadranel, A. Swalduz, Matthieu Coudron, Guillaume Bouchard, R. Kahoul, Evgueni Jacob, J. Bosley, J. Palgen, A. L’Hostis, C. Monteiro
          </td>
          <td>2024-06-01</td>
          <td>Journal of Clinical Oncology</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="Many causal systems such as biological processes in cells can only be observed indirectly via measurements, such as gene expression. Causal representation learning -- the task of correctly mapping low-level observations to latent causal variables -- could advance scientific understanding by enabling inference of latent variables such as pathway activation. In this paper, we develop methods for inferring latent variables from multiple related datasets (environments) and tasks. As a running example, we consider the task of predicting a phenotype from gene expression, where we often collect data from multiple cell types or organisms that are related in known ways. The key insight is that the mapping from latent variables driven by gene expression to the phenotype of interest changes sparsely across closely related environments. To model sparse changes, we introduce Tree-Based Regularization (TBR), an objective that minimizes both prediction error and regularizes closely related environments to learn similar predictors. We prove that under assumptions about the degree of sparse changes, TBR identifies the true latent variables up to some simple transformations. We evaluate the theory empirically with both simulations and ground-truth gene expression data. We find that TBR recovers the latent causal variables better than related methods across these settings, even under settings that violate some assumptions of the theory.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/828daf97066cdcac9612888fea4854de9c05ee25" target='_blank'>
              Sparsity regularization via tree-structured environments for disentangled representations
              </a>
            </td>
          <td>
            Elliot Layne, Dhanya Sridhar, Jason S. Hartford, M. Blanchette
          </td>
          <td>2024-05-30</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>50</td>
        </tr>

        <tr id="Understanding and forecasting infectious disease spread is pivotal for effective public health management. Traditional dynamic disease modeling is an essential tool for characterization and prediction, but often requires extensive expertise and specialized software, which may not be readily available in low-resource environments. To address these challenges, we introduce an AI-powered modeling assistant that utilizes advanced capabilities from OpenAI's latest models and functionality. This tool enhances the accessibility and usability of infectious disease models and simulation frameworks by allowing users to generate or modify model configurations through intuitive natural language inputs or by importing explicit model descriptions. Our prototype integrates with an established open-source disease simulation framework called the Compartmental Modeling Software (CMS) to provide a seamless modeling experience from setup to analysis. The AI assistant efficiently interprets disease model parameters, constructs accurate model files, executes simulations in a controlled environment, and assists in result interpretation using advanced analytics tools. It encapsulates expert knowledge and adheres to best practices to support users ranging from novices to expert modelers. Furthermore, we discuss the limitations of this AI assistant, particularly its performance in complex scenarios where it might generate inaccurate specifications. By enhancing the ease of disease modeling and supporting ongoing capacity-building initiatives, we believe that AI assistants like this one could significantly contribute to global health efforts by empowering researchers, especially in regions with limited resources, to develop and refine their disease models independently. This innovative approach has the potential to democratize disease modeling in global health, offering a scalable solution that adapts to diverse needs across a wide-range of geographies, languages, and populations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4369b57b079189f7be357907512cabf00ac5b4b1" target='_blank'>
              Democratizing Infectious Disease Modeling: An AI Assistant for Generating, Simulating, and Analyzing Dynamic Models
              </a>
            </td>
          <td>
            Joshua L. Proctor, Guillaume Chabot-Couture
          </td>
          <td>2024-07-17</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Latent dynamical systems have been widely used to characterize the dynamics of neural population activity in the brain. However, these models typically ignore the fact that the brain contains multiple cell types. This limits their ability to capture the functional roles of distinct cell classes, or to accurately predict the effects of cell-specific optogenetic perturbations on neural activity or behavior. To overcome these limitations, we introduce the “cell-type dynamical systems” (CTDS) model. This model extends latent linear dynamical systems to contain distinct latent variables for each cell class, with biologically inspired constraints on both dynamics and emissions. To illustrate our approach, we consider neural recordings with distinct excitatory (E) and inhibitory (I) populations. The CTDS model defines separate latents for E and I cells, and constrains the dynamics so that E (I) latents have a strictly positive (negative) effects on other latents. We applied CTDS to recordings from rat frontal orienting fields (FOF) and anterior dorsal striatum (ADS) during an auditory decision-making task. The model achieved higher accuracy than a standard linear dynamical system (LDS), and revealed that both E and I latents could be used to decode the animal’s choice, showing that choice-related information is not restricted to a single cell class. We also performed in-silico optogenetic perturbation experiments in the FOF and ADS, and found that CTDS was able to replicate the causal effects of different perturbations on behavior, whereas a standard LDS model—which lacks the ability to capture cell-specific perturbations—did not. Crucially, our model allowed us to understand the effects of these perturbations by revealing the dynamics of different cell-specific latents. Finally, CTDS can also be used to identify cell types for neurons whose class labels are unknown in electrophysiological recordings. These results illustrate the power of the CTDS model to provide more accurate and more biologically interpretable descriptions of neural population dynamics and their relationship to behavior.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/31c1cd3d2f0ffab7d6fe212ee7ca7fdd655d8dfa" target='_blank'>
              Disentangling the Roles of Distinct Cell Classes with Cell-Type Dynamical Systems
              </a>
            </td>
          <td>
            Aditi Jha, Diksha Gupta, Carlos D. Brody, Jonathan W. Pillow
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="
 Connectome-based models, also known as Virtual Brain Models (VBMs), have been well established in network neuroscience to investigate pathophysiological causes underlying a large range of brain diseases. The integration of an individual's brain imaging data in VBMs has improved patient-specific predictivity, although Bayesian estimation of spatially distributed parameters remains challenging even with state-of-the-art Monte Carlo sampling. VBMs imply latent nonlinear state space models driven by noise and network input, necessitating advanced probabilistic machine learning techniques for widely applicable Bayesian estimation. Here we present Simulation-based Inference on Virtual Brain Models (SBI-VBMs), and demonstrate that training deep neural networks on both spatio-temporal and functional features allows for accurate estimation of generative parameters in brain disorders. The systematic use of brain stimulation provides an effective remedy for the non-identifiability issue in estimating the degradation limited to smaller subset of connections. By prioritizing model structure over data, we show that the hierarchical structure in SBI-VBMs renders the inference more effective, precise and biologically plausible. This approach could broadly advance precision medicine by enabling fast and reliable prediction of patient-specific brain disorders.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/91a633b67282271ed1d985aa26404ebe177cca22" target='_blank'>
              Simulation-based Inference on Virtual Brain Models of Disorders
              </a>
            </td>
          <td>
            Meysam Hashemi, Abolfazl Ziaeemehr, M. Woodman, Jan Fousek, S. Petkoski, V. Jirsa
          </td>
          <td>2024-07-11</td>
          <td>Machine Learning: Science and Technology</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Intratumour phenotypic heterogeneity is nowadays understood to play a critical role in disease progression and treatment failure. Accordingly, there has been increasing interest in the development of mathematical models capable of capturing its role in cancer cell adaptation. This can be systematically achieved by means of models comprising phenotype-structured nonlocal partial differential equations, tracking the evolution of the phenotypic density distribution of the cell population, which may be compared to gene and protein expression distributions obtained experimentally. Nevertheless, given the high analytical and computational cost of solving these models, much is to be gained from reducing them to systems of ordinary differential equations for the moments of the distribution. We propose a generalised method of model-reduction, relying on the use of a moment generating function, Taylor series expansion and truncation closure, to reduce a nonlocal reaction-advection-diffusion equation, with general phenotypic drift and proliferation rate functions, to a system of moment equations up to arbitrary order. Our method extends previous results in the literature, which we address via two examples, by removing any \textit{a priori} assumption on the shape of the distribution, and provides a flexible framework for mathematical modellers to account for the role of phenotypic heterogeneity in cancer adaptive dynamics, in a simpler mathematical framework.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4d951ebfbeb9789f55de6ae13ba4fa4d0ddb0abc" target='_blank'>
              Reducing phenotype-structured PDE models of cancer evolution to systems of ODEs: a generalised moment dynamics approach
              </a>
            </td>
          <td>
            Chiara Villa, P. K. Maini, Alexander P Browning, A. Jenner, Sara Hamis, Tyler Cassidy
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>26</td>
        </tr>

        <tr id="In the present article, we propose a paradigm shift on evolving Artificial Neural Networks (ANNs) towards a new bio-inspired design that is grounded on the structural properties, interactions, and dynamics of protein networks (PNs): the Artificial Protein Network (APN). This introduces several advantages previously unrealized by state-of-the-art approaches in NE: (1) We can draw inspiration from how nature, thanks to millions of years of evolution, efficiently encodes protein interactions in the DNA to translate our APN to silicon DNA. This helps bridge the gap between syntax and semantics observed in current NE approaches. (2) We can learn from how nature builds networks in our genes, allowing us to design new and smarter networks through EA evolution. (3) We can perform EA crossover/mutation operations and evolution steps, replicating the operations observed in nature directly on the genotype of networks, thus exploring and exploiting the phenotypic space, such that we avoid getting trapped in sub-optimal solutions. (4) Our novel definition of APN opens new ways to leverage our knowledge about different living things and processes from biology. (5) Using biologically inspired encodings, we can model more complex demographic and ecological relationships (e.g., virus-host or predator-prey interactions), allowing us to optimise for multiple, often conflicting objectives.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ebdbe8cd6d06b80d544472cbb9efc805782d0782" target='_blank'>
              Protein pathways as a catalyst to directed evolution of the topology of artificial neural networks
              </a>
            </td>
          <td>
            Oscar Lao, Konstantinos Zacharopoulos, A. Fournaris, Rossano Schifanella, Ioannis Arapakis
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>27</td>
        </tr>

        <tr id="The prediction of time series is a challenging task relevant in such diverse applications as analyzing financial data, forecasting flow dynamics or understanding biological processes. Especially chaotic time series that depend on a long history pose an exceptionally difficult problem. While machine learning has shown to be a promising approach for predicting such time series, it either demands long training time and much training data when using deep recurrent neural networks. Alternative, when using a reservoir computing approach it comes with high uncertainty and typically a high number of random initializations and extensive hyper-parameter tuning when using a reservoir computing approach. In this paper, we focus on the reservoir computing approach and propose a new mapping of input data into the reservoir's state space. Furthermore, we incorporate this method in two novel network architectures increasing parallelizability, depth and predictive capabilities of the neural network while reducing the dependence on randomness. For the evaluation, we approximate a set of time series from the Mackey-Glass equation, inhabiting non-chaotic as well as chaotic behavior and compare our approaches in regard to their predictive capabilities to echo state networks and gated recurrent units. For the chaotic time series, we observe an error reduction of up to $85.45\%$ and up to $87.90\%$ in contrast to echo state networks and gated recurrent units respectively. Furthermore, we also observe tremendous improvements for non-chaotic time series of up to $99.99\%$ in contrast to existing approaches.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/d488897997df288927a73befa218f42726e3f36d" target='_blank'>
              Temporal Convolution Derived Multi-Layered Reservoir Computing
              </a>
            </td>
          <td>
            Johannes Viehweg, Dominik Walther, Prof. Dr.-Ing. Patrick Mader
          </td>
          <td>2024-07-09</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Co-folding models are the newest innovation in deep-learning-based protein-ligand structure prediction. The recent publications of RoseTTAFold All-Atom and AlphaFold 3 have shown high-quality results on predicting the structures of proteins interacting with small-molecules, other proteins and nucleic-acids. Despite these advanced capabilities and broad potential, the current study presents critical findings that question these models’ adherence to fundamental physical principles and its susceptibility to overfitting specific protein families. Through adversarial examples based on established physical, chemical, and biological principles, we demonstrate notable discrepancies in protein-ligand structural predictions when subjected to biologically plausible perturbations. These discrepancies reveal a significant divergence from expected physical behaviors, indicating potential overfitting to particular data subsets within its training corpus. Our findings underscore the models’ limitations in generalizing effectively across diverse biological structures and highlight the necessity of integrating robust physical and chemical priors in the development of such predictive tools. The results advocate a measured reliance on deep-learning-based models for critical applications in drug discovery and protein engineering, where a deep understanding of the underlying physical and chemical properties is crucial.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/246574ae80d8bdbec4e6397732952f7f393a2387" target='_blank'>
              Do Deep Learning Models for Co-Folding Learn the Physics of Protein-Ligand Interactions?
              </a>
            </td>
          <td>
            Matthew Masters, Amr H. Mahmoud, Markus A. Lill
          </td>
          <td>2024-06-04</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="Cancer is typically treated with combinatorial therapy, and such combinations may be synergistic. However, discovery of these combinations has proven difficult as brute force combinatorial screening approaches are both logistically complex and resource-intensive. Therefore, computational approaches to augment synergistic drug discovery are of interest, but current approaches are limited by their dependencies on combinatorial drug screening training data or molecular profiling data. These dataset dependencies can limit the number and diversity of drugs for which these approaches can make inferences. Herein, we describe a novel computational framework, ReCorDE (Recurrent Correlation of Drugs with Enrichment), that uses publicly-available cell line-derived monotherapy cytotoxicity datasets to identify drug classes targeting shared vulnerabilities across multiple cancer lineages; and we show how these inferences can be used to augment synergistic drug combination discovery. Additionally, we demonstrate in preclinical models that a drug class combination predicted by ReCorDE to target shared vulnerabilities (PARP inhibitors and Aurora kinase inhibitors) exhibits class-class synergy across lineages. ReCorDE functions independently of combinatorial drug screening and molecular profiling data, using only extensive monotherapy cytotoxicity datasets as its input. This allows ReCorDE to make robust inferences for a large, diverse array of drugs. In conclusion, we have described a novel framework for the identification of drug classes targeting shared vulnerabilities using monotherapy cytotoxicity datasets, and we showed how these inferences can be used to aid discovery of novel synergistic drug combinations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/30727e4fdcf06bbebec309952629b57642d3a2b5" target='_blank'>
              ReCorDE: a framework for identifying drug classes targeting shared vulnerabilities with applications to synergistic drug discovery
              </a>
            </td>
          <td>
            August J. John, Emily T. Ghose, Huanyao Gao, Meagan Luck, Dabin Jeong, Krishna R. Kalari, Liewei Wang
          </td>
          <td>2024-05-31</td>
          <td>Frontiers in Oncology</td>
          <td>0</td>
          <td>42</td>
        </tr>

        <tr id="Emerging infectious diseases and climate change are two of the major challenges in 21st century. Although over the past decades, highly-resolved mathematical models have contributed in understanding dynamics of infectious diseases and are of great aid when it comes to finding suitable intervention measures, they may need substantial computational effort and produce significant $CO_2$ emissions. Two popular modeling approaches for mitigating infectious disease dynamics are agent-based and differential equation-based models. Agent-based models (ABMs) offer an arbitrary level of detail and are thus able to capture heterogeneous human contact behavior and mobility patterns. However, insights on individual-level dynamics come with high computational effort that scales with the number of agents. On the other hand, (differential) equation-based models (EBMs) are computationally efficient even for large populations due to their complexity being independent of the population size. Yet, equation-based models are restricted in their granularity as they assume a well-mixed population. To manage the trade-off between complexity and detail, we propose spatial- and temporal-hybrid models that use agent-based models only in an area or time frame of interest. To account for relevant influences to disease dynamics, we use EBMs, only adding moderate computational costs. Our hybridization approach demonstrates significant reduction in computational effort by up to 98 % -- without losing the required depth in information in the focus frame. The hybrid models used in our numerical simulations are based on two recently proposed models, however, any suitable combination of ABM-EBM could be used, too. Concluding, hybrid epidemiological models can provide insights on the individual scale where necessary, using aggregated models where possible, thereby making an important contribution to green computing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/99d94ff9f5243a9bff742e71899a13aa3a159acb" target='_blank'>
              Hybrid metapopulation agent-based epidemiological models for efficient insight on the individual scale: a contribution to green computing
              </a>
            </td>
          <td>
            Julia Bicker, Ren´e Schmieding, Martin J. Kuhn
          </td>
          <td>2024-06-06</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Over half cancer patients lack safe, effective, targeted therapies despite abundant molecular profiling data. Statistically recurrent cancer drivers have provided fertile ground for drug discovery where they exist. But in rare, complex, and heterogeneous cancers, strong driver signals are elusive. Moreover, therapeutically exploitable molecular vulnerabilities extend beyond classical drivers. Here we describe a novel, integrative, generalizable graph-based, cooperativity-led Markov chain model, A3D3a’s MVP (Adaptive AI-Augmented Drug Discovery and Development Molecular Vulnerability Picker), to identify and prioritize key druggable molecular vulnerabilities in cancer. The algorithm exploits cooperativity of weak signals within a cancer molecular network to enhance the signal of true molecular vulnerabilities. We apply A3D3a’s MVP to 19 solid cancer types and demonstrate that it outperforms standard approaches for target hypothesis generation by >3-fold as benchmarked against cell line genetic perturbation and drug screening data. Importantly, we demonstrate its ability to identify non-driver druggable vulnerabilities and highlight 43 novel or emergent druggable targets for these tumors.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe5dec6f605a79a26e33a96eedcfe5b2b3c71dd8" target='_blank'>
              Probabilistic graph-based model uncovers previously unseen druggable vulnerabilities in major solid cancers
              </a>
            </td>
          <td>
            Ying Zhu, Stephanie T. Schmidt, Li Zhao, Chunjie Jiang, Patrizio Di Micco, C. Mitsopoulos, Andrew Futreal, B. Al-Lazikani
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>38</td>
        </tr>

        <tr id="The Cox regression model or accelerated failure time regression models are often used for describing the relationship between survival outcomes and potential explanatory variables. These models assume the studied covariates are connected to the survival time or its distribution or their transformations through a function of a linear regression form. In this article, we propose nonparametric, nonlinear algorithms (deepAFT methods) based on deep artificial neural networks to model survival outcome data in the broad distribution family of accelerated failure time models. The proposed methods predict survival outcomes directly and tackle the problem of censoring via an imputation algorithm as well as re-weighting and transformation techniques based on the inverse probabilities of censoring. Through extensive simulation studies, we confirm that the proposed deepAFT methods achieve accurate predictions. They outperform the existing regression models in prediction accuracy, while being flexible and robust in modeling covariate effects of various nonlinear forms. Their prediction performance is comparable to other established deep learning methods such as deepSurv and random survival forest methods. Even though the direct output is the expected survival time, the proposed AFT methods also provide predictions for distributional functions such as the cumulative hazard and survival functions without additional learning efforts. For situations where the popular Cox regression model may not be appropriate, the deepAFT methods provide useful and effective alternatives, as shown in simulations, and demonstrated in applications to a lymphoma clinical trial study.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6862622b0783ec03a8c5760315e34b81914feca4" target='_blank'>
              deepAFT: A nonlinear accelerated failure time model with artificial neural network.
              </a>
            </td>
          <td>
            Patrick A Norman, Wanlu Li, Wenyu Jiang, Bingshu E. Chen
          </td>
          <td>2024-06-18</td>
          <td>Statistics in medicine</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Deep learning approaches achieved significant progress in predicting protein structures. These methods are often applied to protein-protein interactions (PPIs) yet require Multiple Sequence Alignment (MSA) which is unavailable for various interactions, such as antibody-antigen. Computational docking methods are capable of sampling accurate complex models, but also produce thousands of invalid configurations. The design of scoring functions for identifying accurate models is a long-standing challenge. We develop a novel attention-based Graph Neural Network (GNN), ContactNet, for classifying PPI models obtained from docking algorithms into accurate and incorrect ones. When trained on docked antigen and modeled antibody structures, ContactNet doubles the accuracy of current state-of-the-art scoring functions, achieving accurate models among its Top-10 at 43% of the test cases. When applied to unbound antibodies, its Top-10 accuracy increases to 65%. This performance is achieved without MSA and the approach is applicable to other types of interactions, such as host-pathogens or general PPIs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c685fe4d28871577e834704f37c26c7f53fa0191" target='_blank'>
              ContactNet: Geometric-Based Deep Learning Model for Predicting Protein-Protein Interactions
              </a>
            </td>
          <td>
            Matan Halfon, Tomer Cohen, Raanan Fattal, D. Schneidman-Duhovny
          </td>
          <td>2024-06-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>43</td>
        </tr>

        <tr id="Deep learning has emerged as a powerful tool for phylodynamic analysis, addressing common computational limitations affecting existing methods. However, notable disparities exist between simulated phylogenetic trees used for training existing deep learning models and those derived from real-world sequence data, necessitating a thorough examination of their practicality. We conducted a comprehensive evaluation of model performance by assessing an existing deep learning inference tool for phylodynamics, PhyloDeep, against realistic phylogenetic trees characterized from SARS-CoV-2. Our study reveals the poor predictive accuracy of PhyloDeep models trained on simulated trees when applied to realistic data. Conversely, models trained on realistic trees demonstrate improved predictions, despite not being infallible, especially in scenarios where superspreading dynamics are challenging to capture accurately. Consequently, we find markedly improved performance through the integration of minimal contact tracing data. Applying this approach to a sample of SARS-CoV-2 sequences partially matched to contact tracing from Hong Kong yields informative estimates of SARS-CoV-2 superspreading potential beyond the scope of contact tracing data alone. Our findings demonstrate the potential for enhancing deep learning phylodynamic models processing low resolution trees through complementary data integration, ultimately increasing the precision of epidemiological predictions crucial for public health decision making and outbreak control.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/48c89a21622264ef2e2c8849ba79571131cf44dd" target='_blank'>
              Deep learning of SARS-CoV-2 outbreak phylodynamics with contact tracing data
              </a>
            </td>
          <td>
            Ruopeng Xie, D. Adam, Shu Hu, Benjamin J. Cowling, Olivier Gascuel, Anna Zhukova, V. Dhanasekaran, Kong S.A.R
          </td>
          <td>2024-06-11</td>
          <td>None</td>
          <td>0</td>
          <td>15</td>
        </tr>

        <tr id="Drug–target interactions underlie the actions of chemical substances in medicine. Moreover, drug repurposing can expand use profiles while reducing costs and development time by exploiting potential multi-functional pharmacological properties based upon additional target interactions. Nonetheless, drug repurposing relies on the accurate identification and validation of drug–target interactions (DTIs). In this study, a novel drug–target interaction prediction model was developed. The model, based on an interactive inference network, contains embedding, encoding, interaction, feature extraction, and output layers. In addition, this study used Morgan and PubChem molecular fingerprints as additional information for drug encoding. The interaction layer in our model simulates the drug–target interaction process, which assists in understanding the interaction by representing the interaction space. Our method achieves high levels of predictive performance, as well as interpretability of drug–target interactions. Additionally, we predicted and validated 22 Alzheimer’s disease-related targets, suggesting our model is robust and effective and thus may be beneficial for drug repurposing.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ce82b8dd369b4cc3b6dcccd4cba4293007b3585e" target='_blank'>
              Drug–Target Interaction Prediction Based on an Interactive Inference Network
              </a>
            </td>
          <td>
            Yuqi Chen, Xiaomin Liang, Wei Du, Yanchun Liang, Garry Wong, Liang Chen
          </td>
          <td>2024-07-15</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Behaviour change lies at the heart of many observable collective phenomena such as the transmission and control of infectious diseases, adoption of public health policies, and migration of animals to new habitats. Representing the process of individual behaviour change in computer simulations of these phenomena remains an open challenge. Often, computational models use phenomenological implementations with limited support from behavioural data. Without a strong connection to observable quantities, such models have limited utility for simulating observed and counterfactual scenarios of emergent phenomena because they cannot be validated or calibrated. Here, we present a simple stochastic individual-based model of reversal learning that captures fundamental properties of individual behaviour change, namely, the capacity to learn based on accumulated reward signals, and the transient persistence of learned behaviour after rewards are removed or altered. The model has only two parameters, and we use approximate Bayesian computation to demonstrate that they are fully identifiable from empirical reversal learning time series data. Finally, we demonstrate how the model can be extended to account for the increased complexity of behavioural dynamics over longer time scales involving fluctuating stimuli. This work is a step towards the development and evaluation of fully identifiable individual-level behaviour change models that can function as validated submodels for complex simulations of collective behaviour change.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f644e0376aeabf0dada3dac47c58b26bee7e1acc" target='_blank'>
              An agent-based model of behaviour change calibrated to reversal learning data
              </a>
            </td>
          <td>
            Roben Delos Reyes, Hugo Lyons Keenan, Cameron Zachreson
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="Accurate prediction of the future evolution of observational time series is a paramount challenge in current data‐driven research. While existing techniques struggle to learn useful representations from the temporal correlations, the high dimensionality in spatial domain is always considered as obstacle, leading to the curse of dimensionality and excessive resource consumption. This work designs a novel structure‐aware reservoir computing aiming at enhancing the predictability of coupled time series, by incorporating their historical dynamics as well as structural information. Paralleled reservoir computers with redesigned mixing inputs based on spatial relationships are implemented to cope with the multiple time series, whose core idea originates from the principle of the celebrated Granger causality. Representative numerical simulations and comparisons demonstrate the superior performance of the approach over the traditional ones. This work provides valuable insights into deeply mining both temporal and spatial information to enhance the representation learning of data in various machine learning techniques.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3af7d40101accb02cd7367635eb89b2f0f3967db" target='_blank'>
              Enhancing Time Series Predictability via Structure‐Aware Reservoir Computing
              </a>
            </td>
          <td>
            Suzhen Guo, Chun Guan, Siyang Leng
          </td>
          <td>2024-06-02</td>
          <td>Advanced Intelligent Systems</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Agent-based models (ABM) provide an excellent framework for modeling outbreaks and interventions in epidemiology by explicitly accounting for diverse individual interactions and environments. However, these models are usually stochastic and highly parametrized, requiring precise calibration for predictive performance. When considering realistic numbers of agents and properly accounting for stochasticity, this high dimensional calibration can be computationally prohibitive. This paper presents a random forest based surrogate modeling technique to accelerate the evaluation of ABMs and demonstrates its use to calibrate an epidemiological ABM named CityCOVID via Markov chain Monte Carlo (MCMC). The technique is first outlined in the context of CityCOVID's quantities of interest, namely hospitalizations and deaths, by exploring dimensionality reduction via temporal decomposition with principal component analysis (PCA) and via sensitivity analysis. The calibration problem is then presented and samples are generated to best match COVID-19 hospitalization and death numbers in Chicago from March to June in 2020. These results are compared with previous approximate Bayesian calibration (IMABC) results and their predictive performance is analyzed showing improved performance with a reduction in computation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b4ba5a747ad383f428c440aad51df2e9d201eb52" target='_blank'>
              Bayesian calibration of stochastic agent based model via random forest
              </a>
            </td>
          <td>
            Connor Robertson, C. Safta, Nicholson T. Collier, J. Ozik, Jaideep Ray
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="Emerging developments in artificial intelligence have opened infinite possibilities for material simulation. Depending on the powerful fitting of machine learning algorithms to first-principles data, machine learning interatomic potentials (MLIPs) can effectively balance the accuracy and efficiency problems in molecular dynamics (MD) simulations, serving as powerful tools in various complex physicochemical systems. Consequently, this brings unprecedented enthusiasm for researchers to apply such novel technology in multiple fields to revisit the major scientific problems that have remained controversial owing to the limitations of previous computational methods. Herein, we introduce the evolution of MLIPs, provide valuable application examples for solid-liquid interfaces, and present current challenges. Driven by solving multitudinous difficulties in terms of the accuracy, efficiency, and versatility of MLIPs, this booming technique, combined with molecular simulation methods, will provide an underlying and valuable understanding of interdisciplinary scientific challenges, including materials, physics, and chemistry.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2e9798b714816b66e1577ce3f8a8e63b5b7b470e" target='_blank'>
              Improving Molecular Dynamics Simulations for Solid-Liquid Interface with Machine Learning Interatomic Potentials.
              </a>
            </td>
          <td>
            Pengfei Hou, Yumiao Tian, Xing Meng
          </td>
          <td>2024-06-14</td>
          <td>Chemistry</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Biological neural networks can perform complex computations to predict their environment, far above the limited predictive capabilities of individual neurons. While conventional approaches to understanding these computations often focus on isolating the contributions of single neurons, here we argue that a deeper understanding requires considering emergent dynamics - dynamics that make the whole system"more than the sum of its parts". Specifically, we examine the relationship between prediction performance and emergence by leveraging recent quantitative metrics of emergence, derived from Partial Information Decomposition, and by modelling the prediction of environmental dynamics in a bio-inspired computational framework known as reservoir computing. Notably, we reveal a bidirectional coupling between prediction performance and emergence, which generalises across task environments and reservoir network topologies, and is recapitulated by three key results: 1) Optimising hyperparameters for performance enhances emergent dynamics, and vice versa; 2) Emergent dynamics represent a near sufficient criterion for prediction success in all task environments, and an almost necessary criterion in most environments; 3) Training reservoir computers on larger datasets results in stronger emergent dynamics, which contain task-relevant information crucial for performance. Overall, our study points to a pivotal role of emergence in facilitating environmental predictions in a bio-inspired computational architecture.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2572feb06e7ea060ecb305043a5d4b589af5886d" target='_blank'>
              Evolving reservoir computers reveals bidirectional coupling between predictive power and emergent dynamics
              </a>
            </td>
          <td>
            Hanna M. Tolle, A. Luppi, Anil K. Seth, P. Mediano
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="

 In the rapidly evolving pharmaceutical industry, drug efficacy and safety stand as critical concerns. The vast accumulation of data, including customer feedback, drug popularity, and usage details, offers a rich resource for improving healthcare outcomes.



 The primary aim of this study is to harness machine learning and Natural Language Processing (NLP) techniques to sift through extensive pharmaceutical data, identifying the most effective drugs for various conditions and uncovering patterns that could guide better decision-making in drug efficacy and safety.



 This research seeks to construct a sophisticated model capable of analyzing diverse data points to pinpoint the most efficacious drugs for specific health conditions, thereby providing pharmaceutical companies with data-driven insights to optimize drug safety and effectiveness.



 Employing a blend of Natural Language Processing (NLP) and machine learning strategies, the study analyzes a comprehensive dataset featuring customer reviews, drug popularity metrics, usage information, and other relevant data collected over an extended period. This methodological approach aims to reveal latent trends and patterns that are crucial for assessing drug performance.



 The developed model adeptly identifies leading medications for various conditions, elucidating efficacy and safety profiles derived from patient reviews and drug utilization trends. These findings furnish pharmaceutical companies with actionable intelligence for enhancing drug development and patient care strategies.



 The integration of machine learning and NLP for the analysis of vast drug-related datasets presents a powerful method for advancing drug efficacy and safety. The insights yielded by the proposed model significantly empower the decision-making processes of the pharmaceutical industry, ultimately fostering improved health outcomes for patients.
">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/efa74cbe78ea6c68e73d842096a74cc272cfa464" target='_blank'>
              Leveraging Machine Learning and Patient Reviews for Developing a Drug Recommendation System to Reduce Medical Errors
              </a>
            </td>
          <td>
            K. P. Swain, S.K. Mohapatra, Vinayakumar Ravi, S. R. Nayak, Tahani Jaser Alahmadi, Prabhishek Singh, M. Diwakar
          </td>
          <td>2024-06-25</td>
          <td>The Open Bioinformatics Journal</td>
          <td>0</td>
          <td>22</td>
        </tr>

        <tr id="Antimicrobial resistance is a critical public health concern, necessitating the exploration of alternative treatments. While antimicrobial peptides (AMPs) show promise, assessing their toxicity using traditional wet lab methods is both time‐consuming and costly. We introduce tAMPer, a novel multi‐modal deep learning model designed to predict peptide toxicity by integrating the underlying amino acid sequence composition and the three‐dimensional structure of peptides. tAMPer adopts a graph‐based representation for peptides, encoding ColabFold‐predicted structures, where nodes represent amino acids and edges represent spatial interactions. Structural features are extracted using graph neural networks, and recurrent neural networks capture sequential dependencies. tAMPer's performance was assessed on a publicly available protein toxicity benchmark and an AMP hemolysis data we generated. On the latter, tAMPer achieves an F1‐score of 68.7%, outperforming the second‐best method by 23.4%. On the protein benchmark, tAMPer exhibited an improvement of over 3.0% in the F1‐score compared to current state‐of‐the‐art methods. We anticipate tAMPer to accelerate AMP discovery and development by reducing the reliance on laborious toxicity screening experiments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/06ba4c20a378ada12e9e1b9bec5db0ae6b8fa9a8" target='_blank'>
              Structure‐aware deep learning model for peptide toxicity prediction
              </a>
            </td>
          <td>
            Hossein Ebrahimikondori, Darcy Sutherland, Anat Yanai, Amelia Richter, Ali Salehi, Chenkai Li, Lauren Coombe, Monica Kotkoff, René L Warren, I. Birol
          </td>
          <td>2024-06-22</td>
          <td>Protein Science : A Publication of the Protein Society</td>
          <td>0</td>
          <td>64</td>
        </tr>

        <tr id="The convergence of ensemble deep learning and machine learning has become a critical strategy for tackling intricate challenges across diverse fields such as healthcare, finance, and autonomous systems. Ensemble approaches, which combine the strengths of multiple models, are known for enhancing predictive accuracy, robustness, and generalizability. This paper investigates the applications of ensemble techniques, emphasizing their role in improving diagnostic precision in medical imaging, advancing fraud detection mechanisms in financial services, and refining decision-making in autonomous vehicles. Recent advancements in ensemble methods, including stacking, boosting, and bagging, have shown to outperform single models in various contexts. However, several challenges accompany the opportunities offered by ensemble learning, such as high computational demands, issues with model interpretability, and the potential for overfitting. This study explores ways to address these challenges, including the creation of more efficient algorithms and the incorporation of explainable AI (XAI) frameworks to enhance transparency and user trust. Furthermore, we discuss the future impact of cutting-edge technologies like quantum computing and federated learning on the evolution of ensemble techniques. The future of ensemble deep learning and machine learning is set to be shaped by the proliferation of big data, advancements in computational hardware, and the need for real-time, scalable solutions. This paper provides an extensive review of the current state of ensemble learning, identifies significant challenges, and suggests future research directions to fully harness the potential of these techniques in addressing real-world problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ce2d695680709ca225454e5c5f788cc6ae0cb87" target='_blank'>
              Ensemble Deep Learning and Machine Learning: Applications, Opportunities, Challenges, and Future Directions
              </a>
            </td>
          <td>
            N. Rane, Saurabh Choudhary, Jayesh Rane
          </td>
          <td>2024-07-04</td>
          <td>SSRN Electronic Journal</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Wildland fires pose terrifying natural hazards, underscoring the urgent need to develop data-driven and physics-informed digital twins for wildfire prevention, monitoring, intervention, and response. In this direction of research, this work introduces a physics-informed neural network (PiNN) to learn the unknown parameters of an interpretable wildfire spreading model. The considered wildfire spreading model integrates fundamental physical laws articulated by key model parameters, essential for capturing the complex behavior of wildfires. The proposed machine learning approach leverages the theory of artificial neural networks with the physical constraints governing wildfire dynamics, such as the first principles of mass and energy conservation. Training of the PiNN for physics-informed parameter identification is realized using data of the temporal evolution of one- and two-dimensional (plane surface) fire fronts that have been obtained from a high-fidelity simulator of the wildfire spreading model under consideration. The parameter learning results demonstrate the remarkable predictive ability of the proposed PiNN in uncovering the unknown coefficients in both the one- and two-dimensional fire spreading scenarios. Additionally, this methodology exhibits robustness by identifying the same parameters in the presence of noisy data. The proposed framework is envisioned to be incorporated in a physics-informed digital twin for intelligent wildfire management and risk assessment.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62a95332da195a72205da26346da59e96afa55ce" target='_blank'>
              Physics-informed neural networks for parameter learning of wildfire spreading
              </a>
            </td>
          <td>
            K. Vogiatzoglou, C. Papadimitriou, V. Bontozoglou, Konstantinos Ampountolas
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>45</td>
        </tr>

        <tr id="The dynamics of biological systems, from proteins to cells to organisms, is complex and stochastic. To decipher their physical laws, we need to bridge between experimental observations and theoretical modeling. Thanks to progress in microscopy and tracking, there is today an abundance of experimental trajectories reflecting these dynamical laws. Inferring physical models from noisy and imperfect experimental data, however, is challenging. Because there are no inference methods that are robust and efficient, model reconstruction from experimental trajectories is a bottleneck to data-driven biophysics. In this Thesis, I present a set of tools developed to bridge this gap and permit robust and universal inference of stochastic dynamical models from experimental trajectories. These methods are rooted in an information-theoretical framework that quantifies how much can be inferred from trajectories that are short, partial and noisy. They permit the efficient inference of dynamical models for overdamped and underdamped Langevin systems, as well as the inference of entropy production rates. I finally present early applications of these techniques, as well as future research directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/36d5a9b10d628b238c485c3e97f0ad456101dedb" target='_blank'>
              Learning dynamical models from stochastic trajectories
              </a>
            </td>
          <td>
            Pierre Ronceray
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Abstract Motivation Complex diseases are often caused and characterized by misregulation of multiple biological pathways. Differential network analysis aims to detect significant rewiring of biological network structures under different conditions and has become an important tool for understanding the molecular etiology of disease progression and therapeutic response. With few exceptions, most existing differential network analysis tools perform differential tests on separately learned network structures that are computationally expensive and prone to collapse when grouped samples are limited or less consistent. Results We previously developed an accurate differential network analysis method—differential dependency networks (DDN), that enables joint learning of common and rewired network structures under different conditions. We now introduce the DDN3.0 tool that improves this framework with three new and highly efficient algorithms, namely, unbiased model estimation with a weighted error measure applicable to imbalance sample groups, multiple acceleration strategies to improve learning efficiency, and data-driven determination of proper hyperparameters. The comparative experimental results obtained from both realistic simulations and case studies show that DDN3.0 can help biologists more accurately identify, in a study-specific and often unknown conserved regulatory circuitry, a network of significantly rewired molecular players potentially responsible for phenotypic transitions. Availability and implementation The Python package of DDN3.0 is freely available at https://github.com/cbil-vt/DDN3. A user’s guide and a vignette are provided at https://ddn-30.readthedocs.io/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0f7f53646bc589a6aedd4807e452f6e2c56e13c4" target='_blank'>
              DDN3.0: determining significant rewiring of biological network structure with differential dependency networks
              </a>
            </td>
          <td>
            Yi Fu, Yingzhou Lu, Yizhi Wang, Bai Zhang, Zhen Zhang, Guoqiang Yu, Chunyu Liu, Robert Clarke, David M. Herrington, Yue Wang
          </td>
          <td>2024-06-01</td>
          <td>Bioinformatics</td>
          <td>3</td>
          <td>19</td>
        </tr>

        <tr id="Causal models provide rich descriptions of complex systems as sets of mechanisms by which each variable is influenced by its direct causes. They support reasoning about manipulating parts of the system and thus hold promise for addressing some of the open challenges of artificial intelligence (AI), such as planning, transferring knowledge in changing environments, or robustness to distribution shifts. However, a key obstacle to more widespread use of causal models in AI is the requirement that the relevant variables be specified a priori, which is typically not the case for the high-dimensional, unstructured data processed by modern AI systems. At the same time, machine learning (ML) has proven quite successful at automatically extracting useful and compact representations of such complex data. Causal representation learning (CRL) aims to combine the core strengths of ML and causality by learning representations in the form of latent variables endowed with causal model semantics. In this thesis, we study and present new results for different CRL settings. A central theme is the question of identifiability: Given infinite data, when are representations satisfying the same learning objective guaranteed to be equivalent? This is an important prerequisite for CRL, as it formally characterises if and when a learning task is, at least in principle, feasible. Since learning causal models, even without a representation learning component, is notoriously difficult, we require additional assumptions on the model class or rich data beyond the classical i.i.d. setting. By partially characterising identifiability for different settings, this thesis investigates what is possible for CRL without direct supervision, and thus contributes to its theoretical foundations. Ideally, the developed insights can help inform data collection practices or inspire the design of new practical estimation methods.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e04aac37f37d12a0fb6d08faf3b9b67cbb5ea13" target='_blank'>
              Identifiable Causal Representation Learning: Unsupervised, Multi-View, and Multi-Environment
              </a>
            </td>
          <td>
            Julius von Kügelgen
          </td>
          <td>2024-06-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>18</td>
        </tr>

        <tr id="Neuropeptides are biomolecules with crucial physiological functions. Accurate identification of neuropeptides is essential for understanding nervous system regulatory mechanisms. However, traditional analysis methods are expensive and laborious, and the development of effective machine learning models continues to be a subject of current research. Hence, in this research, we constructed an SVM-based machine learning neuropeptide predictor, iNP_ESM, by integrating protein language models Evolutionary Scale Modeling (ESM) and Unified Representation (UniRep) for the first time. Our model utilized feature fusion and feature selection strategies to improve prediction accuracy during optimization. In addition, we validated the effectiveness of the optimization strategy with UMAP (Uniform Manifold Approximation and Projection) visualization. iNP_ESM outperforms existing models on a variety of machine learning evaluation metrics, with an accuracy of up to 0.937 in cross-validation and 0.928 in independent testing, demonstrating optimal neuropeptide recognition capabilities. We anticipate improved neuropeptide data in the future, and we believe that the iNP_ESM model will have broader applications in the research and clinical treatment of neurological diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/ed8e92215fee91abba9a480dfc2b4401a6c9ecf2" target='_blank'>
              iNP_ESM: Neuropeptide Identification Based on Evolutionary Scale Modeling and Unified Representation Embedding Features
              </a>
            </td>
          <td>
            Honghao Li, Liangzhen Jiang, Kaixiang Yang, Shulin Shang, Mingxin Li, Zhibin Lv
          </td>
          <td>2024-06-27</td>
          <td>International Journal of Molecular Sciences</td>
          <td>0</td>
          <td>9</td>
        </tr>

        <tr id="The estimation of directed couplings between the nodes of a network from indirect measurements is a central methodological challenge in scientific fields such as neuroscience, systems biology and economics. Unfortunately, the problem is generally ill-posed due to the possible presence of unknown delays in the measurements. In this paper, we offer a solution of this problem by using a variational Bayes framework, where the uncertainty over the delays is marginalized in order to obtain conservative coupling estimates. To overcome the well-known overconfidence of classical variational methods, we use a hybrid-VI scheme where the (possibly flat or multimodal) posterior over the measurement parameters is estimated using a forward KL loss while the (nearly convex) conditional posterior over the couplings is estimated using the highly scalable gradient-based VI. In our ground-truth experiments, we show that the network provides reliable and conservative estimates of the couplings, greatly outperforming similar methods such as regression DCM.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a02bb50bd2145ddb88212879e50602c28462b00e" target='_blank'>
              Robust and highly scalable estimation of directional couplings from time-shifted signals
              </a>
            </td>
          <td>
            Luca Ambrogioni, Louis Rouillard, Demian Wassermann
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Single-cell technologies enable comprehensive profiling of diverse immune cell-types through the measurement of multiple genes or proteins per cell. In order to translate data from immune profiling assays into powerful diagnostics, machine learning approaches are used to compute per-sample immunological summaries, or featurizations that can be used as inputs to models for outcomes of interest. Current supervised learning approaches for computing per-sample representations are optimized based only on the outcome variable to be predicted and do not take into account clinically-relevant covariates that are likely to also be measured. Here we expand the optimization problem to also take into account such additional patient covariates to directly inform the learned per-sample representations. To do this, we introduce CytoCoSet, a set-based encoding method, which formulates a loss function with an additional triplet term penalizing samples with similar covariates from having disparate embedding results in per-sample representations. Overall, incorporating clinical covariates leads to improved prediction of clinical phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ca96995fe3a8574dda746ff07923576169480ac" target='_blank'>
              Conditional Similarity Triplets Enable Covariate-Informed Representations of Single-Cell Data
              </a>
            </td>
          <td>
            Chi-Jane Chen, Haidong Yi, Natalie Stanley
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Survival analysis models time-to-event distributions with censorship. Recently, deep survival models using neural networks have dominated due to their representational power and state-of-the-art performance. However, their"black-box"nature hinders interpretability, which is crucial in real-world applications. In contrast,"white-box"tree-based survival models offer better interpretability but struggle to converge to global optima due to greedy expansion. In this paper, we bridge the gap between previous deep survival models and traditional tree-based survival models through deep rectified linear unit (ReLU) networks. We show that a deliberately constructed deep ReLU network (SurvReLU) can harness the interpretability of tree-based structures with the representational power of deep survival models. Empirical studies on both simulated and real survival benchmark datasets show the effectiveness of the proposed SurvReLU in terms of performance and interoperability. The code is available at \href{https://github.com/xs018/SurvReLU}{\color{magenta}{ https://github.com/xs018/SurvReLU}}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7f1480d0dd0cd199978f7d5c4c15ebce856837d3" target='_blank'>
              SurvReLU: Inherently Interpretable Survival Analysis via Deep ReLU Networks
              </a>
            </td>
          <td>
            Xiaotong Sun, Peijie Qiu, Shengfan Zhang
          </td>
          <td>2024-07-19</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Gene regulatory networks (GRNs) govern many core developmental and biological processes underlying human complex traits. Even with broad-scale efforts to characterize the effects of molecular perturbations and interpret gene coexpression, it remains challenging to infer the architecture of gene regulation in a precise and efficient manner. Key properties of GRNs, like hierarchical structure, modular organization, and sparsity, provide both challenges and opportunities for this objective. Here, we seek to better understand properties of GRNs using a new approach to simulate their structure and model their function. We produce realistic network structures with a novel generating algorithm based on insights from small-world network theory, and we model gene expression regulation using stochastic differential equations formulated to accommodate modeling molecular perturbations. With these tools, we systematically describe the effects of gene knockouts within and across GRNs, finding a subset of networks that recapitulate features of a recent genome-scale perturbation study. With deeper analysis of these exemplar networks, we consider future avenues to map the architecture of gene expression regulation using data from cells in perturbed and unperturbed states, finding that while perturbation data are critical to discover specific regulatory interactions, data from unperturbed cells may be sufficient to reveal regulatory programs.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4ce6cbb43359b3ba6091349b1f3352dd2910fbfb" target='_blank'>
              Gene regulatory network structure informs the distribution of perturbation effects
              </a>
            </td>
          <td>
            Matthew Aguirre, J. Spence, Guy Sella, Jonathan K. Pritchard
          </td>
          <td>2024-07-05</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We present an application of Physics-Informed Neural Networks to handle MultiPhase-Field simulations of microstructure evolution. It has been showcased that a combination of optimization techniques extended and adapted from the PINNs literature, and the introduction of specific techniques inspired by the MPF Method background, is required. The numerical resolution is realized through a multi-variable time-series problem by using fully discrete resolution. Within each interval, space, time, and phases are treated separately, constituting discrete subdomains. An extended multi-networking concept is implemented to subdivide the simulation domain into multiple batches, with each batch associated with an independent Neural Network trained to predict the solution. To ensure efficient interaction across different phasesand in the spatio-temporal-phasic subdomain, a Master NN handles efficient interaction among the multiple networks, as well as the transfer of learning in different directions. A set of systematic simulations with increasing complexity was performed, that benchmarks various critical aspects of MPF simulations, including different geometries, types of interface dynamics and the evolution of an interfacial triple junction. A comprehensive approach is adopted to specifically focus the attention on the interfacial regions through an automatic and dynamic meshing process, significantly simplifying the tuning of hyper-parameters and serving as a fundamental key for addressing MPF problems using Machine Learning. The pyramidal training approach is proposed to the PINN community as a dual-impact method: it facilitates the initialization of training and allows an extended transfer of learning. The proposed PINNs-MPF framework successfully reproduces benchmark tests with high fidelity and Mean Squared Error loss values ranging from 10$^{-4}$ to 10$^{-6}$ compared to ground truth solutions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f31d909eac4f45aba2c5d7eda80b9159e9f0772f" target='_blank'>
              PINNs-MPF: A Physics-Informed Neural Network Framework for Multi-Phase-Field Simulation of Interface Dynamics
              </a>
            </td>
          <td>
            Seifallah Elfetni, R. D. Kamachali
          </td>
          <td>2024-07-02</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="In this study, we develop and implement the Extended Kalman Filter (EKF) to forecast the rate of change in tumor cells, healthy host cells, and effector immune cells within the Itik-Banks model. This novel application of EKF in cancer dynamics modeling aims to provide precise real-time estimations of cellular interactions, especially in constructing a new state space representation from the Itik-Banks model. We use a first-order Taylor series to linearize the model. The numerical simulations were performed to analyze the accuracy of this new state space with data from William Gilpin’s GitHub repository. The results show that the EKF predictions strongly align with actual data, i.e., in the prior and posterior steps for tumor and healthy host cells, there is a strong agreement between the predictions and the actual data. The EKF captures the oscillatory nature of the tumor and healthy host cell population well. The peaks and troughs of the predictions align closely with the actual data, indicating the EKF’s effectiveness in modeling the dynamic behavior of the tumor and healthy host cells. However, for effector immune cells, the oscillatory nature of the data in these cells gives rise to slight deviations. This represents a significant challenge in the future for updating the state space representations. Despite minor discrepancies, the EKF demonstrates a strong performance in both the training and testing data, with the posterior step estimates significantly improving the prior step accuracy. This study emphasizes the importance of data availability for accurate predictions, noting a symmetric Mean Absolute Percentage Error (sMAPE) of 35.92% when data is unavailable. Prompt correction with new data is essential to maintain accuracy. This research underscores the EKF’s potential for real-time monitoring and prediction in complex biological systems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c47c21057eed2151b03ae81e7e3e1609e8f172d" target='_blank'>
              Prediction of the Change Rate of Tumor Cells, Healthy Host Cells, and Effector Immune Cells in a Three-Dimensional Cancer Model using Extended Kalman Filter
              </a>
            </td>
          <td>
            Nina Fitriyati, Salma Abidah Faizah, Taufik Edy Sutanto
          </td>
          <td>2024-06-27</td>
          <td>Jambura Journal of Biomathematics (JJBM)</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In the growing body of scientific literature, the structure and information of drugs are usually represented in two-dimensional vector graphics. Drug compound structures in vector graphics form are difficult to recognize and utilize by computers. Although the current OCSR paradigm has shown good performance, most existing work treats it as a single isolated whole. This paper proposes a multi-stage cognitive neural network model that predicts molecular vector graphics more finely. Based on cognitive methods, we construct a model for fine-grained perceptual representation of molecular images from bottom to top, and in stages, the primary representation of atoms and bonds is potential discrete label sequence (atom type, bond type, functional group, etc.). The second stage represents the molecular graph according to the label sequence, and the final stage evolves in an extensible manner from the molecular graph to a machine-readable sequence. Experimental results show that MMSSC-Net outperforms current advanced methods on multiple public datasets. It achieved an accuracy rate of 75–94% on cognitive recognition at different resolutions. MMSSC-Net uses a sequence cognitive method to make it more reliable in interpretability and transferability, and provides new ideas for drug information discovery and exploring the unknown chemical space.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f1f508af6af937f4dbf83d5e0c186caa5500fb28" target='_blank'>
              MMSSC-Net: multi-stage sequence cognitive networks for drug molecule recognition
              </a>
            </td>
          <td>
            Dehai Zhang, Di Zhao, Zhengwu Wang, Junhui Li, Jin Li
          </td>
          <td>2024-06-06</td>
          <td>RSC Advances</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="We describe version 2 of the SPICE dataset, a collection of quantum chemistry calculations for training machine learning potentials. It expands on the original dataset by adding much more sampling of chemical space and more data on non-covalent interactions. We train a set of potential energy functions called Nutmeg on it. They use a novel mechanism to improve performance on charged and polar molecules, injecting precomputed partial charges into the model to provide a reference for the large scale charge distribution. Evaluation of the new models shows they do an excellent job of reproducing energy differences between conformations, even on highly charged molecules or ones that are significantly larger than the molecules in the training set. They also produce stable molecular dynamics trajectories, and are fast enough to be useful for routine simulation of small molecules.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/173c7c4074c9adca31fa419f912637ed59b884a7" target='_blank'>
              Nutmeg and SPICE: Models and Data for Biomolecular Machine Learning
              </a>
            </td>
          <td>
            Peter K. Eastman, Benjamin P. Pritchard, J. Chodera, T. Markland
          </td>
          <td>2024-06-18</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="The complexity of cancer development involves intricate interactions among multiple biomarkers, such as gene-environment interactions. Utilizing microarray gene expression profile data for cancer classification is anticipated to be effective, thus drawing considerable interest in the fields of bioinformatics and computational biology. Due to the characteristics of genomic data, problems of high-dimensional interactions and noise interference do exist during the analysis process. When building cancer diagnosis models, we often face the dilemma of model adaptation errors due to an imbalance of data types. To mitigate the issues, we apply the SMOTE-Tomek procedure to rectify the imbalance problem. Following this, we utilize the overlapping group screening method alongside a binary logistic regression model to integrate gene pathway information, facilitating the identification of significant biomarkers associated with clinically imbalanced cancer or normal outcomes. Simulation studies across different imbalanced rates and gene structures validate our proposed method’s effectiveness, surpassing common machine learning techniques in terms of classification prediction accuracy. We also demonstrate that prediction performance improves with SMOTE-Tomek treatment compared to no imbalance treatment and SMOTE treatment across various imbalance rates. In the real-world application, we integrate clinical and gene expression data with prior pathway information. We employ SMOTE-Tomek and our proposed methods to identify critical biomarkers and gene-environment interactions linked to the imbalanced binary outcomes (cancer or normal) in patients from the Cancer Genome Atlas datasets of lung adenocarcinoma and breast invasive carcinoma. Our proposed method consistently achieves satisfactory classification accuracy. Additionally, we have identified biomarkers indicative of gene-environment interactions relevant to cancer and have provided corresponding estimates of odds ratios. Moreover, in high-dimensional imbalanced data, for achieving good prediction results, we recommend considering the order of balancing processing and feature screening.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9c92db672ff40d3a3f4698753d212345af0e7055" target='_blank'>
              Cancer Diagnosis by Gene-Environment Interactions via Combination of SMOTE-Tomek and Overlapped Group Screening Approaches with Application to Imbalanced TCGA Clinical and Genomic Data
              </a>
            </td>
          <td>
            Jie-Huei Wang, Cheng-Yu Liu, You-Ruei Min, Zih-Han Wu, Po-Lin Hou
          </td>
          <td>2024-07-15</td>
          <td>Mathematics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We consider population modelling using parametrised ordinary differential equation initial value problems (ODE-IVPs), in the setting where the parameter vector for each individual is not known, but a vector of covariates is available for each individual and follows an unknown distribution. For each individual, only one component of the solution to the ODE-IVP can be observed at only finitely many observation times. An important problem is to identify a covariate-parameter relationship that maps covariate vectors to parameter vectors. Such settings and problems arise in pharmacology, where the observations are blood drug concentrations, the covariates are clinically observable quantities, and the covariate-parameter relationship is used for personalised drug dosing. We use the framework presented in (Nickl, 2023) to analyse a nonparametric, nonlinear Bayesian inverse problem for the unknown covariate-parameter relationship, for a fixed finite time design, random covariate design, and a family of rescaled Gaussian priors. We provide sufficient conditions for posterior contraction, local asymptotic normality, and a Bernstein-von Mises theorem for linear functionals, and show theoretically that these conditions are met on a two-compartment model from the pharmacokinetics literature.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/93423a9411167dbeef9eaca6cbe4d75f84294ff3" target='_blank'>
              Bayesian inference of covariate-parameter relationships for population modelling
              </a>
            </td>
          <td>
            Han Cheng Lie
          </td>
          <td>2024-07-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="The estimation of Conditional Average Treatment Effects (CATE) is crucial for understanding the heterogeneity of treatment effects in clinical trials. We evaluate the performance of common methods, including causal forests and various meta-learners, across a diverse set of scenarios, revealing that each of the methods struggles in one or more of the tested scenarios. Given the inherent uncertainty of the data-generating process in real-life scenarios, the robustness of a CATE estimator to various scenarios is critical for its reliability. To address this limitation of existing methods, we propose two new ensemble methods that integrate multiple estimators to enhance prediction stability and performance - Stacked X-Learner which uses the X-Learner with model stacking for estimating the nuisance functions, and Consensus Based Averaging (CBA), which averages only the models with highest internal agreement. We show that these models achieve good performance across a wide range of scenarios varying in complexity, sample size and structure of the underlying-mechanism, including a biologically driven model for PD-L1 inhibition pathway for cancer treatment. Furthermore, we demonstrate improved performance by the Stacked X-Learner also when comparing to other ensemble methods, including R-Stacking, Causal-Stacking and others.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f50bfe51bd232f8abec5b22f45b9929f97ffd47" target='_blank'>
              Robust CATE Estimation Using Novel Ensemble Methods
              </a>
            </td>
          <td>
            Oshri Machluf, Tzviel Frostig, Gal Shoham, Tomer Milo, Elad Berkman, Raviv Pryluk
          </td>
          <td>2024-07-04</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Spaceflight studies present novel insights into biological processes through exposure to stressors outside the evolutionary path of terrestrial organisms. Despite limited access to space environments, numerous transcriptomic datasets from spaceflight experiments are now available through NASA’s GeneLab data repository, which allows public access to these datasets, encouraging further analysis. While various computational pipelines and methods have been used to process these transcriptomic datasets, learning-model-driven analyses have yet to be applied to a broad array of such spaceflight-related datasets. In this study, we propose an open-source framework, GLARE: GeneLAb Representation learning pipelinE, which consists of training different representation learning approaches from manifold learning to self-supervised learning that enhances the performance of downstream analytical tasks such as pattern recognition. We illustrate the utility of GLARE by applying it to gene-level transcriptional values from the results of the CARA spaceflight experiment, an Arabidopsis root tip transcriptome dataset that spanned light, dark, and microgravity treatments. We show that GLARE not only substantiated the findings of the original study concerning cell wall remodeling but also revealed additional patterns of gene expression affected by the treatments, including evidence of hypoxia. This work suggests there is great potential to supplement the insights drawn from initial studies on spaceflight omics-level data through further machine-learning-enabled analyses.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f6c46ff5b99e48dd2b0f16b61252b7124d1818f6" target='_blank'>
              GLARE: Discovering hidden patterns in spaceflight transcriptome using representation learning
              </a>
            </td>
          <td>
            DongHyeon Seo, Hunter F Strickland, Mingqi Zhou, R. Barker, R. Ferl, A. Paul, Simon Gilroy
          </td>
          <td>2024-06-06</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>56</td>
        </tr>

        <tr id="Summary The retarded transient function (RTF) approach has been introduced as a complementary approach to employing ordinary differential equations (ODEs) for modelling dynamics typically observed for cellular signalling processes. Here, we introduce the R package to the RTF approach, which has originally been implemented within the MATLAB-based Data2Dynamics modelling framework. The package enables modelling of time and dose dependencies and includes model reduction to minimize overfitting. It can be applied not only to experimental data but also to trajectories of ODE models, to characterize the dynamics or to identify major targets of experimental perturbations in the low-dimensional representations calculated by the package. Availability and Implementation The R package RTF is available at https://github.com/kreutz-lab/RTF. Contact clemens.kreutz@uniklinik-freiburg.de">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/0620e9d50945b256e9edbd53674e984230196e5b" target='_blank'>
              RTF: An R package for modelling time course data
              </a>
            </td>
          <td>
            Eva Brombacher, Clemens Kreutz
          </td>
          <td>2024-06-26</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Generative models have the potential to accelerate key steps in the discovery of novel molecular therapeutics and materials. Diffusion models have recently emerged as a powerful approach, excelling at unconditional sample generation and, with data-driven guidance, conditional generation within their training domain. Reliably sampling from high-value regions beyond the training data, however, remains an open challenge -- with current methods predominantly focusing on modifying the diffusion process itself. In this paper, we develop context-guided diffusion (CGD), a simple plug-and-play method that leverages unlabeled data and smoothness constraints to improve the out-of-distribution generalization of guided diffusion models. We demonstrate that this approach leads to substantial performance gains across various settings, including continuous, discrete, and graph-structured diffusion processes with applications across drug discovery, materials science, and protein design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/564ed6e6f523b59a9eadbed1339a8278b556e2c2" target='_blank'>
              Context-Guided Diffusion for Out-of-Distribution Molecular and Protein Design
              </a>
            </td>
          <td>
            Leo Klarner, Tim G. J. Rudner, Garrett M. Morris, Charlotte M. Deane, Y. W. Teh
          </td>
          <td>2024-07-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Drug-side effect prediction has become an essential area of research in the field of pharmacology. As the use of medications continues to rise, so does the importance of understanding and mitigating the potential risks associated with them. At present, researchers have turned to data-driven methods to predict drug-side effects. Drug-side effect prediction is a link prediction problem, and the related data can be described from various perspectives. To process these kinds of data, a multi-view method, called Multiple Kronecker RLS fusion-based link propagation (MKronRLSF-LP), is proposed. MKronRLSF-LP extends the Kron-RLS by finding the consensus partitions and multiple graph Laplacian constraints in the multi-view setting. Both of these multi-view settings contribute to a higher quality result. Extensive experiments have been conducted on drug-side effect datasets, and our empirical results provide evidence that our approach is effective and robust.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a63527bfc5bdbb286aecbc16d31c2e4de057109d" target='_blank'>
              Multiple Kronecker RLS fusion-based link propagation for drug-side effect prediction
              </a>
            </td>
          <td>
            Yuqing Qian, Ziyu Zheng, Prayag Tiwari, Yijie Ding, Quan Zou
          </td>
          <td>2024-06-27</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Model parameter inference is a universal problem across science. This challenge is particularly pronounced in developmental biology, where faithful mechanistic descriptions require spatial-stochastic models with numerous parameters, yet quantitative empirical data often lack sufficient granularity due to experimental limitations. Parameterizing such complex models thus necessitates methods that elaborate on classical Bayesian inference by incorporating notions of optimality and goal-orientation through low-dimensional objective functions that quantitatively capture the target behavior of the underlying system. In this study, we contrast two such inference workflows and apply them to biophysics-inspired spatial-stochastic models. Technically, both workflows are simulation-based inference (SBI) methods. The first method leverages a modern deep-learning technique known as sequential neural posterior estimation (SNPE), while the second is based on a classical optimization technique called simulated annealing (SA). We evaluate these workflows by inferring the parameters of two complementary models for the inner cell mass (ICM) lineage differentiation in the blastocyst-stage mouse embryo. This developmental biology system serves as a paradigmatic example of a highly robust and reproducible cell-fate proportioning process that self-organizes under strongly stochastic conditions, such as intrinsic biochemical noise and cell-cell signaling delays. Our results indicate that while both methods largely agree in their predictions, the modern SBI workflow provides substantially richer inferred distributions at an equivalent computational cost. We identify the computational scenarios that favor the modern SBI method over its classical counterpart. Finally, we propose a plausible approach to integrate these two methods, thereby synergistically exploiting their parameter space exploration capabilities.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/64ec5827bd7f9a7b3d935cb3a63d276e77e4cbb9" target='_blank'>
              Comparing AI versus Optimization Workflows for Simulation-Based Inference of Spatial-Stochastic Systems
              </a>
            </td>
          <td>
            Michael A. Ramirez-Sierra, T. Sokolowski
          </td>
          <td>2024-07-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Predicting drug toxicity is a critical aspect of ensuring patient safety during the drug design process. Although conventional machine learning techniques have shown some success in this field, the scarcity of annotated toxicity data poses a significant challenge in enhancing models’ performance. In this study, we explore the potential of leveraging large unlabeled datasets using semi-supervised learning to improve predictive performance for cardiotoxicity across three targets: the voltage-gated potassium channel (hERG), the voltage-gated calcium channel (Cav1.2), and the voltage-gated sodium channel (Nav1.5). We extensively mined the ChEMBL database, comprising approximately 2 million small molecules, then employed semi-supervised learning to construct robust classification models for this purpose. We achieved a performance boost on highly diverse (i.e. structurally dissimilar) test datasets across all three targets. Using our built models, we screened the whole ChEMBL database and a large set of FDA-approved drugs, identifying several compounds with potential cardiac channel activity. To ensure broad accessibility and usability for both technical and non-technical users, we developed a cross-platform graphical user interface that allows users to make predictions and gain insights into the cardiotoxicity of drugs and other small molecules. The software is made available as open source under the permissive MIT license at https://github.com/issararab/CToxPred2.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f4f0e09b146e54215719071ab7b9f648e458c75b" target='_blank'>
              Semi-Supervised Learning to Boost Cardiotoxicity Prediction by Mining a Large Unlabeled Small Molecule Dataset
              </a>
            </td>
          <td>
            Issar Arab, Kris Laukens, Wout Bittremieux
          </td>
          <td>2024-05-30</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>24</td>
        </tr>

        <tr id="Single-cell multi-omics technologies have empowered increasingly refined characterisation of the heterogeneity of cell populations. Automated cell type annotation methods have been developed to transfer cell type labels from well-annotated reference datasets to emerging query datasets. However, these methods suffer from some common caveats, including the failure to characterise transitional and novel cell states, sensitivity to batch effects and under-utilisation of phenotypic information other than cell types (e.g. sample source and disease conditions). We developed Φ-Space, a computational framework for the continuous phenotyping of single-cell multi-omics data. In Φ-Space we adopt a highly versatile modelling strategy to continuously characterise query cell identity in a low-dimensional phenotype space, defined by reference phenotypes. The phenotype space embedding enables various downstream analyses, including insightful visualisations, clustering and cell type labelling. We demonstrate through three case studies that Φ-Space (i) characterises developing and out-of-reference cell states; (ii) is robust against batch effects in both reference and query; (iii) adapts to annotation tasks involving multiple omics types; (iv) overcomes technical differences between reference and query. The versatility of Φ-Space makes it applicable to a wide range analytical tasks beyond cell type transfer, and its ability to model complex phenotypic variation will facilitate biological discoveries from different omics types.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6f8f58c02f14e009289058adb00560711a02d08b" target='_blank'>
              Φ-Space: Continuous phenotyping of single-cell multi-omics data
              </a>
            </td>
          <td>
            Jiadong Mao, Yidi Deng, K. Lê Cao
          </td>
          <td>2024-06-22</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>29</td>
        </tr>

        <tr id="Unveiling the specificity in T-cell-receptor and antigen recognition represents a major step to understand the immune system response. Many supervised machine learning approaches have been designed to build sequence-based predictive models of such specificity using binding and non-binding examples of data. Due to the presence of few specific and many non-specific T-cell receptors for each antigen, available datasets are heavily imbalanced and make the goal of achieving solid predictive performances very challenging. Here, we propose to restore data balance through data augmentation using generative unsupervised models. We then use these augmented data to train supervised models for prediction of peptide-specific T-cell receptors and binding pairs of peptide and T-cell receptors sequences. We show that our pipeline yields increased performance in terms of T-cell receptors specificity prediction tasks. More broadly, our work provides a general framework to restore balance in computational problems involving biological sequence data. Significance Statement The adaptive immune system carries a diverse set of T-cell receptors capable of recognizing pathogens and protect the host from diseases. Predicting whether a receptor binds a pathogenic peptide is a fundamental computational problem, made difficult by the imbalance in available data: relatively few binding pairs are known compared to all possible pairs of receptors and peptides. Here, we propose to mitigate this imbalance problem by generating putative binding pairs through data augmentation machine-learning methods. We show that these extra data helps training binding prediction models and improves their performances. Our framework for sequence data augmentation is generic and could be applied to other biological computational problems.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/228b069d2aba2649c66d209396ce118d1a2838c4" target='_blank'>
              Restoring data balance via generative models of T-cell receptors for antigen-binding prediction
              </a>
            </td>
          <td>
            Emanuele Loffredo, Mauro Pastore, Simona Cocco, R. Monasson
          </td>
          <td>2024-07-15</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>40</td>
        </tr>

        <tr id="Bayesian graphical models are powerful tools to infer complex relationships in high dimension, yet are often fraught with computational and statistical challenges. If exploited in a principled way, the increasing information collected alongside the data of primary interest constitutes an opportunity to mitigate these difficulties by guiding the detection of dependence structures. For instance, gene network inference may be informed by the use of publicly available summary statistics on the regulation of genes by genetic variants. Here we present a novel Gaussian graphical modeling framework to identify and leverage information on the centrality of nodes in conditional independence graphs. Specifically, we consider a fully joint hierarchical model to simultaneously infer (i) sparse precision matrices and (ii) the relevance of node-level information for uncovering the sought-after network structure. We encode such information as candidate auxiliary variables using a spike-and-slab submodel on the propensity of nodes to be hubs, which allows hypothesis-free selection and interpretation of a sparse subset of relevant variables. As efficient exploration of large posterior spaces is needed for real-world applications, we develop a variational expectation conditional maximization algorithm that scales inference to hundreds of samples, nodes and auxiliary variables. We illustrate and exploit the advantages of our approach in simulations and in a gene network study which identifies hub genes involved in biological pathways relevant to immune-mediated diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/bfbfe802cb5f3c9c2e7e580929c13fe21fe00c25" target='_blank'>
              A modeling framework for detecting and leveraging node-level information in Bayesian network inference.
              </a>
            </td>
          <td>
            Xiaoyue Xi, H. Ruffieux
          </td>
          <td>2024-06-25</td>
          <td>Biostatistics</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Interpretable mathematical expressions defining discrete-time dynamical systems (iterated maps) can model many phenomena of scientific interest, enabling a deeper understanding of system behaviors. Since formulating governing expressions from first principles can be difficult, it is of particular interest to identify expressions for iterated maps given only their data streams. In this work, we consider a modified Symbolic Artificial Neural Network-Trained Expressions (SymANNTEx) architecture for this task, an architecture more expressive than others in the literature. We make a modification to the model pipeline to optimize the regression, then characterize the behavior of the adjusted model in identifying several classical chaotic maps. With the goal of parsimony, sparsity-inducing weight regularization and information theory-informed simplification are implemented. We show that our modified SymANNTEx model properly identifies single-state maps and achieves moderate success in approximating a dual-state attractor. These performances offer significant promise for data-driven scientific discovery and interpretation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3502a1e8d5f9351f334ca3982d09f0f0f0433666" target='_blank'>
              Expressive Symbolic Regression for Interpretable Models of Discrete-Time Dynamical Systems
              </a>
            </td>
          <td>
            Adarsh Iyer, N. Boddupalli, Jeff Moehlis
          </td>
          <td>2024-06-05</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>5</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32686276485d13ba0f45ab92508b824a86c9f0fd" target='_blank'>
              Mixed effect estimation in deep compartment models: Variational methods outperform first-order approximations.
              </a>
            </td>
          <td>
            A. Janssen, Frank C Bennis, M. Cnossen, R. Mathôt
          </td>
          <td>2024-07-04</td>
          <td>Journal of pharmacokinetics and pharmacodynamics</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Generating ligand molecules for specific protein targets, known as structure-based drug design, is a fundamental problem in therapeutics development and biological discovery. Recently, target-aware generative models, especially diffusion models, have shown great promise in modeling protein-ligand interactions and generating candidate drugs. However, existing models primarily focus on learning the chemical distribution of all drug candidates, which lacks effective steerability on the chemical quality of model generations. In this paper, we propose a novel and general alignment framework to align pretrained target diffusion models with preferred functional properties, named AliDiff. AliDiff shifts the target-conditioned chemical distribution towards regions with higher binding affinity and structural rationality, specified by user-defined reward functions, via the preference optimization approach. To avoid the overfitting problem in common preference optimization objectives, we further develop an improved Exact Energy Preference Optimization method to yield an exact and efficient alignment of the diffusion models, and provide the closed-form expression for the converged distribution. Empirical studies on the CrossDocked2020 benchmark show that AliDiff can generate molecules with state-of-the-art binding energies with up to -7.07 Avg. Vina Score, while maintaining strong molecular properties.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e0d28b551dffb1dff8c2ce3b2f100ddc5b86a634" target='_blank'>
              Aligning Target-Aware Molecule Diffusion Models with Exact Energy Optimization
              </a>
            </td>
          <td>
            Siyi Gu, Minkai Xu, Alexander Powers, Weili Nie, Tomas Geffner, Karsten Kreis, J. Leskovec, Arash Vahdat, Stefano Ermon
          </td>
          <td>2024-07-01</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>134</td>
        </tr>

        <tr id="Background A number of deep learning models have been developed to predict epigenetic features such as chromatin accessibility from DNA sequence. Model evaluations commonly report performance genome-wide; however, cis regulatory elements (CREs), which play critical roles in gene regulation, make up only a small fraction of the genome. Furthermore, cell type specific CREs contain a large proportion of complex disease heritability. Results We evaluate genomic deep learning models in chromatin accessibility regions with varying degrees of cell type specificity. We assess two modeling directions in the field: general purpose models trained across thousands of outputs (cell types and epigenetic marks), and models tailored to specific tissues and tasks. We find that the accuracy of genomic deep learning models, including two state-of-the-art general purpose models – Enformer and Sei – varies across the genome and is reduced in cell type specific accessible regions. Using accessibility models trained on cell types from specific tissues, we find that increasing model capacity to learn cell type specific regulatory syntax – through single-task learning or high capacity multi-task models – can improve performance in cell type specific accessible regions. We also observe that improving reference sequence predictions does not consistently improve variant effect predictions, indicating that novel strategies are needed to improve performance on variants. Conclusions Our results provide a new perspective on the performance of genomic deep learning models, showing that performance varies across the genome and is particularly reduced in cell type specific accessible regions. We also identify strategies to maximize performance in cell type specific accessible regions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a185b92c9bb56abf7f4ab84ab12ddec3a037f4cc" target='_blank'>
              Current genomic deep learning models display decreased performance in cell type specific accessible regions
              </a>
            </td>
          <td>
            Pooja Kathail, Richard W. Shuai, Ryan Chung, C. Ye, Gabriel B. Loeb, Nilah M. Ioannidis
          </td>
          <td>2024-07-10</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="In the past two decades, machine learning potentials (MLPs) have driven significant developments in chemical, biological, and material sciences. The construction and training of MLPs enable fast and accurate simulations and analysis of thermodynamic and kinetic properties. This review focuses on the application of MLPs to reaction systems with consideration of bond breaking and formation. We review the development of MLP models, primarily with neural network and kernel-based algorithms, and recent applications of reactive MLPs (RMLPs) to systems at different scales. We show how RMLPs are constructed, how they speed up the calculation of reactive dynamics, and how they facilitate the study of reaction trajectories, reaction rates, free energy calculations, and many other calculations. Different data sampling strategies applied in building RMLPs are also discussed with a focus on how to collect structures for rare events and how to further improve their performance with active learning.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/fe08859388b658711cbf4ac096f214e5d2eca580" target='_blank'>
              Machine Learning of Reactive Potentials.
              </a>
            </td>
          <td>
            Yinuo Yang, Shuhao Zhang, Kavindri D. Ranasinghe, O. Isayev, A. Roitberg
          </td>
          <td>2024-06-01</td>
          <td>Annual review of physical chemistry</td>
          <td>0</td>
          <td>57</td>
        </tr>

        <tr id="Recent lineage tracing single-cell techniques (LT-scSeq), e.g., the Lineage And RNA RecoverY (LARRY) barcoding system, have enabled clonally resolved interpretation of differentiation trajectories. However, the heterogeneity of clone-specific kinetics remains understudied, both quantitatively and in terms of interpretability, thus limiting the power of bar-coding systems to unravel how heterogeneous stem cell clones drive overall cell population dynamics. Here, we present CLADES, a NeuralODE-based framework to faithfully estimate clone-specific kinetics of cell states from newly generated and publicly available human cord blood LARRY LT-scSeq data. By incorporating a stochastic simulation algorithm (SSA) and differential expression gene (DEGs) analysis, CLADES yields cell division dynamics across differentiation timecourses and fate bias predictions for the early progenitor cells. Moreover, clone-level quantitative behaviours can be grouped into characteristic types by pooling individual clones into meta-clones. By benchmarking with CoSpar, we found that CLADES improves fate bias prediction accuracy at the meta-clone level. In conclusion, we report a broadly applicable approach to robustly quantify differentiation kinetics using meta-clones while providing valuable insights into the fate bias of cellular populations for any organ system maintained by a pool of heterogeneous stem and progenitor cells.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/181cc1f16a3c5b14fb171a92ae7dc42356fcbe11" target='_blank'>
              Unveiling Clonal Cell Fate and Differentiation Dynamics: A Hybrid NeuralODE-Gillespie Approach
              </a>
            </td>
          <td>
            Mingze Gao, Melania Barile, Shirom Chabra, M. Haltalli, Emily F. Calderbank, Yiming Chao, E. Laurenti, B. Gottgens, Yuanhua Huang
          </td>
          <td>2024-07-11</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>34</td>
        </tr>

        <tr id="Our interest is in replicated network data with multiple networks observed across the same set of nodes. Examples include brain connection networks, in which nodes corresponds to brain regions and replicates to different individuals, and ecological networks, in which nodes correspond to species and replicates to samples collected at different locations and/or times. Our goal is to infer a hierarchical structure of the nodes at a population level, while performing multi-resolution clustering of the individual replicates. In brain connectomics, the focus is on inferring common relationships among the brain regions, while characterizing inter-individual variability in an easily interpretable manner. To accomplish this, we propose a Bayesian hierarchical model, while providing theoretical support in terms of identifiability and posterior consistency, and design efficient methods for posterior computation. We provide novel technical tools for proving model identifiability, which are of independent interest. Our simulations and application to brain connectome data provide support for the proposed methodology.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/aa4dd13bd0298bf88d6e8709bdc41fb4258c3107" target='_blank'>
              Bayesian Deep Generative Models for Replicated Networks with Multiscale Overlapping Clusters
              </a>
            </td>
          <td>
            Yuren Zhou, Yuqi Gu, D. Dunson
          </td>
          <td>2024-05-31</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Finding strong inhibitors of the BCL2 target, which is essential for controlling apoptosis and ensuring the survival of cancer cells, has prompted research into FDA-approved drugs. This study uses an advanced deep generative model called NeuralPlexer to produce protein-ligand complex conformations one-by-one and perform in silico analysis. This is the first time in the literature using NeuralPlexer in virtual screening, as we comprehensively evaluate the conformations of an FDA-approved drug library to ascertain their potential efficacy in suppressing BCL2 by utilizing NeuralPlexer’s capabilities. Obtained results were re-confirmed by physics-based molecular simulations and neural relational inference (NRI) analysis. Our study reveals several intriguing candidates such as Lathyrol and Fadrozole with potent inhibitory interactions with the BCL2 target, offering important new information for repurposing currently available drugs in cancer treatment. This work highlights the promise of deep learning technology in pharmaceutical research by integrating NeuralPlexer into the process of drug development, while also improving the accuracy of predictions made about protein-ligand interactions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7363a3e1594a6ca2a458a9fec09140ae5ed87f2d" target='_blank'>
              Deep Learning-Driven Discovery of FDA-Approved BCL2 Inhibitors: In Silico Analysis Using a Deep Generative Model NeuralPlexer for Drug Repurposing in Cancer Treatment
              </a>
            </td>
          <td>
            Ehsan Sayyah, Huseyin Tunc, S. Durdağı
          </td>
          <td>2024-07-17</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>36</td>
        </tr>

        <tr id="Many real-world applications demand accurate and fast predictions, as well as reliable uncertainty estimates. However, quantifying uncertainty on high-dimensional predictions is still a severely under-invested problem, especially when input-output relationships are non-linear. To handle this problem, the present work introduces an innovative approach that combines autoencoder deep neural networks with the probabilistic regression capabilities of Gaussian processes. The autoencoder provides a low-dimensional representation of the solution space, while the Gaussian process is a Bayesian method that provides a probabilistic mapping between the low-dimensional inputs and outputs. We validate the proposed framework for its application to surrogate modeling of non-linear finite element simulations. Our findings highlight that the proposed framework is computationally efficient as well as accurate in predicting non-linear deformations of solid bodies subjected to external forces, all the while providing insightful uncertainty assessments.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c5ebe3e35ead48c1cec6afd16f55a4873eabdbd1" target='_blank'>
              Gaussian process regression + deep neural network autoencoder for probabilistic surrogate modeling in nonlinear mechanics of solids
              </a>
            </td>
          <td>
            Saurabh Deshpande, Hussein Rappel, Mark Hobbs, Stéphane P. A. Bordas, Jakub Lengiewicz
          </td>
          <td>2024-07-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="In chemical engineering, process data are expensive to acquire, and complex phenomena are difficult to fully model. We explore the use of physics-informed neural networks (PINNs) for dynamic processes with incomplete mechanistic semi-explicit differential-algebraic equation systems and scarce process data. In particular, we focus on estimating states for which neither direct observational data nor constitutive equations are available. We propose an easy-to-apply heuristic to assess whether estimation of such states may be possible. As numerical examples, we consider a continuously stirred tank reactor and a liquid-liquid separator. We find that PINNs can infer unmeasured states with reasonable accuracy, and they generalize better in low-data scenarios than purely data-driven models. We thus show that PINNs are capable of modeling processes when relatively few experimental data and only partially known mechanistic descriptions are available, and conclude that they constitute a promising avenue that warrants further investigation.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7e4e0161b8d6befa3c8a0687b96bbfc9b32e40f7" target='_blank'>
              Physics-Informed Neural Networks for Dynamic Process Operations with Limited Physical Knowledge and Data
              </a>
            </td>
          <td>
            M. Velioglu, Song Zhai, Sophia Rupprecht, Alexander Mitsos, Andreas Jupke, M. Dahmen
          </td>
          <td>2024-06-03</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b8112c18b6d184d24cba044589140338bfc4b2d5" target='_blank'>
              Advanced methods for gene network identification and noise decomposition from single-cell data
              </a>
            </td>
          <td>
            Zhou Fang, Ankit Gupta, Sant Kumar, Mustafa Khammash
          </td>
          <td>2024-06-08</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>16</td>
        </tr>

        <tr id="Many clinical trials generate both longitudinal biomarker and time-to-event data. We might be interested in their relationship, as in the case of tumor size and overall survival in oncology drug development. Many well-established methods exist for analyzing such data either sequentially (two-stage models) or simultaneously (joint models). Two-stage modeling (2stgM) has been challenged (i) for not acknowledging that biomarkers are endogenous covariable to the survival submodel and (ii) for not propagating the uncertainty of the longitudinal biomarker submodel to the survival submodel. On the other hand, joint modeling (JM), which properly circumvents both problems, has been criticized for being time-consuming, and difficult to use in practice. In this paper, we explore a third approach, referred to as a novel two-stage modeling (N2stgM). This strategy reduces the model complexity without compromising the parameter estimate accuracy. The three approaches (2stgM, JM, and N2stgM) are formulated, and a Bayesian framework is considered for their implementation. Both real and simulated data were used to analyze the performance of such approaches. In all scenarios, our proposal estimated the parameters approximately as JM but without being computationally expensive, while 2stgM produced biased results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a1431f8841ec1dea194d5b4d31717f328d35e5b2" target='_blank'>
              Bridging the gap between two-stage and joint models: The case of tumor growth inhibition and overall survival models.
              </a>
            </td>
          <td>
            Danilo Alvares, F. Mercier
          </td>
          <td>2024-06-03</td>
          <td>Statistics in medicine</td>
          <td>1</td>
          <td>13</td>
        </tr>

        <tr id="A central aim in computational neuroscience is to relate the activity of large populations of neurons to an underlying dynamical system. Models of these neural dynamics should ideally be both interpretable and fit the observed data well. Low-rank recurrent neural networks (RNNs) exhibit such interpretability by having tractable dynamics. However, it is unclear how to best fit low-rank RNNs to data consisting of noisy observations of an underlying stochastic system. Here, we propose to fit stochastic low-rank RNNs with variational sequential Monte Carlo methods. We validate our method on several datasets consisting of both continuous and spiking neural data, where we obtain lower dimensional latent dynamics than current state of the art methods. Additionally, for low-rank models with piecewise linear nonlinearities, we show how to efficiently identify all fixed points in polynomial rather than exponential cost in the number of units, making analysis of the inferred dynamics tractable for large RNNs. Our method both elucidates the dynamical systems underlying experimental recordings and provides a generative model whose trajectories match observed trial-to-trial variability.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/579f56813a03517163b86fa89044dc78505bf2cb" target='_blank'>
              Inferring stochastic low-rank recurrent neural networks from neural data
              </a>
            </td>
          <td>
            Matthijs Pals, A. E. Saugtekin, Felix Pei, Manuel Gloeckler, J. H. Macke
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Drug-induced toxicity is one of the leading reasons new drugs fail clinical trials. Machine learning models that predict drug toxicity from molecular structure could help researchers prioritize less toxic drug candidates. However, current toxicity datasets are typically small and limited to a single organ system (e.g., cardio, renal, or liver). Creating these datasets often involved time-intensive expert curation by parsing drug label documents that can exceed 100 pages per drug. Here, we introduce UniTox, a unified dataset of 2,418 FDA-approved drugs with drug-induced toxicity summaries and ratings created by using GPT-4o to process FDA drug labels. UniTox spans eight types of toxicity: cardiotoxicity, liver toxicity, renal toxicity, pulmonary toxicity, hematological toxicity, dermatological toxicity, ototoxicity, and infertility. This is, to the best of our knowledge, the largest such systematic human in vivo database by number of drugs and toxicities, and the first covering nearly all FDA-approved medications for several of these toxicities. We recruited clinicians to validate a random sample of our GPT-4o annotated toxicities, and UniTox toxicity ratings concord with clinician labelers 87-96% of the time. Finally, we benchmark a graph neural network trained on UniTox to demonstrate the utility of this dataset for building molecular toxicity prediction models.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/e359f5059dfc3f10e24d2494dcd3f89838155872" target='_blank'>
              UniTox: Leveraging LLMs to Curate a Unified Dataset of Drug-Induced Toxicity from FDA Labels
              </a>
            </td>
          <td>
            J. Silberg, K. Swanson, E. Simon, A. Zhang, Z. Ghazizadeh, S. Ogden, H. Hamadeh, J. Zou
          </td>
          <td>2024-06-22</td>
          <td>None</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="In this paper, we present a guide to the foundations of learning Dynamic Bayesian Networks (DBNs) from data in the form of multiple samples of trajectories for some length of time. We present the formalism for a generic as well as a set of common types of DBNs for particular variable distributions. We present the analytical form of the models, with a comprehensive discussion on the interdependence between structure and weights in a DBN model and their implications for learning. Next, we give a broad overview of learning methods and describe and categorize them based on the most important statistical features, and how they treat the interplay between learning structure and weights. We give the analytical form of the likelihood and Bayesian score functions, emphasizing the distinction from the static case. We discuss functions used in optimization to enforce structural requirements. We briefly discuss more complex extensions and representations. Finally we present a set of comparisons in different settings for various distinct but representative algorithms across the variants.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/35c93e8d8300c924f0796c2d1e335ce010a7267e" target='_blank'>
              Learning Dynamic Bayesian Networks from Data: Foundations, First Principles and Numerical Comparisons
              </a>
            </td>
          <td>
            Vyacheslav Kungurtsev, Petr Rysavy, Fadwa Idlahcen, Pavel Rytír, Ales Wodecki
          </td>
          <td>2024-06-25</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Multiscale models provide a unique tool for studying complex processes that study events occurring at different scales across space and time. In the context of biological systems, such models can simulate mechanisms happening at the intracellular level such as signaling, and at the extracellular level where cells communicate and coordinate with other cells. They aim to understand the impact of genetic or environmental deregulation observed in complex diseases, describe the interplay between a pathological tissue and the immune system, and suggest strategies to revert the diseased phenotypes. The construction of these multiscale models remains a very complex task, including the choice of the components to consider, the level of details of the processes to simulate, or the fitting of the parameters to the data. One additional difficulty is the expert knowledge needed to program these models in languages such as C++ or Python, which may discourage the participation of non-experts. Simplifying this process through structured description formalisms — coupled with a graphical interface — is crucial in making modeling more accessible to the broader scientific community, as well as streamlining the process for advanced users. This article introduces three examples of multiscale models which rely on the framework PhysiBoSS, an add-on of PhysiCell that includes intracellular descriptions as continuous time Boolean models to the agent-based approach. The article demonstrates how to easily construct such models, relying on PhysiCell Studio, the PhysiCell Graphical User Interface. A step-by-step tutorial is provided as a Supplementary Material and all models are provided at: https://physiboss.github.io/tutorial/.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/339881abe751b4862349d52c646aba9e4910f23d" target='_blank'>
              Building multiscale models with PhysiBoSS, an agent-based modeling tool
              </a>
            </td>
          <td>
            Marco Ruscone, Andrea Checcoli, Randy W. Heiland, Emmanuel Barillot, P. Macklin, Laurence Calzone, Vincent Noël
          </td>
          <td>2024-06-26</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>30</td>
        </tr>

        <tr id="Proteins are complex molecules responsible for different functions in nature. Enhancing the functionality of proteins and cellular fitness can significantly impact various industries. However, protein optimization using computational methods remains challenging, especially when starting from low-fitness sequences. We propose LatProtRL, an optimization method to efficiently traverse a latent space learned by an encoder-decoder leveraging a large protein language model. To escape local optima, our optimization is modeled as a Markov decision process using reinforcement learning acting directly in latent space. We evaluate our approach on two important fitness optimization tasks, demonstrating its ability to achieve comparable or superior fitness over baseline methods. Our findings and in vitro evaluation show that the generated sequences can reach high-fitness regions, suggesting a substantial potential of LatProtRL in lab-in-the-loop scenarios.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a9589c7a5062134012d56bf3ea6aedd1d76bfb1d" target='_blank'>
              Robust Optimization in Protein Fitness Landscapes Using Reinforcement Learning in Latent Space
              </a>
            </td>
          <td>
            Minji Lee, L. Vecchietti, Hyunkyu Jung, Hyun Joo Ro, Meeyoung Cha, Ho Min Kim
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="In the rapidly evolving landscape of Automated Driving Systems (ADS), the dynamic nature of driving environments poses significant challenges to traditional static risk assessment models. Addressing this, our paper introduces an approach focused on predicting severity and controllability ratings using context-specific risk features. We have developed a model that leverages the strengths of Random Forest (RF) and Gradient Boosting Decision Trees (GBDT) to adeptly navigate the complexities of diverse driving conditions.Central to our approach is the effective processing and analysis of data derived from risk-specific contexts, converting this information into actionable risk features through well-established mathematical formulations. This model excels in contextual understanding, providing a more detailed and accurate risk assessment compared to traditional methods. Unlike subjective approaches, our model offers an objective and data-driven analysis. A key implementation of our model is showcased in an Adaptive Cruise Control (ACC) system, particularly in highway lane-following scenarios under various driving conditions. This application highlights the model’s capability to dynamically assess risks and prioritize factors, thereby substantially improving Dynamic Risk Assessment (DRA) in ADS.In essence, our research successfully bridges the gap between static risk models and the dynamic nature of driving environments. By integrating ensemble learning models like RF and GBDT, we present an advanced and dynamic tool for risk assessment in ADS, marking an improvement in the field of automotive safety.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c7fc1a58cba2f92fe8fc5eb8213ab1b099cdb391" target='_blank'>
              Dynamic Risk Assessment: Leveraging Ensemble Learning for Context-Specific Risk Features
              </a>
            </td>
          <td>
            A. Patel, Kunjkumar Thummar, Peter Liggesmeyer
          </td>
          <td>2024-06-02</td>
          <td>2024 IEEE Intelligent Vehicles Symposium (IV)</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="Through integrating the evolutionary correlations across global states in the bidirectional recursion, an explainable Bayesian recurrent neural smoother (EBRNS) is proposed for offline data-assisted fixed-interval state smoothing. At first, the proposed model, containing global states in the evolutionary interval, is transformed into an equivalent model with bidirectional memory. This transformation incorporates crucial global state information with support for bi-directional recursive computation. For the transformed model, the joint state-memory-trend Bayesian filtering and smoothing frameworks are derived by introducing the bidirectional memory iteration mechanism and offline data into Bayesian estimation theory. The derived frameworks are implemented using the Gaussian approximation to ensure analytical properties and computational efficiency. Finally, the neural network modules within EBRNS and its two-stage training scheme are designed. Unlike most existing approaches that artificially combine deep learning and model-based estimation, the bidirectional recursion and internal gated structures of EBRNS are naturally derived from Bayesian estimation theory, explainably integrating prior model knowledge, online measurement, and offline data. Experiments on representative real-world datasets demonstrate that the high smoothing accuracy of EBRNS is accompanied by data efficiency and a lightweight parameter scale.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/57349533774d9106325c9d199314485305e9ade6" target='_blank'>
              Explainable Bayesian Recurrent Neural Smoother to Capture Global State Evolutionary Correlations
              </a>
            </td>
          <td>
            Shi Yan, Yan Liang, Huayu Zhang, Le Zheng, Difan Zou, Binglu Wang
          </td>
          <td>2024-06-17</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="The COVID-19 pandemic has significantly impacted global health, resulting in numerous fatalities and presenting substantial challenges to national healthcare systems due to a sharp increase in cases. Key to managing this crisis is the rapid and accurate identification of COVID-19 infections, a task that can be enhanced with Machine Learning (ML) techniques. However, ML applications can also generate biased and potentially unfair outcomes for certain demographic groups. This paper introduces a ML model designed for detecting both COVID-19 cases and biases associated with specific patient attributes. The model employs Decision Tree and XGBoost algorithms for case detection, while bias analysis is performed using the DALEX library, which focuses on protected attributes such as age, gender, race, and ethnicity. DALEX works by creating an "explainer" object that represents the model, enabling exploration of the model's functions without requiring in-depth knowledge of its workings. This approach helps pinpoint influential attributes and uncover potential biases within the model. Model performance is assessed through accuracy metrics, with the Decision Tree algorithm achieving the highest accuracy at 99% following Bayesian hyperparameter optimization. However, high accuracy does not ensure fairness, as biases related to protected attributes may still persist.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/58d7007642a823154e2cacfd8a9efed7afe91302" target='_blank'>
              Detection of Bias in Machine Learning Models for Predicting Deaths Caused by COVID-19
              </a>
            </td>
          <td>
            Fatimatus Zachra, Setio Basuki
          </td>
          <td>2024-06-30</td>
          <td>Jurnal ELTIKOM</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Antibiotic resistance, particularly among Gram-negative bacteria, poses a significant healthcare challenge due to their ability to evade antibiotic action through various mechanisms. In this study, we explore the prediction of small molecule accumulation in Gram-negative bacteria by using machine learning techniques enhanced with statistical descriptors derived from molecular dynamics simulations. We begin by identifying a minimal set of molecular descriptors that maximize the model's predictive power while preserving human interpretability. We optimize model accuracy, precision, and the area under the receiver operating characteristic curve through an iterative process. We demonstrate that the inclusion of statistical descriptors significantly improves model performance across various prediction metrics. Particularly, the addition of statistical descriptors related to dipole moment and minimum projection radius enhances the model's predictive capabilities, shedding light on the physicochemical properties crucial for small molecule accumulation. Our findings highlight the importance of considering statistical moments beyond mean values in predictive modeling and suggest avenues for future research. Overall, our study provides insights into the complex dynamics of antibiotic accumulation in Escherichia coli bacterial cells, generalizable to other Gram-negative species, offering a promising approach for the discovery of effective antibacterial agents, identifying new hits, and improving them to define effective lead agents.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2333a64256d0c89d90b425ee3579e4557a652710" target='_blank'>
              Machine Learning Prediction of Small Molecule Accumulation in Escherichia Coli Enhanced with Descriptor Statistics.
              </a>
            </td>
          <td>
            Stefan Milenkovic, Sara Boi, Mariano A. Scorciapino, I. Bodrenko, Matteo Ceccarelli
          </td>
          <td>2024-07-08</td>
          <td>Journal of chemical theory and computation</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="Therapeutic interventions are designed to perturb the function of a biological system. However, there are many types of proteins that cannot be targeted with conventional small molecule drugs. Accordingly, many identified gene-regulatory drivers and downstream effectors are currently undruggable. Drivers and effectors are often connected by druggable signaling and regulatory intermediates. Methods to identify druggable intermediates therefore have general value in expanding the set of targets available for hypothesis-driven validation. Here we identify and prioritize potential druggable intermediates by developing a network perturbation theory, termed NetPert, for response functions of biological networks. Dynamics are defined by a network structure in which vertices represent genes and proteins, and edges represent gene-regulatory interactions and protein-protein interactions. Perturbation theory for network dynamics prioritizes targets that interfere with signaling from driver to response genes. Applications to organoid models for metastatic breast cancer demonstrate the ability of this mathematical framework to identify and prioritize druggable intermediates. While the short-time limit of the perturbation theory resembles betweenness centrality, NetPert is superior in generating target rankings that correlate with previous wet-lab assays and are more robust to incomplete or noisy network data. NetPert also performs better than a related graph diffusion approach. Wet-lab assays demonstrate that drugs for targets identified by NetPert, including targets that are not themselves differentially expressed, are active in suppressing additional metastatic phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8e0d901129a82304d94972066bb691c9c95e075f" target='_blank'>
              Prioritizing drug targets by perturbing biological network response functions
              </a>
            </td>
          <td>
            Matthew C Perrone, Michael G Lerner, Matthew Dunworth, Andrew J. Ewald, Joel S Bader
          </td>
          <td>2024-06-01</td>
          <td>PLOS Computational Biology</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="During the 2022–2023 unprecedented mpox epidemic, near real-time short-term forecasts of the epidemic’s trajectory were essential in intervention implementation and guiding policy. However, as case levels have significantly decreased, evaluating model performance is vital to advancing the field of epidemic forecasting. Using laboratory-confirmed mpox case data from the Centers for Disease Control and Prevention and Our World in Data teams, we generated retrospective sequential weekly forecasts for Brazil, Canada, France, Germany, Spain, the United Kingdom, the United States and at the global scale using an auto-regressive integrated moving average (ARIMA) model, generalized additive model, simple linear regression, Facebook’s Prophet model, as well as the sub-epidemic wave and n-sub-epidemic modelling frameworks. We assessed forecast performance using average mean squared error, mean absolute error, weighted interval scores, 95% prediction interval coverage, skill scores and Winkler scores. Overall, the n-sub-epidemic modelling framework outcompeted other models across most locations and forecasting horizons, with the unweighted ensemble model performing best most frequently. The n-sub-epidemic and spatial-wave frameworks considerably improved in average forecasting performance relative to the ARIMA model (greater than 10%) for all performance metrics. Findings further support sub-epidemic frameworks for short-term forecasting epidemics of emerging and re-emerging infectious diseases.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/439e78425773c1891f76b1f1247c998a52c94d80" target='_blank'>
              Evaluating the forecasting performance of ensemble sub-epidemic frameworks and other time series models for the 2022–2023 mpox epidemic
              </a>
            </td>
          <td>
            A. Bleichrodt, R. Luo, A. Kirpich, Gerardo Chowell
          </td>
          <td>2024-07-01</td>
          <td>Royal Society Open Science</td>
          <td>0</td>
          <td>13</td>
        </tr>

        <tr id="Identifying genes that display spatial patterns is critical to investigating expression interactions within a spatial context and further dissecting biological understanding of complex mechanistic functionality. Despite the increase in statistical methods designed to identify spatially variable genes, they are mostly based on marginal analysis and share the limitation that the dependence (network) structures among genes are not well accommodated, where a biological process usually involves changes in multiple genes that interact in a complex network. Moreover, the latent cellular composition within spots may introduce confounding variations, negatively affecting identification accuracy. In this study, we develop a novel Bayesian regularization approach for spatial transcriptomic data, with the confounding variations induced by varying cellular distributions effectively corrected. Significantly advancing from the existing studies, a thresholded graph Laplacian regularization is proposed to simultaneously identify spatially variable genes and accommodate the network structure among genes. The proposed method is based on a zero-inflated negative binomial distribution, effectively accommodating the count nature, zero inflation, and overdispersion of spatial transcriptomic data. Extensive simulations and the application to real data demonstrate the competitive performance of the proposed method.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/60c08516985e7b922d5cf162cca65398e4a54521" target='_blank'>
              Joint identification of spatially variable genes via a network-assisted Bayesian regularization approach
              </a>
            </td>
          <td>
            Mingcong Wu, Yang Li, Shuangge Ma, Mengyun Wu
          </td>
          <td>2024-07-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Deep learning is a subfield of artificial intelligence and machine learning based mostly on neural networks and often combined with attention algorithms that has been used to detect and identify objects in text, audio, images, and video. Serghiou and Rough (Am J Epidemiol. 0000;000(00):0000-0000) present a primer for epidemiologists on deep learning models. These models provide substantial opportunities for epidemiologists to expand and amplify their research in both data collection and analyses by increasing the geographic reach of studies, including more research subjects, and working with large or high dimensional data. The tools for implementing deep learning methods are not quite yet as straightforward or ubiquitous for epidemiologists as traditional regression methods found in standard statistical software, but there are exciting opportunities for interdisciplinary collaboration with deep learning experts, just as epidemiologists have with statisticians, healthcare providers, urban planners, and other professionals. Despite the novelty of these methods, epidemiological principles of assessing bias, study design, interpretation and others still apply when implementing deep learning methods or assessing the findings of studies that have used them.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/24ccadf30b95ea23fed03f6ca4a3820748a650f8" target='_blank'>
              Deep Learning - Methods to Amplify Epidemiological Data Collection and Analyses.
              </a>
            </td>
          <td>
            D. Alex Quistberg, Stephen J. Mooney, Tolga Tasdizen, Pablo Arbelaez, Quynh C. Nguyen
          </td>
          <td>2024-07-16</td>
          <td>American journal of epidemiology</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/98c47eca070ef4fc936f0fdb8bd32925e209619e" target='_blank'>
              Self-consistent signal transduction analysis for modeling context-specific signaling cascades and perturbations
              </a>
            </td>
          <td>
            John Cole
          </td>
          <td>2024-07-19</td>
          <td>npj Systems Biology and Applications</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Small molecules play a pivotal role in modern medicine, and scrutinizing their interactions with protein targets is essential for the discovery and development of novel, life-saving therapeutics. The term"bioactivity"encompasses various biological effects resulting from these interactions, including both binding and functional responses. The magnitude of bioactivity dictates the therapeutic or toxic pharmacological outcomes of small molecules, rendering accurate bioactivity prediction crucial for the development of safe and effective drugs. However, existing structural datasets of small molecule-protein interactions are often limited in scale and lack systematically organized bioactivity labels, thereby impeding our understanding of these interactions and precise bioactivity prediction. In this study, we introduce a comprehensive dataset of small molecule-protein interactions, consisting of over a million binding structures, each annotated with real biological activity labels. This dataset is designed to facilitate unbiased bioactivity prediction. We evaluated several classical models on this dataset, and the results demonstrate that the task of unbiased bioactivity prediction is challenging yet essential.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/952f37bafb072cc8958cea93b6e7ccc0443d8e76" target='_blank'>
              SIU: A Million-Scale Structural Small Molecule-Protein Interaction Dataset for Unbiased Bioactivity Prediction
              </a>
            </td>
          <td>
            Yanwen Huang, Bowen Gao, Yinjun Jia, Hongbo Ma, Wei-Ying Ma, Ya-Qin Zhang, Yanyan Lan
          </td>
          <td>2024-06-13</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4c5916a879490d67fb48ad872b52d27539ca4a54" target='_blank'>
              Deep representation learning of chemical-induced transcriptional profile for phenotype-based drug discovery
              </a>
            </td>
          <td>
            X. Tong, Ning Qu, Xiangtai Kong, Shengkun Ni, Jingyi Zhou, Kun Wang, Lehan Zhang, Yiming Wen, Jiangshan Shi, Sulin Zhang, Xutong Li, Mingyue Zheng
          </td>
          <td>2024-06-25</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>12</td>
        </tr>

        <tr id="With the development of high-throughput technologies, genomics datasets rapidly grow in size, including functional genomics data. This has allowed the training of large Deep Learning (DL) models to predict epigenetic readouts, such as protein binding or histone modifications, from genome sequences. However, large dataset sizes come at a price of data consistency, often aggregating results from a large number of studies, conducted under varying experimental conditions. While data from large-scale consortia are useful as they allow studying the effects of different biological conditions, they can also contain unwanted biases from confounding experimental factors. Here, we introduce Metadata-guided Feature Disentanglement (MFD) - an approach that allows disentangling biologically relevant features from potential technical biases. MFD incorporates target metadata into model training, by conditioning weights of the model output layer on different experimental factors. It then separates the factors into disjoint groups and enforces independence of the corresponding feature subspaces with an adversarially learned penalty. We show that the metadata-driven disentanglement approach allows for better model introspection, by connecting latent features to experimental factors, without compromising, or even improving performance in downstream tasks, such as enhancer prediction, or genetic variant discovery. The code for our implemementation is available at https://github.com/HealthML/MFD">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/32dd14e31f771ca226153c9210adaac58da256a4" target='_blank'>
              Metadata-guided Feature Disentanglement for Functional Genomics
              </a>
            </td>
          <td>
            Alexander Rakowski, Remo Monti, Viktoriia Huryn, Marta Lemanczyk, Uwe Ohler, Christoph Lippert
          </td>
          <td>2024-05-29</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Recent advancements in single-cell genomics necessitate precision in gene panel selection to interpret complex biological data effectively. Those methods aim to streamline the analysis of scRNA-seq data by focusing on the most informative genes that contribute significantly to the specific analysis task. Traditional selection methods, which often rely on expert domain knowledge, embedded machine learning models, or heuristic-based iterative optimization, are prone to biases and inefficiencies that may obscure critical genomic signals. Recognizing the limitations of traditional methods, we aim to transcend these constraints with a refined strategy. In this study, we introduce an iterative gene panel selection strategy that is applicable to clustering tasks in single-cell genomics. Our method uniquely integrates results from other gene selection algorithms, providing valuable preliminary boundaries or prior knowledge as initial guides in the search space to enhance the efficiency of our framework. Furthermore, we incorporate the stochastic nature of the exploration process in reinforcement learning (RL) and its capability for continuous optimization through reward-based feedback. This combination mitigates the biases inherent in the initial boundaries and harnesses RL's adaptability to refine and target gene panel selection dynamically. To illustrate the effectiveness of our method, we conducted detailed comparative experiments, case studies, and visualization analysis.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f8e89dc4802b5da2d3584b95250446f7b5148b7c" target='_blank'>
              Enhanced Gene Selection in Single-Cell Genomics: Pre-Filtering Synergy and Reinforced Optimization
              </a>
            </td>
          <td>
            Weiliang Zhang, Zhen Meng, Dongjie Wang, Min Wu, Kunpeng Liu, Yuanchun Zhou, Meng Xiao
          </td>
          <td>2024-06-11</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>11</td>
        </tr>

        <tr id="Histone deacetylase 6 (HDAC6) is a promising drug target for the treatment of human diseases such as cancer, neurodegenerative diseases (in particular, Alzheimer's disease), and multiple sclerosis. Considerable attention is paid to the development of selective non-toxic HDAC6 inhibitors. To this end, we successfully form a set of 3854 compounds and proposed adequate regression QSAR models for HDAC6 inhibitors. The models have been developed using the PubChem, Klekota-Roth, 2D atom pair fingerprints, and RDkit descriptors and the gradient boosting, support vector machines, neural network, and k-nearest neighbours methods. The models are integrated into the developed HT_PREDICT application, which is freely available at https://htpredict.streamlit.app/. In vitro studies have confirmed the predictive ability of the proposed QSAR models integrated into the HT_PREDICT web application. In addition, the virtual screening performed with the HT_PREDICT web application allowed us to propose two promising inhibitors for further investigations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7ade705c2cda434811a04f6bdd3591f00719f55b" target='_blank'>
              HT_PREDICT: a machine learning-based computational open-source tool for screening HDAC6 inhibitors.
              </a>
            </td>
          <td>
            O. Tinkov, V. Osipov, A. Kolotaev, D. Khachatryan, V. Y. Grigorev
          </td>
          <td>2024-06-01</td>
          <td>SAR and QSAR in environmental research</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Genotype-to-phenotype mapping is an essential problem in the current genomic era. While qualitative case-control predictions have received significant attention, less emphasis has been placed on predicting quantitative phenotypes. This emerging field holds great promise in revealing intricate connections between microbial communities and host health. However, the presence of heterogeneity in microbiome datasets poses a substantial challenge to the accuracy of predictions and undermines the reproducibility of models. To tackle this challenge, we investigated 22 normalization methods that aimed at removing heterogeneity across multiple datasets, conducted a comprehensive review of them, and evaluated their effectiveness in predicting quantitative phenotypes in three simulation scenarios and 31 real datasets. The results indicate that none of these methods demonstrate significant superiority in predicting quantitative phenotypes or attain a noteworthy reduction in Root Mean Squared Error (RMSE) of the predictions. Given the frequent occurrence of batch effects and the satisfactory performance of batch correction methods in predicting datasets affected by these effects, we strongly recommend utilizing batch correction methods as the initial step in predicting quantitative phenotypes. In summary, the performance of normalization methods in predicting metagenomic data remains a dynamic and ongoing research area. Our study contributes to this field by undertaking a comprehensive evaluation of diverse methods and offering valuable insights into their effectiveness in predicting quantitative phenotypes.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/9a680651a3cf41ba9343ce9f9c1efc81ca6d5163" target='_blank'>
              Evaluation of normalization methods for predicting quantitative phenotypes in metagenomic data analysis
              </a>
            </td>
          <td>
            Beibei Wang, Yihui Luan
          </td>
          <td>2024-06-05</td>
          <td>Frontiers in Genetics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="Immune checkpoint inhibitors (ICIs) have revolutionized cancer treatment, still most patients experience limited or no clinical benefit. The immune component dynamics during treatment and how it correlated with treatment efficacy remained largely known. We hypothesize that understanding the modulation dynamics of the TME upon ICI treatment would be key to elucidate the mechanisms of response and resistance. To systematically characterize these dynamics, we aggregated a large-scale single-cell RNA-sequencing (scRNA-seq) dataset of 141 patients across multiple cancer types and ICI treatment modalities, with longitudinally paired samples. We developed a robust, integration-free, and cluster-free deep phenotyping framework, leveraging multiple pan-cancer references to annotate 876,410 high-quality cells into 80 granular cell states. Our analysis revealed consistent compositional changes in 17 cell subtypes following ICI treatment, including an enhanced adaptive immune response such as CD8+ IL7R+ central memory T cells, GZMK+ effector memory T cells, germinal center B cells, precursor exhausted T cells, and CD4+ TfhTh1 cells, accompanied by a broad reduction in interferon responsive immune cells quiescent CD4+CREM-memory T cells. We uncovered co-regulated cell communities within the TME, highlighting the coordinated interplay between adaptive and innate immune cells, as well as immune and non-immune components. Furthermore, we identified two distinct patient groups exhibiting tightly correlated cellular dynamics within the TME post-treatment. The first group, enriched for responders, displayed a marked expansion of naive lymphocytes, while the second group, predominantly composed of non-responders, showed an increased abundance of immune experienced/suppressive cell states. This dichotomy in TME dynamics offers a potential predictive biomarker for patient stratification and personalized therapeutic strategies. Our study presents a comprehensive landscape of the cellular dynamics within the TME during ICI treatment, enabled by a powerful deep phenotyping approach showcasing the importance of a systems-level understanding of the TME dynamics in improving patient stratification and advancing personalized cancer immunotherapy.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/a5e90093265f7348aad7b591e51e89915aa6d67b" target='_blank'>
              Cellular Dynamics Upon Immune Checkpoint Inhibition
              </a>
            </td>
          <td>
            Zhongyang Lin, Eketerina Petrenko, Mitalee Chandra, Nalini Srinivas, Selma Ugurel, Jürgen C. Becker, Dvir Aran
          </td>
          <td>2024-07-02</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>7</td>
        </tr>

        <tr id="Drug repositioning is a strategy of repurposing approved drugs for treating new indications, which can accelerate the drug discovery process, reduce development costs, and lower the safety risk. The advancement of biotechnology has significantly accelerated the speed and scale of biological data generation, offering significant potential for drug repositioning through biomedical knowledge graphs that integrate diverse entities and relations from various biomedical sources. To fully learn the semantic information and topological structure information from the biological knowledge graph, we propose a knowledge graph convolutional network with a heuristic search, named KGCNH, which can effectively utilize the diversity of entities and relationships in biological knowledge graphs, as well as topological structure information, to predict the associations between drugs and diseases. Specifically, we design a relation-aware attention mechanism to compute the attention scores for each neighboring entity of a given entity under different relations. To address the challenge of randomness of the initial attention scores potentially impacting model performance and to expand the search scope of the model, we designed a heuristic search module based on Gumbel-Softmax, which uses attention scores as heuristic information and introduces randomness to assist the model in exploring more optimal embeddings of drugs and diseases. Following this module, we derive the relation weights, obtain the embeddings of drugs and diseases through neighborhood aggregation, and then predict drug-disease associations. Additionally, we employ feature-based augmented views to enhance model robustness and mitigate overfitting issues. We have implemented our method and conducted experiments on two data sets. The results demonstrate that KGCNH outperforms competing methods. In particular, case studies on lithium and quetiapine confirm that KGCNH can retrieve more actual drug-disease associations in the top prediction results.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3d972c669224e3e3436cbfc83c2a78a67493378c" target='_blank'>
              Knowledge Graph Convolutional Network with Heuristic Search for Drug Repositioning.
              </a>
            </td>
          <td>
            Xiang Du, Xingliang Sun, Min Li
          </td>
          <td>2024-06-05</td>
          <td>Journal of chemical information and modeling</td>
          <td>0</td>
          <td>2</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/23356ea102a5d24c6a10dc65b1b52d1036f20fb3" target='_blank'>
              Application of machine learning models for property prediction to targeted protein degraders
              </a>
            </td>
          <td>
            Giulia Peteani, Minh Tam Davide Huynh, Grégori Gerebtzoff, Raquel Rodríguez-Pérez
          </td>
          <td>2024-07-09</td>
          <td>Nature Communications</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="Deep learning has been increasingly popular in omics data analysis. Recent works incorporating variable selection into deep learning have greatly enhanced the model’s interpretability. However, because deep learning desires a large sample size, the existing methods may result in uncertain findings when the dataset has a small sample size, commonly seen in omics data analysis. With the explosion and availability of omics data from multiple populations/studies, the existing methods naively pool them into one dataset to enhance the sample size while ignoring that variable structures can differ across datasets, which might lead to inaccurate variable selection results. We propose a penalized integrative deep neural network (PIN) to simultaneously select important variables from multiple datasets. PIN directly aggregates multiple datasets as input and considers both homogeneity and heterogeneity situations among multiple datasets in an integrative analysis framework. Results from extensive simulation studies and applications of PIN to gene expression datasets from elders with different cognitive statuses or ovarian cancer patients at different stages demonstrate that PIN outperforms existing methods with considerably improved performance among multiple datasets. The source code is freely available on Github (rucliyang/PINFunc). We speculate that the proposed PIN method will promote the identification of disease‐related important variables based on multiple studies/datasets from diverse origins.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1667454b3841428b4c868841dbc397f6a3e10d6" target='_blank'>
              A penalized integrative deep neural network for variable selection among multiple omics datasets
              </a>
            </td>
          <td>
            Yang Li, Xiaonan Ren, Haochen Yu, Tao Sun, Shuangge Ma
          </td>
          <td>2024-06-07</td>
          <td>Quantitative Biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="
Fusing known physics into data-driven learners allows modelling practitioners to combine the expressive power of traditional machine learning with known mechanistic laws, where the objective is to enhance predictive performance, interpretability, and model generalisation. A core consideration that must be made when implementing a physics-informed learning architecture is how relevant knowledge will be embedded into the model structure, which, generally, is informed by the type of physics that is available. Frequently this knowledge may not be complete, with only a partial understanding of the governing physics available. 

In this work, possible paths for deriving Gaussian process kernels that are representative of partial knowledge will be considered. How the type of knowledge that is possessed influences the derivation will be explored, particularly when there is the potential for some aspect of misspecified physics. An example of deriving partially structured kernels will be investigated for modelling the decoupled response of a GARTEUR laboratory aircraft structure, where the derived kernels are used to decompose the dynamics of the aircraft into modal contributions. 

">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/da9bbf07470b8cc6d6606c4b51a040b8e45ce3d6" target='_blank'>
              Gaussian process kernels for partial physical insight
              </a>
            </td>
          <td>
            Matthew R. Jones, D. J. Pitchforth, E. J. Cross
          </td>
          <td>2024-07-01</td>
          <td>e-Journal of Nondestructive Testing</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Computational prediction of the interaction of T cell receptors (TCRs) and their ligands is a grand challenge in immunology. Despite advances in high-throughput assays, specificity-labelled TCR data remains sparse. In other domains, the pre-training of language models on unlabelled data has been successfully used to address data bottlenecks. However, it is unclear how to best pre-train protein language models for TCR specificity prediction. Here we introduce a TCR language model called SCEPTR (Simple Contrastive Embedding of the Primary sequence of T cell Receptors), capable of data-efficient transfer learning. Through our model, we introduce a novel pre-training strategy combining autocontrastive learning and masked-language modelling, which enables SCEPTR to achieve its state-of-the-art performance. In contrast, existing protein language models and a variant of SCEPTR pre-trained without autocontrastive learning are outperformed by sequence alignment-based methods. We anticipate that contrastive learning will be a useful paradigm to decode the rules of TCR specificity.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/61bbb71fad60d2297bcf0ace69de8e1cd0bf9b43" target='_blank'>
              Contrastive learning of T cell receptor representations
              </a>
            </td>
          <td>
            Yuta Nagano, Andrew Pyo, Martina Milighetti, James Henderson, John Shawe-Taylor, Benjamin Chain, Andreas Tiffeau-Mayer
          </td>
          <td>2024-06-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/62559a2f08e304d5a6149f4605e45529ac2c150e" target='_blank'>
              Multi-modal generative modeling for joint analysis of single-cell T cell receptor and gene expression data
              </a>
            </td>
          <td>
            F. Drost, Yang An, Irene Bonafonte-Pardàs, L. Dratva, Rik G. H. Lindeboom, M. Haniffa, Sarah A. Teichmann, F. Theis, M. Lotfollahi, Benjamin Schubert
          </td>
          <td>2024-07-03</td>
          <td>Nature Communications</td>
          <td>1</td>
          <td>66</td>
        </tr>

        <tr id="Radiobiological data, whether obtained at the clinical, biological or molecular level has significantly contributed to a better description and prediction of the individual dose-response to ionizing radiation and a better estimation of the radiation-induced risks. Particularly, over the last seventy years, the amount of radiobiological data has considerably increased, and permitted the mathematical formulas describing dose-response to become less empirical. A better understanding of the basic radiobiological mechanisms has also contributed to establish quantitative inter-correlations between clinical, biological and molecular biomarkers, refining again the mathematical models of description. Today, big data approaches and, more recently, artificial intelligence may finally complete and secure this long process of thinking from the multi-scale description of radiation-induced events to their prediction. Here, we reviewed the major dose-response models applied in radiobiology for quantifying molecular and cellular radiosensitivity and aimed to explain their evolution: Specifically, we highlighted the advances concerning the target theory with the cell survival models and the progressive introduction of the DNA repair process in the mathematical models. Furthermore, we described how the technological advances have changed the description of DNA double-strand break (DSB) repair kinetics by introducing the important notion of DSB recognition, independent of that of DSB repair. Initially developed separately, target theory on one hand and, DSB recognition and repair, on the other hand may be now fused into a unified model involving the cascade of phosphorylations mediated by the ATM kinase in response to any genotoxic stress.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8cdbdf7288e7e261d821e90454903709c5fbfc0e" target='_blank'>
              Seventy Years of Dose-response Models: From the Target Theory to the Use of Big Databases Involving Cell Survival and DNA Repair.
              </a>
            </td>
          <td>
            L. Bodgi, Laurent Pujo-Menjouet, Audrey Bouchet, Michel Bourguignon, N. Foray
          </td>
          <td>2024-05-28</td>
          <td>Radiation research</td>
          <td>0</td>
          <td>37</td>
        </tr>

        <tr id="Computational biology involves applying computer science and informatics techniques in biology to understand complex biological data. It allows us to collect, connect, and analyze biological data at a large scale and build predictive models. In the twenty first century, computational resources along with Artificial Intelligence (AI) have been widely used in various fields of biological sciences such as biochemistry, structural biology, immunology, microbiology, and genomics to handle massive data for decision-making, including in applications such as drug design and vaccine development, one of the major areas of focus for human and animal welfare. The knowledge of available computational resources and AI-enabled tools in vaccine design and development can improve our ability to conduct cutting-edge research. Therefore, this review article aims to summarize important computational resources and AI-based tools. Further, the article discusses the various applications and limitations of AI tools in vaccine development.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1c9123caa6702ec925f4ac25dcf8d43c358e9940" target='_blank'>
              Vaccine design and development: Exploring the interface with computational biology and AI.
              </a>
            </td>
          <td>
            Ananya, Darshan C Panchariya, Anandakrishnan Karthic, S. P. Singh, Ashutosh Mani, A. Chawade, Sandeep Kushwaha
          </td>
          <td>2024-07-10</td>
          <td>International reviews of immunology</td>
          <td>0</td>
          <td>25</td>
        </tr>

        <tr id="Nonlinear and non-stationary processes are prevalent in various natural and physical phenomena, where system dynamics can change qualitatively due to bifurcation phenomena. Traditional machine learning methods have advanced our ability to learn and predict such systems from observed time series data. However, predicting the behavior of systems with temporal parameter variations without knowledge of true parameter values remains a significant challenge. This study leverages the reservoir computing framework to address this problem by unsupervised extraction of slowly varying system parameters from time series data. We propose a model architecture consisting of a slow reservoir with long timescale internal dynamics and a fast reservoir with short timescale dynamics. The slow reservoir extracts the temporal variation of system parameters, which are then used to predict unknown bifurcations in the fast dynamics. Through experiments using data generated from chaotic dynamical systems, we demonstrate the ability to predict bifurcations not present in the training data. Our approach shows potential for applications in fields such as neuroscience, material science, and weather prediction, where slow dynamics influencing qualitative changes are often unobservable.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1f7ba13092aa22ef11ae95641fe8961d09c8fcd7" target='_blank'>
              Prediction of Unobserved Bifurcation by Unsupervised Extraction of Slowly Time-Varying System Parameter Dynamics from Time Series Using Reservoir Computing
              </a>
            </td>
          <td>
            Keita Tokuda, Yuichi Katori
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>10</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/4e135484519a0cd156be5938f03561932564edee" target='_blank'>
              Predicting herb-disease associations using network-based measures in human protein interactome
              </a>
            </td>
          <td>
            Seunghyun Wang, Hyun Chang Lee, Sunjae Lee
          </td>
          <td>2024-06-06</td>
          <td>BMC Complementary Medicine and Therapies</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Extracting time-varying latent variables from computational cognitive models is a key step in model-based neural analysis, which aims to understand the neural correlates of cognitive processes. However, existing methods only allow researchers to infer latent variables that explain subjects' behavior in a relatively small class of cognitive models. For example, a broad class of relevant cognitive models with analytically intractable likelihood is currently out of reach from standard techniques, based on Maximum a Posteriori parameter estimation. Here, we present an approach that extends neural Bayes estimation to learn a direct mapping between experimental data and the targeted latent variable space using recurrent neural networks and simulated datasets. We show that our approach achieves competitive performance in inferring latent variable sequences in both tractable and intractable models. Furthermore, the approach is generalizable across different computational models and is adaptable for both continuous and discrete latent spaces. We then demonstrate its applicability in real world datasets. Our work underscores that combining recurrent neural networks and simulation-based inference to identify latent variable sequences can enable researchers to access a wider class of cognitive models for model-based neural analyses, and thus test a broader set of theories.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/67a4c103c072f7b3f0d2c1b6d720fa9f79ed4657" target='_blank'>
              Latent Variable Sequence Identification for Cognitive Models with Neural Bayes Estimation
              </a>
            </td>
          <td>
            Ti-Fen Pan, Jing-Jing Li, Bill Thompson, Anne Collins
          </td>
          <td>2024-06-20</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="The domain of computational chemistry has experienced a significant evolution due to the introduction of Machine Learning (ML) technologies. Despite its potential to revolutionize the field, researchers are often encumbered by obstacles, such as the complexity of selecting optimal algorithms, the automation of data pre-processing steps, the necessity for adaptive feature engineering, and the assurance of model performance consistency across different datasets. Addressing these issues head-on, DeepMol stands out as an Automated ML (AutoML) tool by automating critical steps of the ML pipeline. DeepMol rapidly and automatically identifies the most effective data representation, pre-processing methods and model configurations for a specific molecular property/activity prediction problem. On 22 benchmark datasets, DeepMol obtained competitive pipelines compared with those requiring time-consuming feature engineering, model design and selection processes. As one of the first AutoML tools specifically developed for the computational chemistry domain, DeepMol stands out with its open-source code, in-depth tutorials, detailed documentation, and examples of real-world applications, all available at https://github.com/BioSystemsUM/DeepMol and https://deepmol.readthedocs.io/en/latest/. By introducing AutoML as a ground-breaking feature in computational chemistry, DeepMol establishes itself as the pioneering state-of-the-art tool in the field.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7c5ba098f216308a6a20c38712d1cbe18a657191" target='_blank'>
              DeepMol: An Automated Machine and Deep Learning Framework for Computational Chemistry
              </a>
            </td>
          <td>
            João Correia, João Capela, M. Rocha
          </td>
          <td>2024-06-01</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>4</td>
        </tr>

        <tr id="Modeling dynamical systems, e.g. in climate and engineering sciences, often necessitates solving partial differential equations. Neural operators are deep neural networks designed to learn nontrivial solution operators of such differential equations from data. As for all statistical models, the predictions of these models are imperfect and exhibit errors. Such errors are particularly difficult to spot in the complex nonlinear behaviour of dynamical systems. We introduce a new framework for approximate Bayesian uncertainty quantification in neural operators using function-valued Gaussian processes. Our approach can be interpreted as a probabilistic analogue of the concept of currying from functional programming and provides a practical yet theoretically sound way to apply the linearized Laplace approximation to neural operators. In a case study on Fourier neural operators, we show that, even for a discretized input, our method yields a Gaussian closure--a structured Gaussian process posterior capturing the uncertainty in the output function of the neural operator, which can be evaluated at an arbitrary set of points. The method adds minimal prediction overhead, can be applied post-hoc without retraining the neural operator, and scales to large models and datasets. We showcase the efficacy of our approach through applications to different types of partial differential equations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2fb48eb416a7b362eb444cb5d9c111a6939db5fe" target='_blank'>
              Linearization Turns Neural Operators into Function-Valued Gaussian Processes
              </a>
            </td>
          <td>
            Emilia Magnani, Marvin Pförtner, Tobias Weber, Philipp Hennig
          </td>
          <td>2024-06-07</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>3</td>
        </tr>

        <tr id="Conformal prediction has seen many applications in pharmaceutical science, being able to calibrate outputs of machine learning models and producing valid prediction intervals. We here present the open source software CPSign that is a complete implementation of conformal prediction for cheminformatics modeling. CPSign implements inductive and transductive conformal prediction for classification and regression, and probabilistic prediction with the Venn-ABERS methodology. The main chemical representation is signatures but other types of descriptors are also supported. The main modeling methodology is support vector machines (SVMs), but additional modeling methods are supported via an extension mechanism, e.g. DeepLearning4j models. We also describe features for visualizing results from conformal models including calibration and efficiency plots, as well as features to publish predictive models as REST services. We compare CPSign against other common cheminformatics modeling approaches including random forest, and a directed message-passing neural network. The results show that CPSign produces robust predictive performance with comparative predictive efficiency, with superior runtime and lower hardware requirements compared to neural network based models. CPSign has been used in several studies and is in production-use in multiple organizations. The ability to work directly with chemical input files, perform descriptor calculation and modeling with SVM in the conformal prediction framework, with a single software package having a low footprint and fast execution time makes CPSign a convenient and yet flexible package for training, deploying, and predicting on chemical data.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2ec5cf7da98b190c651db009febf9ee314d81055" target='_blank'>
              CPSign: conformal prediction for cheminformatics modeling
              </a>
            </td>
          <td>
            Staffan Arvidsson McShane, Ulf Norinder, Jonathan Alvarsson, Ernst Ahlberg, Lars Carlsson, O. Spjuth
          </td>
          <td>2023-11-22</td>
          <td>Journal of Cheminformatics</td>
          <td>1</td>
          <td>33</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/f007b787d6d9ef2d12cb5a178877181131370a1f" target='_blank'>
              Nonlinear compartmental modeling to monitor ovarian follicle population dynamics on the whole lifespan.
              </a>
            </td>
          <td>
            Guillaume Ballif, Frédérique Clément, Romain Yvinec
          </td>
          <td>2024-06-06</td>
          <td>Journal of mathematical biology</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="We report synthesis of an open source test data set and work in progress to expand it. Sophisticated data mining and Machine Learning (ML) techniques can discover statistical associations among variables that may or may not reflect actual causal dependencies. In many applications, systems must discriminate between associations that are mere coincidences and those that are at least plausibly causal. Further, a graph of causal relationships may be complex, with fan-in, fan-out, transitive, and various combinations of, dependencies. To test a system’s power to filter out non-causal associations and untangle the causal web, suitable synthetic data is needed. We report the development, in Wolfram Mathematica, of code that synthesizes data with subtle, complex, causal dependencies among some but not all of the generated observable variables. We implement several simple dissipative chaotic flows. Four (4) are autonomous, six (6) are driven. Among the resulting ten (10) observable state vectors, there are forty-five (45) potential pairwise (1:1) relationships, of which four (4) are strong, five (5) are moderate, three (3) are weak, for a total of twelve (12) that are actually causal, and any others are mere statistical artifacts that a tool under test should reject. Each system’s observables are corrupted by additive Gaussian noise. Each system’s hidden dynamics are disturbed by a normal Wiener process. The levels of these stochastic components are parameterized to make problem difficulty tunable. A set of generated data and code for generating more will be released openly on-line.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/8c21bbba8736b5afd19182b829dfd6049fad4e46" target='_blank'>
              Coupled dynamical systems data for testing detection of subtle causal dependencies
              </a>
            </td>
          <td>
            Stuart W. Card
          </td>
          <td>2024-06-07</td>
          <td>None</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="The connection and causality between cancer and neurodevelopmental disorders have been puzzling. How can the same cellular pathways, proteins, and mutations lead to pathologies with vastly different clinical presentations? And why do individuals with neurodevelopmental disorders, such as autism and schizophrenia, face higher chances of cancer emerging throughout their lifetime? Our broad review emphasizes the multi-scale aspect of this type of reasoning. As these examples demonstrate, rather than focusing on a specific organ system or disease, we aim at the new understanding that can be gained. Within this framework, our review calls attention to computational strategies which can be powerful in discovering connections, causalities, predicting clinical outcomes, and are vital for drug discovery. Thus, rather than centering on the clinical features, we draw on the rapidly increasing data on the molecular level, including mutations, isoforms, three-dimensional structures, and expression levels of the respective disease-associated genes. Their integrated analysis, together with chromatin states, can delineate how, despite being connected, neurodevelopmental disorders and cancer differ, and how the same mutations can lead to different clinical symptoms. Here, we seek to uncover the emerging connection between cancer, including pediatric tumors, and neurodevelopmental disorders, and the tantalizing questions that this connection raises.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/6b49a76ac7576a46bdaadfd1301beea5f47bf43b" target='_blank'>
              Review: Cancer and neurodevelopmental disorders: multi-scale reasoning and computational guide
              </a>
            </td>
          <td>
            Ruth Nussinov, B. R. Yavuz, Habibe Cansu Demirel, M. K. Arici, Hyunbum Jang, Nurcan Tuncbag
          </td>
          <td>2024-07-02</td>
          <td>Frontiers in Cell and Developmental Biology</td>
          <td>0</td>
          <td>46</td>
        </tr>

        <tr id="Mass spectrometry-based omics technologies are increasingly used in perturbation studies to map drug effects to biological pathways by identifying significant molecular events. Significance is influenced by fold change and variation of each molecular parameter, but also by multiple testing corrections. While the fold change is largely determined by the biological system, the variation is determined by experimental workflows. Here, it is shown that memory effects of prior subculture can influence the variation of perturbation profiles using the two colon carcinoma cell lines SW480 and HCT116. These memory effects are largely driven by differences in growth states that persist into the perturbation experiment. In SW480 cells, memory effects combined with moderate treatment effects amplify the variation in multiple omics levels, including eicosadomics, proteomics, and phosphoproteomics. With stronger treatment effects, the memory effect was less pronounced, as demonstrated in HCT116 cells. Subculture homogeneity was controlled by real-time monitoring of cell growth. Controlled homogeneous subculture resulted in a perturbation network of 321 causal conjectures based on combined proteomic and phosphoproteomic data, compared to only 58 causal conjectures without controlling subculture homogeneity in SW480 cells. Some cellular responses and regulatory events were identified that extend the mode of action of arsenic trioxide (ATO) only when accounting for these memory effects. Controlled prior subculture led to the finding of a synergistic combination treatment of ATO with the thioredoxin reductase 1 inhibitor auranofin, which may prove useful in the management of NRF2-mediated resistance mechanisms.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/be05f0c8143f944276f17116a13424e530481de5" target='_blank'>
              Memory effects of prior subculture may impact the quality of multiomic perturbation profiles.
              </a>
            </td>
          <td>
            Patricia Bortel, G. Hagn, L. Skos, A. Bileck, Verena Paulitschke, Philipp Paulitschke, Lion Gleiter, Thomas Mohr, Christopher Gerner, Samuel M Meier-Menches
          </td>
          <td>2024-07-08</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>23</td>
        </tr>

        <tr id="Biologically detailed models of brain circuitry are challenging to build and simulate due to the large number of neurons, their complex interactions, and the many unknown physiological parameters. Simplified mathematical models are more tractable, but harder to evaluate when too far removed from neuroanatomy/physiology. We propose that a multiscale model, coarse-grained (CG) while preserving local biological details, offers the best balance between biological realism and computability. This paper presents such a model. Generally, CG models focus on the interaction between groups of neurons-here termed "pixels"-rather than individual cells. In our case, dynamics are alternately updated at intra- and interpixel scales, with one informing the other, until convergence to equilibrium is achieved on both scales. An innovation is how we exploit the underlying biology: Taking advantage of the similarity in local anatomical structures across large regions of the cortex, we model intrapixel dynamics as a single dynamical system driven by "external" inputs. These inputs vary with events external to the pixel, but their ranges can be estimated a priori. Precomputing and tabulating all potential local responses speed up the updating procedure significantly compared to direct multiscale simulation. We illustrate our methodology using a model of the primate visual cortex. Except for local neuron-to-neuron variability (necessarily lost in any CG approximation) our model reproduces various features of large-scale network models at a tiny fraction of the computational cost. These include neuronal responses as a consequence of their orientation selectivity, a primary function of visual neurons.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/1e473f5dcdb151d8bc6c17393e9489ea4f23400e" target='_blank'>
              Efficient models of cortical activity via local dynamic equilibria and coarse-grained interactions.
              </a>
            </td>
          <td>
            Zhuo-Cheng Xiao, Kevin K. Lin, Lai-Sang Young
          </td>
          <td>2024-06-26</td>
          <td>Proceedings of the National Academy of Sciences of the United States of America</td>
          <td>0</td>
          <td>0</td>
        </tr>

        <tr id="Significant interests have recently risen in leveraging sequence-based large language models (LLMs) for drug design. However, most current applications of LLMs in drug discovery lack the ability to comprehend three-dimensional (3D) structures, thereby limiting their effectiveness in tasks that explicitly involve molecular conformations. In this study, we introduced Token-Mol, a token-only 3D drug design model. This model encodes all molecular information, including 2D and 3D structures, as well as molecular property data, into tokens, which transforms classification and regression tasks in drug discovery into probabilistic prediction problems, thereby enabling learning through a unified paradigm. Token-Mol is built on the transformer decoder architecture and trained using random causal masking techniques. Additionally, we proposed the Gaussian cross-entropy (GCE) loss function to overcome the challenges in regression tasks, significantly enhancing the capacity of LLMs to learn continuous numerical values. Through a combination of fine-tuning and reinforcement learning (RL), Token-Mol achieves performance comparable to or surpassing existing task-specific methods across various downstream tasks, including pocket-based molecular generation, conformation generation, and molecular property prediction. Compared to existing molecular pre-trained models, Token-Mol exhibits superior proficiency in handling a wider range of downstream tasks essential for drug design. Notably, our approach improves regression task accuracy by approximately 30% compared to similar token-only methods. Token-Mol overcomes the precision limitations of token-only models and has the potential to integrate seamlessly with general models such as ChatGPT, paving the way for the development of a universal artificial intelligence drug design model that facilitates rapid and high-quality drug design by experts.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/c12c29b5f50659cd7ad9d60c687df454e2dac4c5" target='_blank'>
              Token-Mol 1.0: Tokenized drug design with large language model
              </a>
            </td>
          <td>
            Jike Wang, Rui Qin, Mingyang Wang, Meijing Fang, Yangyang Zhang, Yuchen Zhu, Qun Su, Qiaolin Gou, Chao Shen, Odin Zhang, Zhenxing Wu, Dejun Jiang, Xujun Zhang, Huifeng Zhao, Xiaozhe Wan, Zhourui Wu, Liwei Liu, Yu Kang, Chang-Yu Hsieh, Tingjun Hou
          </td>
          <td>2024-07-10</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="By accelerating time-consuming processes with high efficiency, computing has become an essential part of many modern chemical pipelines. Machine learning is a class of computing methods that can discover patterns within chemical data and utilize this knowledge for a wide variety of downstream tasks, such as property prediction or substance generation. The complex and diverse chemical space requires complex machine learning architectures with great learning power. Recently, learning models based on transformer architectures have revolutionized multiple domains of machine learning, including natural language processing and computer vision. Naturally, there have been ongoing endeavors in adopting these techniques to the chemical domain, resulting in a surge of publications within a short period. The diversity of chemical structures, use cases, and learning models necessitate a comprehensive summarization of existing works. In this paper, we review recent innovations in adapting transformers to solve learning problems in chemistry. Because chemical data is diverse and complex, we structure our discussion based on chemical representations. Specifically, we highlight the strengths and weaknesses of each representation, the current progress of adapting transformer architectures, and future directions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b87d12995df53c3c70af414ab54a764866506152" target='_blank'>
              Application of Transformers in Cheminformatics
              </a>
            </td>
          <td>
            Kha-Dinh Luong, Ambuj Singh
          </td>
          <td>2024-05-30</td>
          <td>Journal of Chemical Information and Modeling</td>
          <td>1</td>
          <td>1</td>
        </tr>

        <tr id="Kolmogorov-Arnold Networks (KAN) is a groundbreaking model recently proposed by the MIT team, representing a revolutionary approach with the potential to be a game-changer in the field. This innovative concept has rapidly garnered worldwide interest within the AI community. Inspired by the Kolmogorov-Arnold representation theorem, KAN utilizes spline-parametrized univariate functions in place of traditional linear weights, enabling them to dynamically learn activation patterns and significantly enhancing interpretability. In this paper, we explore the application of KAN to time series forecasting and propose two variants: T-KAN and MT-KAN. T-KAN is designed to detect concept drift within time series and can explain the nonlinear relationships between predictions and previous time steps through symbolic regression, making it highly interpretable in dynamically changing environments. MT-KAN, on the other hand, improves predictive performance by effectively uncovering and leveraging the complex relationships among variables in multivariate time series. Experiments validate the effectiveness of these approaches, demonstrating that T-KAN and MT-KAN significantly outperform traditional methods in time series forecasting tasks, not only enhancing predictive accuracy but also improving model interpretability. This research opens new avenues for adaptive forecasting models, highlighting the potential of KAN as a powerful and interpretable tool in predictive analytics.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/10145b2238569436754c4d9be3f9c7db501cc65c" target='_blank'>
              Kolmogorov-Arnold Networks for Time Series: Bridging Predictive Power and Interpretability
              </a>
            </td>
          <td>
            Kunpeng Xu, Lifei Chen, Shengrui Wang
          </td>
          <td>2024-06-04</td>
          <td>ArXiv</td>
          <td>7</td>
          <td>4</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2a838384985bce96bffd76cccdd013e7c0961848" target='_blank'>
              Generative deep learning for data generation in natural hazard analysis: motivations, advances, challenges, and opportunities
              </a>
            </td>
          <td>
            Zhengjing Ma, Gang Mei, N. Xu
          </td>
          <td>2024-05-30</td>
          <td>Artif. Intell. Rev.</td>
          <td>0</td>
          <td>17</td>
        </tr>

        <tr id="We consider solving complex spatiotemporal dynamical systems governed by partial differential equations (PDEs) using frequency domain-based discrete learning approaches, such as Fourier neural operators. Despite their widespread use for approximating nonlinear PDEs, the majority of these methods neglect fundamental physical laws and lack interpretability. We address these shortcomings by introducing Physics-embedded Fourier Neural Networks (PeFNN) with flexible and explainable error control. PeFNN is designed to enforce momentum conservation and yields interpretable nonlinear expressions by utilizing unique multi-scale momentum-conserving Fourier (MC-Fourier) layers and an element-wise product operation. The MC-Fourier layer is by design translation- and rotation-invariant in the frequency domain, serving as a plug-and-play module that adheres to the laws of momentum conservation. PeFNN establishes a new state-of-the-art in solving widely employed spatiotemporal PDEs and generalizes well across input resolutions. Further, we demonstrate its outstanding performance for challenging real-world applications such as large-scale flood simulations.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b1987fcbdf3f85b47d4df2efa239c0c5050a329a" target='_blank'>
              Physics-embedded Fourier Neural Network for Partial Differential Equations
              </a>
            </td>
          <td>
            Qingsong Xu, Nils Thuerey, Yilei Shi, Jonathan Bamber, Chaojun Ouyang, Xiao Xiang Zhu
          </td>
          <td>2024-07-15</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>19</td>
        </tr>

        <tr id="Single-cell datasets often lack individual cell labels, making it challenging to identify cells associated with disease. To address this, we introduce Mixture Modeling for Multiple Instance Learning (MMIL), an expectation maximization method that enables the training and calibration of cell-level classifiers using patient-level labels. Our approach can be used to train e.g. lasso logistic regression models, gradient boosted trees, and neural networks. When applied to clinically-annotated, primary patient samples in Acute Myeloid Leukemia (AML) and Acute Lymphoblastic Leukemia (ALL), our method accurately identifies cancer cells, generalizes across tissues and treatment timepoints, and selects biologically relevant features. In addition, MMIL is capable of incorporating cell labels into model training when they are known, providing a powerful framework for leveraging both labeled and unlabeled data simultaneously. Mixture Modeling for MIL offers a novel approach for cell classification, with significant potential to advance disease understanding and management, especially in scenarios with unknown gold-standard labels and high dimensionality.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/3cf614cea92ffef23dcd05fbea55a21a48d81f7c" target='_blank'>
              MMIL: A novel algorithm for disease associated cell type discovery.
              </a>
            </td>
          <td>
            Erin Craig, Timothy Keyes, J. Sarno, Maxim E. Zaslavsky, Garry Nolan, Kara Davis, Trevor Hastie, R. Tibshirani
          </td>
          <td>2024-06-12</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>154</td>
        </tr>

        <tr id="Machine learning (ML) is changing the world of computational protein design, with data- driven methods surpassing biophysical-based methods in experimental success rates. However, they are most often reported as case studies, lack integration and standardization across platforms, and are therefore hard to objectively compare. In this study, we established a streamlined and diverse toolbox for methods that predict amino acid probabilities inside the Rosetta software framework that allows for the side-by-side comparison of these models. Subsequently, existing protein fitness landscapes were used to benchmark novel self- supervised machine learning methods in realistic protein design settings. We focused on the traditional problems of protein sequence design: sampling and scoring. A major finding of our study is that novel ML approaches are better at purging the sampling space from deleterious mutations. Nevertheless, scoring resulting mutations without model fine-tuning showed no clear improvement over scoring with Rosetta. This study fills an important gap in the field and allows for the first time a comprehensive head-to-head comparison of different ML and biophysical methods. We conclude that ML currently acts as a complement to, rather than a replacement for, biophysical methods in protein design.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/5d74eec4ceff71302ee342c63682798408eacb28" target='_blank'>
              Self-supervised machine learning methods for protein design improve sampling, but not the identification of high-fitness variants
              </a>
            </td>
          <td>
            Moritz Ertelt, Rocco Moretti, Jens Meiler, Clara T. Schoeder
          </td>
          <td>2024-06-24</td>
          <td>bioRxiv</td>
          <td>0</td>
          <td>14</td>
        </tr>

        <tr id="Structure-based drug design (SBDD) aims to generate potential drugs that can bind to a target protein and is greatly expedited by the aid of AI techniques in generative models. However, a lack of systematic understanding persists due to the diverse settings, complex implementation, difficult reproducibility, and task singularity. Firstly, the absence of standardization can lead to unfair comparisons and inconclusive insights. To address this dilemma, we propose CBGBench, a comprehensive benchmark for SBDD, that unifies the task as a generative heterogeneous graph completion, analogous to fill-in-the-blank of the 3D complex binding graph. By categorizing existing methods based on their attributes, CBGBench facilitates a modular and extensible framework that implements various cutting-edge methods. Secondly, a single task on \textit{de novo} molecule generation can hardly reflect their capabilities. To broaden the scope, we have adapted these models to a range of tasks essential in drug design, which are considered sub-tasks within the graph fill-in-the-blank tasks. These tasks include the generative designation of \textit{de novo} molecules, linkers, fragments, scaffolds, and sidechains, all conditioned on the structures of protein pockets. Our evaluations are conducted with fairness, encompassing comprehensive perspectives on interaction, chemical properties, geometry authenticity, and substructure validity. We further provide the pre-trained versions of the state-of-the-art models and deep insights with analysis from empirical studies. The codebase for CBGBench is publicly accessible at \url{https://github.com/Edapinenut/CBGBench}.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/2557e9cc9e0c96fccee61e8655e8a4966e5412bc" target='_blank'>
              CBGBench: Fill in the Blank of Protein-Molecule Complex Binding Graph
              </a>
            </td>
          <td>
            Haitao Lin, Guojiang Zhao, Odin Zhang, Yufei Huang, Lirong Wu, Zicheng Liu, Siyuan Li, Cheng Tan, Zhifeng Gao, Stan Z. Li
          </td>
          <td>2024-06-16</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>21</td>
        </tr>

        <tr id="None">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/b393bee7cf5ce703bc767943b9da4324d84f453c" target='_blank'>
              CPMI: comprehensive neighborhood-based perturbed mutual information for identifying critical states of complex biological processes
              </a>
            </td>
          <td>
            Jing Ren, Peiluan Li, Jinling Yan
          </td>
          <td>2024-06-15</td>
          <td>BMC Bioinformatics</td>
          <td>0</td>
          <td>1</td>
        </tr>

        <tr id="In vitro cell biology experiments are routinely used to characterize cell migration properties under various experimental conditions. These experiments can be interpreted using lattice-based random walk models to provide insight into underlying biological mechanisms, and continuum limit partial differential equation (PDE) descriptions of the stochastic models can be used to efficiently explore model properties instead of relying on repeated stochastic simulations. Working with efficient PDE models is of high interest for parameter estimation algorithms that typically require a large number of forward model simulations. Quantitative data from cell biology experiments usually involves non-negative cell counts in different regions of the experimental images, and it is not obvious how to relate finite, noisy count data to the solutions of continuous PDE models that correspond to noise-free density profiles. In this work we illustrate how to develop and implement likelihood-based methods for parameter estimation, parameter identifiability and model prediction for lattice-based models describing collective migration with an arbitrary number of interacting subpopulations. We implement a standard additive Gaussian measurement error model as well as a new physically-motivated multinomial measurement error model that relates noisy count data with the solution of continuous PDE models. Both measurement error models lead to similar outcomes for parameter estimation and parameter identifiability, whereas the standard additive Gaussian measurement error model leads to non-physical prediction outcomes. In contrast, the new multinomial measurement error model involves a lower computational overhead for parameter estimation and identifiability analysis, as well as leading to physically meaningful model predictions.">
          <td id="tag"><i class="material-icons">visibility_off</i></td>
          <td><a href="https://www.semanticscholar.org/paper/7501b082e09fd3e991e69561d882bcf1981f2a48" target='_blank'>
              Likelihood-based inference, identifiability and prediction using count data from lattice-based random walk models
              </a>
            </td>
          <td>
            Yihan Liu, D. Warne, Matthew J. Simpson
          </td>
          <td>2024-06-24</td>
          <td>ArXiv</td>
          <td>0</td>
          <td>12</td>
        </tr>

    </tbody>
    <tfoot>
      <tr>
          <th>Abstract</th>
          <th>Title</th>
          <th>Authors</th>
          <th>Publication Date</th>
          <th>Journal/Conference</th>
          <th>Citation count</th>
          <th>Highest h-index</th>
      </tr>
    </tfoot>
    </table>
    </p>
  </div>

</body>

<script>

  function create_author_list(author_list) {
    let td_author_element = document.getElementById();
    for (let i = 0; i < author_list.length; i++) {
          // tdElements[i].innerHTML = greet(tdElements[i].innerHTML);
          alert (author_list[i]);
      }
  }

  var trace1 = {
    x: ['2023', '2024'],
    y: [1, 27],
    name: 'Num of citations',
    yaxis: 'y1',
    type: 'scatter'
  };

  var data = [trace1];

  var layout = {
    yaxis: {
      title: 'Num of citations',
      }
  };
  Plotly.newPlot('myDiv1', data, layout);
</script>
<script>
var dataTableOptions = {
        initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;

                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);

                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';

                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    scrollX: true,
    scrollCollapse: true,
    paging: true,
    fixedColumns: true,
    columnDefs: [
        {"className": "dt-center", "targets": "_all"},
        // set width for both columns 0 and 1 as 25%
        { width: '7%', targets: 0 },
        { width: '30%', targets: 1 },
        { width: '25%', targets: 2 },
        { width: '15%', targets: 4 }

      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  }
  new DataTable('#table1', dataTableOptions);
  new DataTable('#table2', dataTableOptions);

  var table1 = $('#table1').DataTable();
  $('#table1 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table1.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
    }
  });
  var table2 = $('#table2').DataTable();
  $('#table2 tbody').on('click', 'td:first-child', function () {
    var tr = $(this).closest('tr');
    var row = table2.row( tr );

    var rowId = tr.attr('id');
    // alert(rowId);

    if (row.child.isShown()) {
      // This row is already open - close it.
      row.child.hide();
      tr.removeClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility_off</i>');
    } else {
      // Open row.
      // row.child('foo').show();
      var content = '<div class="child-row-content"><strong>Abstract:</strong> ' + rowId + '</div>';
      row.child(content).show();
      tr.addClass('shown');
      tr.find('td:first-child').html('<i class="material-icons">visibility</i>');
    }
  });
</script>
<style>
  .child-row-content {
    text-align: justify;
    text-justify: inter-word;
    word-wrap: break-word; /* Ensure long words are broken */
    white-space: normal; /* Ensure text wraps to the next line */
    max-width: 100%; /* Ensure content does not exceed the table width */
    padding: 10px; /* Optional: add some padding for better readability */
    /* font size */
    font-size: small;
  }
</style>
</html>







  
  




  



                
              </article>
            </div>
          
          
<script>var target=document.getElementById(location.hash.slice(1));target&&target.name&&(target.checked=target.name.startsWith("__tabbed_"))</script>
        </div>
        
          <button type="button" class="md-top md-icon" data-md-component="top" hidden>
  
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M13 20h-2V8l-5.5 5.5-1.42-1.42L12 4.16l7.92 7.92-1.42 1.42L13 8v12Z"/></svg>
  Back to top
</button>
        
      </main>
      
        <footer class="md-footer">
  
  <div class="md-footer-meta md-typeset">
    <div class="md-footer-meta__inner md-grid">
      <div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
      Material for MkDocs
    </a>
  
</div>
      
    </div>
  </div>
</footer>
      
    </div>
    <div class="md-dialog" data-md-component="dialog">
      <div class="md-dialog__inner md-typeset"></div>
    </div>
    
    
    <script id="__config" type="application/json">{"base": "..", "features": ["navigation.top", "navigation.tabs"], "search": "../assets/javascripts/workers/search.b8dbb3d2.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version": "Select version"}}</script>
    
    

      <script src="../assets/javascripts/bundle.c8d2eff1.min.js"></script>
      
    
<script>
  // Execute intro.js when a button with id 'intro' is clicked
  function startIntro(){
      introJs().setOptions({
          tooltipClass: 'customTooltip'
      }).start();
  }
</script>
<script>
  

  // new DataTable('#table1', {
  //   order: [[5, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });

  // new DataTable('#table2', {
  //   order: [[3, 'desc']],
  //   "columnDefs": [
  //       {"className": "dt-center", "targets": "_all"},
  //       { width: '30%', targets: 0 }
  //     ],
  //   pageLength: 10,
  //   layout: {
  //       topStart: {
  //           buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
  //       }
  //   }
  // });
  new DataTable('#table3', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
  new DataTable('#table4', {
    initComplete: function () {
        this.api()
            .columns()
            .every(function () {
                let column = this;
 
                // Create select element
                let select = document.createElement('select');
                select.add(new Option(''));
                column.footer().replaceChildren(select);
 
                // Apply listener for user change in value
                select.addEventListener('change', function () {
                    column
                        .search(select.value, {exact: true})
                        .draw();
                });

                // keep the width of the select element same as the column
                select.style.width = '100%';
 
                // Add list of options
                column
                    .data()
                    .unique()
                    .sort()
                    .each(function (d, j) {
                        select.add(new Option(d));
                    });
            });
    },
    // order: [[3, 'desc']],
    "columnDefs": [
        {"className": "dt-center", "targets": "_all"},
        { width: '30%', targets: 0 }
      ],
    pageLength: 10,
    layout: {
        topStart: {
            buttons: ['copy', 'csv', 'excel', 'pdf', 'print']
        }
    }
  });
</script>


  </body>
</html>